{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Data Processing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "17-Mar-2025 22:45:12 DEBUG utils - Directory ./ already exists. Skipping.\n",
      "17-Mar-2025 22:45:12 INFO GEOparse - File already exist: using local version.\n",
      "17-Mar-2025 22:45:12 INFO GEOparse - Parsing ./GSE36059_family.soft.gz: \n",
      "17-Mar-2025 22:45:12 DEBUG GEOparse - DATABASE: GeoMiame\n",
      "17-Mar-2025 22:45:12 DEBUG GEOparse - SERIES: GSE36059\n",
      "17-Mar-2025 22:45:12 DEBUG GEOparse - PLATFORM: GPL570\n",
      "C:\\Users\\ajxxs\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\GEOparse\\GEOparse.py:401: DtypeWarning: Columns (2) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  return read_csv(StringIO(data), index_col=None, sep=\"\\t\")\n",
      "17-Mar-2025 22:45:12 DEBUG GEOparse - SAMPLE: GSM880204\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880205\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880206\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880207\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880208\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880209\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880210\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880211\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880212\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880213\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880214\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880215\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880216\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880217\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880218\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880219\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880220\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880221\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880222\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880223\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880224\n",
      "17-Mar-2025 22:45:13 DEBUG GEOparse - SAMPLE: GSM880225\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880226\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880227\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880228\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880229\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880230\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880231\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880232\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880233\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880234\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880235\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880236\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880237\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880238\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880239\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880240\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880241\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880242\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880243\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880244\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880245\n",
      "17-Mar-2025 22:45:14 DEBUG GEOparse - SAMPLE: GSM880246\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880247\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880248\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880249\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880250\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880251\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880252\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880253\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880254\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880255\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880256\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880257\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880258\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880259\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880260\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880261\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880262\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880263\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880264\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880265\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880266\n",
      "17-Mar-2025 22:45:15 DEBUG GEOparse - SAMPLE: GSM880267\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880268\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880269\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880270\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880271\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880272\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880273\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880274\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880275\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880276\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880277\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880278\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880279\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880280\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880281\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880282\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880283\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880284\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880285\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880286\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880287\n",
      "17-Mar-2025 22:45:16 DEBUG GEOparse - SAMPLE: GSM880288\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880289\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880290\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880291\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880292\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880293\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880294\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880295\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880296\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880297\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880298\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880299\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880300\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880301\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880302\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880303\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880304\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880305\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880306\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880307\n",
      "17-Mar-2025 22:45:17 DEBUG GEOparse - SAMPLE: GSM880308\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880309\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880310\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880311\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880312\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880313\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880314\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880315\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880316\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880317\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880318\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880319\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880320\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880321\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880322\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880323\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880324\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880325\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880326\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880327\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880328\n",
      "17-Mar-2025 22:45:18 DEBUG GEOparse - SAMPLE: GSM880329\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880330\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880331\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880332\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880333\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880334\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880335\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880336\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880337\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880338\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880339\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880340\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880341\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880342\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880343\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880344\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880345\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880346\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880347\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880348\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880349\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880350\n",
      "17-Mar-2025 22:45:19 DEBUG GEOparse - SAMPLE: GSM880351\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880352\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880353\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880354\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880355\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880356\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880357\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880358\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880359\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880360\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880361\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880362\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880363\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880364\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880365\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880366\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880367\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880368\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880369\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880370\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880371\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880372\n",
      "17-Mar-2025 22:45:20 DEBUG GEOparse - SAMPLE: GSM880373\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880374\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880375\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880376\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880377\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880378\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880379\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880380\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880381\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880382\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880383\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880384\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880385\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880386\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880387\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880388\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880389\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880390\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880391\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880392\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880393\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880394\n",
      "17-Mar-2025 22:45:21 DEBUG GEOparse - SAMPLE: GSM880395\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880396\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880397\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880398\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880399\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880400\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880401\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880402\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880403\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880404\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880405\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880406\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880407\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880408\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880409\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880410\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880411\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880412\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880413\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880414\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880415\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880416\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880417\n",
      "17-Mar-2025 22:45:22 DEBUG GEOparse - SAMPLE: GSM880418\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880419\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880420\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880421\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880422\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880423\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880424\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880425\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880426\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880427\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880428\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880429\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880430\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880431\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880432\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880433\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880434\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880435\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880436\n",
      "17-Mar-2025 22:45:23 DEBUG GEOparse - SAMPLE: GSM880437\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880438\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880439\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880440\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880441\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880442\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880443\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880444\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880445\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880446\n",
      "17-Mar-2025 22:45:24 DEBUG GEOparse - SAMPLE: GSM880447\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880448\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880449\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880450\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880451\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880452\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880453\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880454\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880455\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880456\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880457\n",
      "17-Mar-2025 22:45:25 DEBUG GEOparse - SAMPLE: GSM880458\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880459\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880460\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880461\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880462\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880463\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880464\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880465\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880466\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880467\n",
      "17-Mar-2025 22:45:26 DEBUG GEOparse - SAMPLE: GSM880468\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880469\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880470\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880471\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880472\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880473\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880474\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880475\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880476\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880477\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880478\n",
      "17-Mar-2025 22:45:27 DEBUG GEOparse - SAMPLE: GSM880479\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880480\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880481\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880482\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880483\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880484\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880485\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880486\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880487\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880488\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880489\n",
      "17-Mar-2025 22:45:28 DEBUG GEOparse - SAMPLE: GSM880490\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880491\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880492\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880493\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880494\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880495\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880496\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880497\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880498\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880499\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880500\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880501\n",
      "17-Mar-2025 22:45:29 DEBUG GEOparse - SAMPLE: GSM880502\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880503\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880504\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880505\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880506\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880507\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880508\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880509\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880510\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880511\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880512\n",
      "17-Mar-2025 22:45:30 DEBUG GEOparse - SAMPLE: GSM880513\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880514\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880515\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880516\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880517\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880518\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880519\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880520\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880521\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880522\n",
      "17-Mar-2025 22:45:31 DEBUG GEOparse - SAMPLE: GSM880523\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880524\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880525\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880526\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880527\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880528\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880529\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880530\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880531\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880532\n",
      "17-Mar-2025 22:45:32 DEBUG GEOparse - SAMPLE: GSM880533\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880534\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880535\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880536\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880537\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880538\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880539\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880540\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880541\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880542\n",
      "17-Mar-2025 22:45:33 DEBUG GEOparse - SAMPLE: GSM880543\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880544\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880545\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880546\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880547\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880548\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880549\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880550\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880551\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880552\n",
      "17-Mar-2025 22:45:34 DEBUG GEOparse - SAMPLE: GSM880553\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880554\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880555\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880556\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880557\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880558\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880559\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880560\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880561\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880562\n",
      "17-Mar-2025 22:45:35 DEBUG GEOparse - SAMPLE: GSM880563\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880564\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880565\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880566\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880567\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880568\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880569\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880570\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880571\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880572\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880573\n",
      "17-Mar-2025 22:45:36 DEBUG GEOparse - SAMPLE: GSM880574\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880575\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880576\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880577\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880578\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880579\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880580\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880581\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880582\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880583\n",
      "17-Mar-2025 22:45:37 DEBUG GEOparse - SAMPLE: GSM880584\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880585\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880586\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880587\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880588\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880589\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880590\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880591\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880592\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880593\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880594\n",
      "17-Mar-2025 22:45:38 DEBUG GEOparse - SAMPLE: GSM880595\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880596\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880597\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880598\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880599\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880600\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880601\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880602\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880603\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880604\n",
      "17-Mar-2025 22:45:39 DEBUG GEOparse - SAMPLE: GSM880605\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880606\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880607\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880608\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880609\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880610\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880611\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880612\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880613\n",
      "17-Mar-2025 22:45:40 DEBUG GEOparse - SAMPLE: GSM880614\n"
     ]
    }
   ],
   "source": [
    "import GEOparse\n",
    "\n",
    "dest_folder = \"./GSE36059_data\"\n",
    "\n",
    "gse = GEOparse.get_GEO(\"GSE36059\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sample size： 411\n",
      "dict_keys(['GSM880204', 'GSM880205', 'GSM880206', 'GSM880207', 'GSM880208', 'GSM880209', 'GSM880210', 'GSM880211', 'GSM880212', 'GSM880213', 'GSM880214', 'GSM880215', 'GSM880216', 'GSM880217', 'GSM880218', 'GSM880219', 'GSM880220', 'GSM880221', 'GSM880222', 'GSM880223', 'GSM880224', 'GSM880225', 'GSM880226', 'GSM880227', 'GSM880228', 'GSM880229', 'GSM880230', 'GSM880231', 'GSM880232', 'GSM880233', 'GSM880234', 'GSM880235', 'GSM880236', 'GSM880237', 'GSM880238', 'GSM880239', 'GSM880240', 'GSM880241', 'GSM880242', 'GSM880243', 'GSM880244', 'GSM880245', 'GSM880246', 'GSM880247', 'GSM880248', 'GSM880249', 'GSM880250', 'GSM880251', 'GSM880252', 'GSM880253', 'GSM880254', 'GSM880255', 'GSM880256', 'GSM880257', 'GSM880258', 'GSM880259', 'GSM880260', 'GSM880261', 'GSM880262', 'GSM880263', 'GSM880264', 'GSM880265', 'GSM880266', 'GSM880267', 'GSM880268', 'GSM880269', 'GSM880270', 'GSM880271', 'GSM880272', 'GSM880273', 'GSM880274', 'GSM880275', 'GSM880276', 'GSM880277', 'GSM880278', 'GSM880279', 'GSM880280', 'GSM880281', 'GSM880282', 'GSM880283', 'GSM880284', 'GSM880285', 'GSM880286', 'GSM880287', 'GSM880288', 'GSM880289', 'GSM880290', 'GSM880291', 'GSM880292', 'GSM880293', 'GSM880294', 'GSM880295', 'GSM880296', 'GSM880297', 'GSM880298', 'GSM880299', 'GSM880300', 'GSM880301', 'GSM880302', 'GSM880303', 'GSM880304', 'GSM880305', 'GSM880306', 'GSM880307', 'GSM880308', 'GSM880309', 'GSM880310', 'GSM880311', 'GSM880312', 'GSM880313', 'GSM880314', 'GSM880315', 'GSM880316', 'GSM880317', 'GSM880318', 'GSM880319', 'GSM880320', 'GSM880321', 'GSM880322', 'GSM880323', 'GSM880324', 'GSM880325', 'GSM880326', 'GSM880327', 'GSM880328', 'GSM880329', 'GSM880330', 'GSM880331', 'GSM880332', 'GSM880333', 'GSM880334', 'GSM880335', 'GSM880336', 'GSM880337', 'GSM880338', 'GSM880339', 'GSM880340', 'GSM880341', 'GSM880342', 'GSM880343', 'GSM880344', 'GSM880345', 'GSM880346', 'GSM880347', 'GSM880348', 'GSM880349', 'GSM880350', 'GSM880351', 'GSM880352', 'GSM880353', 'GSM880354', 'GSM880355', 'GSM880356', 'GSM880357', 'GSM880358', 'GSM880359', 'GSM880360', 'GSM880361', 'GSM880362', 'GSM880363', 'GSM880364', 'GSM880365', 'GSM880366', 'GSM880367', 'GSM880368', 'GSM880369', 'GSM880370', 'GSM880371', 'GSM880372', 'GSM880373', 'GSM880374', 'GSM880375', 'GSM880376', 'GSM880377', 'GSM880378', 'GSM880379', 'GSM880380', 'GSM880381', 'GSM880382', 'GSM880383', 'GSM880384', 'GSM880385', 'GSM880386', 'GSM880387', 'GSM880388', 'GSM880389', 'GSM880390', 'GSM880391', 'GSM880392', 'GSM880393', 'GSM880394', 'GSM880395', 'GSM880396', 'GSM880397', 'GSM880398', 'GSM880399', 'GSM880400', 'GSM880401', 'GSM880402', 'GSM880403', 'GSM880404', 'GSM880405', 'GSM880406', 'GSM880407', 'GSM880408', 'GSM880409', 'GSM880410', 'GSM880411', 'GSM880412', 'GSM880413', 'GSM880414', 'GSM880415', 'GSM880416', 'GSM880417', 'GSM880418', 'GSM880419', 'GSM880420', 'GSM880421', 'GSM880422', 'GSM880423', 'GSM880424', 'GSM880425', 'GSM880426', 'GSM880427', 'GSM880428', 'GSM880429', 'GSM880430', 'GSM880431', 'GSM880432', 'GSM880433', 'GSM880434', 'GSM880435', 'GSM880436', 'GSM880437', 'GSM880438', 'GSM880439', 'GSM880440', 'GSM880441', 'GSM880442', 'GSM880443', 'GSM880444', 'GSM880445', 'GSM880446', 'GSM880447', 'GSM880448', 'GSM880449', 'GSM880450', 'GSM880451', 'GSM880452', 'GSM880453', 'GSM880454', 'GSM880455', 'GSM880456', 'GSM880457', 'GSM880458', 'GSM880459', 'GSM880460', 'GSM880461', 'GSM880462', 'GSM880463', 'GSM880464', 'GSM880465', 'GSM880466', 'GSM880467', 'GSM880468', 'GSM880469', 'GSM880470', 'GSM880471', 'GSM880472', 'GSM880473', 'GSM880474', 'GSM880475', 'GSM880476', 'GSM880477', 'GSM880478', 'GSM880479', 'GSM880480', 'GSM880481', 'GSM880482', 'GSM880483', 'GSM880484', 'GSM880485', 'GSM880486', 'GSM880487', 'GSM880488', 'GSM880489', 'GSM880490', 'GSM880491', 'GSM880492', 'GSM880493', 'GSM880494', 'GSM880495', 'GSM880496', 'GSM880497', 'GSM880498', 'GSM880499', 'GSM880500', 'GSM880501', 'GSM880502', 'GSM880503', 'GSM880504', 'GSM880505', 'GSM880506', 'GSM880507', 'GSM880508', 'GSM880509', 'GSM880510', 'GSM880511', 'GSM880512', 'GSM880513', 'GSM880514', 'GSM880515', 'GSM880516', 'GSM880517', 'GSM880518', 'GSM880519', 'GSM880520', 'GSM880521', 'GSM880522', 'GSM880523', 'GSM880524', 'GSM880525', 'GSM880526', 'GSM880527', 'GSM880528', 'GSM880529', 'GSM880530', 'GSM880531', 'GSM880532', 'GSM880533', 'GSM880534', 'GSM880535', 'GSM880536', 'GSM880537', 'GSM880538', 'GSM880539', 'GSM880540', 'GSM880541', 'GSM880542', 'GSM880543', 'GSM880544', 'GSM880545', 'GSM880546', 'GSM880547', 'GSM880548', 'GSM880549', 'GSM880550', 'GSM880551', 'GSM880552', 'GSM880553', 'GSM880554', 'GSM880555', 'GSM880556', 'GSM880557', 'GSM880558', 'GSM880559', 'GSM880560', 'GSM880561', 'GSM880562', 'GSM880563', 'GSM880564', 'GSM880565', 'GSM880566', 'GSM880567', 'GSM880568', 'GSM880569', 'GSM880570', 'GSM880571', 'GSM880572', 'GSM880573', 'GSM880574', 'GSM880575', 'GSM880576', 'GSM880577', 'GSM880578', 'GSM880579', 'GSM880580', 'GSM880581', 'GSM880582', 'GSM880583', 'GSM880584', 'GSM880585', 'GSM880586', 'GSM880587', 'GSM880588', 'GSM880589', 'GSM880590', 'GSM880591', 'GSM880592', 'GSM880593', 'GSM880594', 'GSM880595', 'GSM880596', 'GSM880597', 'GSM880598', 'GSM880599', 'GSM880600', 'GSM880601', 'GSM880602', 'GSM880603', 'GSM880604', 'GSM880605', 'GSM880606', 'GSM880607', 'GSM880608', 'GSM880609', 'GSM880610', 'GSM880611', 'GSM880612', 'GSM880613', 'GSM880614'])\n"
     ]
    }
   ],
   "source": [
    "print(\"Sample size：\", len(gse.gsms))  # gsms stores all samples (GSM)\n",
    "print(gse.gsms.keys())            # You can see the ID of each GSM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': ['Molecular diagnosis of T cell-mediated rejection in human kidney transplant biopsies; Molecular diagnosis of antibody-mediated rejection in human kidney transplants'], 'geo_accession': ['GSE36059'], 'status': ['Public on Feb 13 2013'], 'submission_date': ['Feb 24 2012'], 'last_update_date': ['Jun 11 2022'], 'pubmed_id': ['23356949', '24700874', '25377077'], 'summary': ['Histologic diagnosis of T cell-mediated rejection in kidney transplant biopsies has limited reproducibility because it is based on non-specific lesions using arbitrary rules that are subject to differing interpretations. We used microarray results from 403 indication biopsies previously given histologic diagnoses to develop a molecular classifier that assigned a molecular T cell-mediated rejection score to each biopsy. Independent assessment of the biopsies by multiple pathologists confirmed considerable disagreement on the presence of TCMR features: 79-88% accuracy and 35-69% sensitivity. The agreement of the molecular T cell-mediated rejection score with the histology diagnosis was similar to agreement among individual pathologists: accuracy 89%, sensitivity 51%. However, the score also predicted the consensus among pathologists, being highest when all agreed. Many discrepancies between the scores and the histologic diagnoses were in situations where histology is unreliable e.g. scarred biopsies. The score correlated with histologic lesions and gene sets associated with T cell-mediated rejection. The transcripts most often selected by the classifier were expressed in effector T cells, dendritic cells, or macrophages or inducible by interferon-gamma. Thus the T cell-mediated rejection score offers an objective assessment of kidney transplant biopsies, predicting the consensus opinion among multiple pathologists, and offering insights into underlying disease mechanisms.', '', 'Antibody-mediated rejection is a major cause of kidney transplant failure, but the current diagnostic system misses most cases due to dependency on subjective non-standardized tests. We hypothesized that molecular features could provide a test to address this problem. We classified 403 biopsies by a reference standard based on microcirculation lesions and donor-specific HLA antibody, and used microarray analysis to develop a classifier that assigned each biopsy a score reflecting the probability of antibody-mediated rejection. The scores correlated with donor-specific antibody and histologic lesions: 42/45 biopsies with antibody-mediated rejection scores >0.5 had both donor-specific antibody and microcirculation lesions. Intermediate scores (0.2-0.5) were more ambiguous, but became more specific combined with donor-specific antibody. Compared to diagnoses based on histology-plus-donor-specific antibody, the scores had sensitivity 0.67; specificity 0.90. Donor-specific antibody improved the specificity to 0.97. The score correlated not only with diagnoses of individual pathologists but with the consensus among multiple pathologists. The classifier used transcripts expressed in endothelial cells (e.g. CDH13, DARC, ROBO4) and NK cells (e.g. CX3CR1, FGFBP2), as well as IFNG-inducible transcripts e.g. CXCL11. Thus the molecular phenotype of antibody-mediated rejection provides not only an objective test that predicts microcirculation lesions and donor-specific HLA antibody, but also offers mechanistic insights.'], 'overall_design': ['All consenting renal transplant patients undergoing biopsies for cause as standard of care. 403 samples and 8 controls (nephrectomies).', '', \"Note: The Sample 'characteristics' were updated on June 11, 2022.\"], 'type': ['Expression profiling by array'], 'contributor': ['Joana,,Selleres', 'Jeff,,Reeve', 'Philip,F,Halloran'], 'sample_id': ['GSM880204', 'GSM880205', 'GSM880206', 'GSM880207', 'GSM880208', 'GSM880209', 'GSM880210', 'GSM880211', 'GSM880212', 'GSM880213', 'GSM880214', 'GSM880215', 'GSM880216', 'GSM880217', 'GSM880218', 'GSM880219', 'GSM880220', 'GSM880221', 'GSM880222', 'GSM880223', 'GSM880224', 'GSM880225', 'GSM880226', 'GSM880227', 'GSM880228', 'GSM880229', 'GSM880230', 'GSM880231', 'GSM880232', 'GSM880233', 'GSM880234', 'GSM880235', 'GSM880236', 'GSM880237', 'GSM880238', 'GSM880239', 'GSM880240', 'GSM880241', 'GSM880242', 'GSM880243', 'GSM880244', 'GSM880245', 'GSM880246', 'GSM880247', 'GSM880248', 'GSM880249', 'GSM880250', 'GSM880251', 'GSM880252', 'GSM880253', 'GSM880254', 'GSM880255', 'GSM880256', 'GSM880257', 'GSM880258', 'GSM880259', 'GSM880260', 'GSM880261', 'GSM880262', 'GSM880263', 'GSM880264', 'GSM880265', 'GSM880266', 'GSM880267', 'GSM880268', 'GSM880269', 'GSM880270', 'GSM880271', 'GSM880272', 'GSM880273', 'GSM880274', 'GSM880275', 'GSM880276', 'GSM880277', 'GSM880278', 'GSM880279', 'GSM880280', 'GSM880281', 'GSM880282', 'GSM880283', 'GSM880284', 'GSM880285', 'GSM880286', 'GSM880287', 'GSM880288', 'GSM880289', 'GSM880290', 'GSM880291', 'GSM880292', 'GSM880293', 'GSM880294', 'GSM880295', 'GSM880296', 'GSM880297', 'GSM880298', 'GSM880299', 'GSM880300', 'GSM880301', 'GSM880302', 'GSM880303', 'GSM880304', 'GSM880305', 'GSM880306', 'GSM880307', 'GSM880308', 'GSM880309', 'GSM880310', 'GSM880311', 'GSM880312', 'GSM880313', 'GSM880314', 'GSM880315', 'GSM880316', 'GSM880317', 'GSM880318', 'GSM880319', 'GSM880320', 'GSM880321', 'GSM880322', 'GSM880323', 'GSM880324', 'GSM880325', 'GSM880326', 'GSM880327', 'GSM880328', 'GSM880329', 'GSM880330', 'GSM880331', 'GSM880332', 'GSM880333', 'GSM880334', 'GSM880335', 'GSM880336', 'GSM880337', 'GSM880338', 'GSM880339', 'GSM880340', 'GSM880341', 'GSM880342', 'GSM880343', 'GSM880344', 'GSM880345', 'GSM880346', 'GSM880347', 'GSM880348', 'GSM880349', 'GSM880350', 'GSM880351', 'GSM880352', 'GSM880353', 'GSM880354', 'GSM880355', 'GSM880356', 'GSM880357', 'GSM880358', 'GSM880359', 'GSM880360', 'GSM880361', 'GSM880362', 'GSM880363', 'GSM880364', 'GSM880365', 'GSM880366', 'GSM880367', 'GSM880368', 'GSM880369', 'GSM880370', 'GSM880371', 'GSM880372', 'GSM880373', 'GSM880374', 'GSM880375', 'GSM880376', 'GSM880377', 'GSM880378', 'GSM880379', 'GSM880380', 'GSM880381', 'GSM880382', 'GSM880383', 'GSM880384', 'GSM880385', 'GSM880386', 'GSM880387', 'GSM880388', 'GSM880389', 'GSM880390', 'GSM880391', 'GSM880392', 'GSM880393', 'GSM880394', 'GSM880395', 'GSM880396', 'GSM880397', 'GSM880398', 'GSM880399', 'GSM880400', 'GSM880401', 'GSM880402', 'GSM880403', 'GSM880404', 'GSM880405', 'GSM880406', 'GSM880407', 'GSM880408', 'GSM880409', 'GSM880410', 'GSM880411', 'GSM880412', 'GSM880413', 'GSM880414', 'GSM880415', 'GSM880416', 'GSM880417', 'GSM880418', 'GSM880419', 'GSM880420', 'GSM880421', 'GSM880422', 'GSM880423', 'GSM880424', 'GSM880425', 'GSM880426', 'GSM880427', 'GSM880428', 'GSM880429', 'GSM880430', 'GSM880431', 'GSM880432', 'GSM880433', 'GSM880434', 'GSM880435', 'GSM880436', 'GSM880437', 'GSM880438', 'GSM880439', 'GSM880440', 'GSM880441', 'GSM880442', 'GSM880443', 'GSM880444', 'GSM880445', 'GSM880446', 'GSM880447', 'GSM880448', 'GSM880449', 'GSM880450', 'GSM880451', 'GSM880452', 'GSM880453', 'GSM880454', 'GSM880455', 'GSM880456', 'GSM880457', 'GSM880458', 'GSM880459', 'GSM880460', 'GSM880461', 'GSM880462', 'GSM880463', 'GSM880464', 'GSM880465', 'GSM880466', 'GSM880467', 'GSM880468', 'GSM880469', 'GSM880470', 'GSM880471', 'GSM880472', 'GSM880473', 'GSM880474', 'GSM880475', 'GSM880476', 'GSM880477', 'GSM880478', 'GSM880479', 'GSM880480', 'GSM880481', 'GSM880482', 'GSM880483', 'GSM880484', 'GSM880485', 'GSM880486', 'GSM880487', 'GSM880488', 'GSM880489', 'GSM880490', 'GSM880491', 'GSM880492', 'GSM880493', 'GSM880494', 'GSM880495', 'GSM880496', 'GSM880497', 'GSM880498', 'GSM880499', 'GSM880500', 'GSM880501', 'GSM880502', 'GSM880503', 'GSM880504', 'GSM880505', 'GSM880506', 'GSM880507', 'GSM880508', 'GSM880509', 'GSM880510', 'GSM880511', 'GSM880512', 'GSM880513', 'GSM880514', 'GSM880515', 'GSM880516', 'GSM880517', 'GSM880518', 'GSM880519', 'GSM880520', 'GSM880521', 'GSM880522', 'GSM880523', 'GSM880524', 'GSM880525', 'GSM880526', 'GSM880527', 'GSM880528', 'GSM880529', 'GSM880530', 'GSM880531', 'GSM880532', 'GSM880533', 'GSM880534', 'GSM880535', 'GSM880536', 'GSM880537', 'GSM880538', 'GSM880539', 'GSM880540', 'GSM880541', 'GSM880542', 'GSM880543', 'GSM880544', 'GSM880545', 'GSM880546', 'GSM880547', 'GSM880548', 'GSM880549', 'GSM880550', 'GSM880551', 'GSM880552', 'GSM880553', 'GSM880554', 'GSM880555', 'GSM880556', 'GSM880557', 'GSM880558', 'GSM880559', 'GSM880560', 'GSM880561', 'GSM880562', 'GSM880563', 'GSM880564', 'GSM880565', 'GSM880566', 'GSM880567', 'GSM880568', 'GSM880569', 'GSM880570', 'GSM880571', 'GSM880572', 'GSM880573', 'GSM880574', 'GSM880575', 'GSM880576', 'GSM880577', 'GSM880578', 'GSM880579', 'GSM880580', 'GSM880581', 'GSM880582', 'GSM880583', 'GSM880584', 'GSM880585', 'GSM880586', 'GSM880587', 'GSM880588', 'GSM880589', 'GSM880590', 'GSM880591', 'GSM880592', 'GSM880593', 'GSM880594', 'GSM880595', 'GSM880596', 'GSM880597', 'GSM880598', 'GSM880599', 'GSM880600', 'GSM880601', 'GSM880602', 'GSM880603', 'GSM880604', 'GSM880605', 'GSM880606', 'GSM880607', 'GSM880608', 'GSM880609', 'GSM880610', 'GSM880611', 'GSM880612', 'GSM880613', 'GSM880614'], 'contact_name': ['Jessica,,Chang'], 'contact_email': ['jjchang@ualberta.ca'], 'contact_laboratory': ['ATAGC'], 'contact_department': ['Medicine'], 'contact_institute': ['University of Alberta'], 'contact_address': ['250 Heritage Medical Research Centre'], 'contact_city': ['Edmonton'], 'contact_zip/postal_code': ['T6G 2S2'], 'contact_country': ['Canada'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE36nnn/GSE36059/suppl/GSE36059_RAW.tar'], 'platform_id': ['GPL570'], 'platform_taxid': ['9606'], 'sample_taxid': ['9606'], 'relation': ['BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA151807']}\n"
     ]
    }
   ],
   "source": [
    "print(gse.metadata)  # View meta-information at the GSE level"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# 创建空的 DataFrame 用来合并所有样本\n",
    "expression_data = None\n",
    "\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    # gsm.table 通常是一个DataFrame，具体列名取决于平台类型、GEO数据格式\n",
    "    # 常见的列名包括: ID_REF, VALUE, ...\n",
    "    # 可以通过 gsm.table.columns 查看有哪些列\n",
    "\n",
    "    # 举例：如果在 gsm.table 中，\"ID_REF\" 是探针ID，\"VALUE\" 是信号值\n",
    "    df_sample = gsm.table[[\"ID_REF\", \"VALUE\"]].copy()\n",
    "    df_sample.columns = [\"Probe\", gsm_name]  # 重命名列，后者为样本名\n",
    "\n",
    "    if expression_data is None:\n",
    "        expression_data = df_sample\n",
    "    else:\n",
    "        # 按探针ID合并\n",
    "        expression_data = pd.merge(expression_data, df_sample, on=\"Probe\", how=\"outer\")\n",
    "\n",
    "# expression_data 的行数是探针数(或基因数)，列数是1（探针）+样本数\n",
    "# 将 Probe 设为行索引\n",
    "expression_data.set_index(\"Probe\", inplace=True)\n",
    "\n",
    "# 至此, expression_data 的列即各GSM样本，行是探针ID。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  Unnamed: 0  DDR1 /// MIR4640      RFC2     HSPA6       PAX8    GUCA1A  \\\n",
      "0  GSM880204          9.808114  6.039012  5.810846  10.342781  3.211123   \n",
      "1  GSM880205         10.682125  6.608906  5.847419  10.667633  2.924908   \n",
      "2  GSM880206          9.659079  6.025781  6.089887  10.385927  3.191762   \n",
      "3  GSM880207         10.083274  6.254128  5.386257  10.383023  3.096177   \n",
      "4  GSM880208         10.460354  6.304075  5.550745  11.367598  3.059629   \n",
      "\n",
      "   MIR5193 /// UBA7      THRA    PTPN21      CCL5  ...  AFFX-r2-Ec-bioB-3_at  \\\n",
      "0          7.615631  5.473264  5.130847  5.967407  ...              8.405380   \n",
      "1          8.282238  5.222516  4.620912  8.913122  ...              8.385829   \n",
      "2          7.013927  5.483255  5.346077  5.872588  ...              8.621434   \n",
      "3          7.217460  5.364841  4.968738  5.622734  ...              9.197614   \n",
      "4          7.658014  5.402205  5.469187  4.728435  ...              8.391848   \n",
      "\n",
      "   AFFX-r2-Ec-bioB-5_at  AFFX-r2-Ec-bioB-M_at  AFFX-r2-Ec-bioC-3_at  \\\n",
      "0              7.976476              7.577744              9.686881   \n",
      "1              8.377783              8.108189             10.007514   \n",
      "2              8.669645              8.406157             10.095641   \n",
      "3              9.086772              8.752838             10.489239   \n",
      "4              8.082121              8.007603              9.887173   \n",
      "\n",
      "   AFFX-r2-Ec-bioC-5_at  AFFX-r2-Ec-bioD-3_at  AFFX-r2-Ec-bioD-5_at  \\\n",
      "0              9.287713             11.949707             11.562166   \n",
      "1              9.596678             12.094172             11.836863   \n",
      "2              9.831340             12.238378             11.861172   \n",
      "3             10.235579             12.551390             12.098305   \n",
      "4              9.587661             12.296905             11.817117   \n",
      "\n",
      "   AFFX-r2-P1-cre-3_at  AFFX-r2-P1-cre-5_at      Diagnosis  \n",
      "0            13.446336            13.311750  Non-rejecting  \n",
      "1            13.640625            13.376220  Non-rejecting  \n",
      "2            13.875352            13.715878  Non-rejecting  \n",
      "3            13.951246            13.789232  Non-rejecting  \n",
      "4            13.796431            13.553796  Non-rejecting  \n",
      "\n",
      "[5 rows x 54677 columns]\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# 读取CSV文件\n",
    "df = pd.read_csv('gse36059.csv')  # 替换为你的文件名\n",
    "\n",
    "# 显示前五行数据\n",
    "print(df.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>DDR1 /// MIR4640</th>\n",
       "      <th>RFC2</th>\n",
       "      <th>HSPA6</th>\n",
       "      <th>PAX8</th>\n",
       "      <th>GUCA1A</th>\n",
       "      <th>MIR5193 /// UBA7</th>\n",
       "      <th>THRA</th>\n",
       "      <th>PTPN21</th>\n",
       "      <th>CCL5</th>\n",
       "      <th>...</th>\n",
       "      <th>AFFX-r2-Ec-bioB-3_at</th>\n",
       "      <th>AFFX-r2-Ec-bioB-5_at</th>\n",
       "      <th>AFFX-r2-Ec-bioB-M_at</th>\n",
       "      <th>AFFX-r2-Ec-bioC-3_at</th>\n",
       "      <th>AFFX-r2-Ec-bioC-5_at</th>\n",
       "      <th>AFFX-r2-Ec-bioD-3_at</th>\n",
       "      <th>AFFX-r2-Ec-bioD-5_at</th>\n",
       "      <th>AFFX-r2-P1-cre-3_at</th>\n",
       "      <th>AFFX-r2-P1-cre-5_at</th>\n",
       "      <th>Diagnosis</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSM880204</td>\n",
       "      <td>9.808114</td>\n",
       "      <td>6.039012</td>\n",
       "      <td>5.810846</td>\n",
       "      <td>10.342781</td>\n",
       "      <td>3.211123</td>\n",
       "      <td>7.615631</td>\n",
       "      <td>5.473264</td>\n",
       "      <td>5.130847</td>\n",
       "      <td>5.967407</td>\n",
       "      <td>...</td>\n",
       "      <td>8.405380</td>\n",
       "      <td>7.976476</td>\n",
       "      <td>7.577744</td>\n",
       "      <td>9.686881</td>\n",
       "      <td>9.287713</td>\n",
       "      <td>11.949707</td>\n",
       "      <td>11.562166</td>\n",
       "      <td>13.446336</td>\n",
       "      <td>13.311750</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSM880205</td>\n",
       "      <td>10.682125</td>\n",
       "      <td>6.608906</td>\n",
       "      <td>5.847419</td>\n",
       "      <td>10.667633</td>\n",
       "      <td>2.924908</td>\n",
       "      <td>8.282238</td>\n",
       "      <td>5.222516</td>\n",
       "      <td>4.620912</td>\n",
       "      <td>8.913122</td>\n",
       "      <td>...</td>\n",
       "      <td>8.385829</td>\n",
       "      <td>8.377783</td>\n",
       "      <td>8.108189</td>\n",
       "      <td>10.007514</td>\n",
       "      <td>9.596678</td>\n",
       "      <td>12.094172</td>\n",
       "      <td>11.836863</td>\n",
       "      <td>13.640625</td>\n",
       "      <td>13.376220</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSM880206</td>\n",
       "      <td>9.659079</td>\n",
       "      <td>6.025781</td>\n",
       "      <td>6.089887</td>\n",
       "      <td>10.385927</td>\n",
       "      <td>3.191762</td>\n",
       "      <td>7.013927</td>\n",
       "      <td>5.483255</td>\n",
       "      <td>5.346077</td>\n",
       "      <td>5.872588</td>\n",
       "      <td>...</td>\n",
       "      <td>8.621434</td>\n",
       "      <td>8.669645</td>\n",
       "      <td>8.406157</td>\n",
       "      <td>10.095641</td>\n",
       "      <td>9.831340</td>\n",
       "      <td>12.238378</td>\n",
       "      <td>11.861172</td>\n",
       "      <td>13.875352</td>\n",
       "      <td>13.715878</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSM880207</td>\n",
       "      <td>10.083274</td>\n",
       "      <td>6.254128</td>\n",
       "      <td>5.386257</td>\n",
       "      <td>10.383023</td>\n",
       "      <td>3.096177</td>\n",
       "      <td>7.217460</td>\n",
       "      <td>5.364841</td>\n",
       "      <td>4.968738</td>\n",
       "      <td>5.622734</td>\n",
       "      <td>...</td>\n",
       "      <td>9.197614</td>\n",
       "      <td>9.086772</td>\n",
       "      <td>8.752838</td>\n",
       "      <td>10.489239</td>\n",
       "      <td>10.235579</td>\n",
       "      <td>12.551390</td>\n",
       "      <td>12.098305</td>\n",
       "      <td>13.951246</td>\n",
       "      <td>13.789232</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSM880208</td>\n",
       "      <td>10.460354</td>\n",
       "      <td>6.304075</td>\n",
       "      <td>5.550745</td>\n",
       "      <td>11.367598</td>\n",
       "      <td>3.059629</td>\n",
       "      <td>7.658014</td>\n",
       "      <td>5.402205</td>\n",
       "      <td>5.469187</td>\n",
       "      <td>4.728435</td>\n",
       "      <td>...</td>\n",
       "      <td>8.391848</td>\n",
       "      <td>8.082121</td>\n",
       "      <td>8.007603</td>\n",
       "      <td>9.887173</td>\n",
       "      <td>9.587661</td>\n",
       "      <td>12.296905</td>\n",
       "      <td>11.817117</td>\n",
       "      <td>13.796431</td>\n",
       "      <td>13.553796</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>GSM880610</td>\n",
       "      <td>9.934819</td>\n",
       "      <td>5.786077</td>\n",
       "      <td>5.567057</td>\n",
       "      <td>10.392874</td>\n",
       "      <td>3.156632</td>\n",
       "      <td>7.294769</td>\n",
       "      <td>5.369574</td>\n",
       "      <td>5.101342</td>\n",
       "      <td>6.439260</td>\n",
       "      <td>...</td>\n",
       "      <td>8.265206</td>\n",
       "      <td>8.087644</td>\n",
       "      <td>7.751108</td>\n",
       "      <td>9.891176</td>\n",
       "      <td>9.537927</td>\n",
       "      <td>11.972551</td>\n",
       "      <td>11.618292</td>\n",
       "      <td>13.642457</td>\n",
       "      <td>13.353197</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>GSM880611</td>\n",
       "      <td>10.331492</td>\n",
       "      <td>6.694437</td>\n",
       "      <td>5.935698</td>\n",
       "      <td>10.768941</td>\n",
       "      <td>3.277184</td>\n",
       "      <td>7.890006</td>\n",
       "      <td>5.652688</td>\n",
       "      <td>5.427916</td>\n",
       "      <td>7.976543</td>\n",
       "      <td>...</td>\n",
       "      <td>8.048401</td>\n",
       "      <td>7.891162</td>\n",
       "      <td>7.706554</td>\n",
       "      <td>9.714787</td>\n",
       "      <td>9.397498</td>\n",
       "      <td>12.002003</td>\n",
       "      <td>11.617847</td>\n",
       "      <td>13.395456</td>\n",
       "      <td>13.256493</td>\n",
       "      <td>Mixed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>GSM880612</td>\n",
       "      <td>10.434603</td>\n",
       "      <td>6.764096</td>\n",
       "      <td>6.189874</td>\n",
       "      <td>10.814140</td>\n",
       "      <td>3.115665</td>\n",
       "      <td>8.154171</td>\n",
       "      <td>5.165332</td>\n",
       "      <td>4.886810</td>\n",
       "      <td>8.652118</td>\n",
       "      <td>...</td>\n",
       "      <td>7.330773</td>\n",
       "      <td>7.173659</td>\n",
       "      <td>7.095531</td>\n",
       "      <td>8.988399</td>\n",
       "      <td>8.574608</td>\n",
       "      <td>11.305505</td>\n",
       "      <td>10.803184</td>\n",
       "      <td>12.971723</td>\n",
       "      <td>12.677655</td>\n",
       "      <td>ABMR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>GSM880613</td>\n",
       "      <td>10.172647</td>\n",
       "      <td>6.226061</td>\n",
       "      <td>5.550959</td>\n",
       "      <td>10.757275</td>\n",
       "      <td>3.154218</td>\n",
       "      <td>7.604247</td>\n",
       "      <td>5.295393</td>\n",
       "      <td>4.996001</td>\n",
       "      <td>6.927812</td>\n",
       "      <td>...</td>\n",
       "      <td>8.184583</td>\n",
       "      <td>7.906788</td>\n",
       "      <td>7.629129</td>\n",
       "      <td>9.619145</td>\n",
       "      <td>9.333553</td>\n",
       "      <td>12.004095</td>\n",
       "      <td>11.629319</td>\n",
       "      <td>13.372509</td>\n",
       "      <td>13.103544</td>\n",
       "      <td>Non-rejecting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>GSM880614</td>\n",
       "      <td>9.717351</td>\n",
       "      <td>6.889039</td>\n",
       "      <td>6.058230</td>\n",
       "      <td>10.178743</td>\n",
       "      <td>3.126419</td>\n",
       "      <td>7.978510</td>\n",
       "      <td>5.521879</td>\n",
       "      <td>5.157721</td>\n",
       "      <td>9.354343</td>\n",
       "      <td>...</td>\n",
       "      <td>7.912465</td>\n",
       "      <td>7.601108</td>\n",
       "      <td>7.632612</td>\n",
       "      <td>9.689859</td>\n",
       "      <td>9.328132</td>\n",
       "      <td>11.980187</td>\n",
       "      <td>11.490682</td>\n",
       "      <td>13.344565</td>\n",
       "      <td>13.074462</td>\n",
       "      <td>TCMR</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>411 rows × 54677 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    Unnamed: 0  DDR1 /// MIR4640      RFC2     HSPA6       PAX8    GUCA1A  \\\n",
       "0    GSM880204          9.808114  6.039012  5.810846  10.342781  3.211123   \n",
       "1    GSM880205         10.682125  6.608906  5.847419  10.667633  2.924908   \n",
       "2    GSM880206          9.659079  6.025781  6.089887  10.385927  3.191762   \n",
       "3    GSM880207         10.083274  6.254128  5.386257  10.383023  3.096177   \n",
       "4    GSM880208         10.460354  6.304075  5.550745  11.367598  3.059629   \n",
       "..         ...               ...       ...       ...        ...       ...   \n",
       "406  GSM880610          9.934819  5.786077  5.567057  10.392874  3.156632   \n",
       "407  GSM880611         10.331492  6.694437  5.935698  10.768941  3.277184   \n",
       "408  GSM880612         10.434603  6.764096  6.189874  10.814140  3.115665   \n",
       "409  GSM880613         10.172647  6.226061  5.550959  10.757275  3.154218   \n",
       "410  GSM880614          9.717351  6.889039  6.058230  10.178743  3.126419   \n",
       "\n",
       "     MIR5193 /// UBA7      THRA    PTPN21      CCL5  ...  \\\n",
       "0            7.615631  5.473264  5.130847  5.967407  ...   \n",
       "1            8.282238  5.222516  4.620912  8.913122  ...   \n",
       "2            7.013927  5.483255  5.346077  5.872588  ...   \n",
       "3            7.217460  5.364841  4.968738  5.622734  ...   \n",
       "4            7.658014  5.402205  5.469187  4.728435  ...   \n",
       "..                ...       ...       ...       ...  ...   \n",
       "406          7.294769  5.369574  5.101342  6.439260  ...   \n",
       "407          7.890006  5.652688  5.427916  7.976543  ...   \n",
       "408          8.154171  5.165332  4.886810  8.652118  ...   \n",
       "409          7.604247  5.295393  4.996001  6.927812  ...   \n",
       "410          7.978510  5.521879  5.157721  9.354343  ...   \n",
       "\n",
       "     AFFX-r2-Ec-bioB-3_at  AFFX-r2-Ec-bioB-5_at  AFFX-r2-Ec-bioB-M_at  \\\n",
       "0                8.405380              7.976476              7.577744   \n",
       "1                8.385829              8.377783              8.108189   \n",
       "2                8.621434              8.669645              8.406157   \n",
       "3                9.197614              9.086772              8.752838   \n",
       "4                8.391848              8.082121              8.007603   \n",
       "..                    ...                   ...                   ...   \n",
       "406              8.265206              8.087644              7.751108   \n",
       "407              8.048401              7.891162              7.706554   \n",
       "408              7.330773              7.173659              7.095531   \n",
       "409              8.184583              7.906788              7.629129   \n",
       "410              7.912465              7.601108              7.632612   \n",
       "\n",
       "     AFFX-r2-Ec-bioC-3_at  AFFX-r2-Ec-bioC-5_at  AFFX-r2-Ec-bioD-3_at  \\\n",
       "0                9.686881              9.287713             11.949707   \n",
       "1               10.007514              9.596678             12.094172   \n",
       "2               10.095641              9.831340             12.238378   \n",
       "3               10.489239             10.235579             12.551390   \n",
       "4                9.887173              9.587661             12.296905   \n",
       "..                    ...                   ...                   ...   \n",
       "406              9.891176              9.537927             11.972551   \n",
       "407              9.714787              9.397498             12.002003   \n",
       "408              8.988399              8.574608             11.305505   \n",
       "409              9.619145              9.333553             12.004095   \n",
       "410              9.689859              9.328132             11.980187   \n",
       "\n",
       "     AFFX-r2-Ec-bioD-5_at  AFFX-r2-P1-cre-3_at  AFFX-r2-P1-cre-5_at  \\\n",
       "0               11.562166            13.446336            13.311750   \n",
       "1               11.836863            13.640625            13.376220   \n",
       "2               11.861172            13.875352            13.715878   \n",
       "3               12.098305            13.951246            13.789232   \n",
       "4               11.817117            13.796431            13.553796   \n",
       "..                    ...                  ...                  ...   \n",
       "406             11.618292            13.642457            13.353197   \n",
       "407             11.617847            13.395456            13.256493   \n",
       "408             10.803184            12.971723            12.677655   \n",
       "409             11.629319            13.372509            13.103544   \n",
       "410             11.490682            13.344565            13.074462   \n",
       "\n",
       "         Diagnosis  \n",
       "0    Non-rejecting  \n",
       "1    Non-rejecting  \n",
       "2    Non-rejecting  \n",
       "3    Non-rejecting  \n",
       "4    Non-rejecting  \n",
       "..             ...  \n",
       "406  Non-rejecting  \n",
       "407          Mixed  \n",
       "408           ABMR  \n",
       "409  Non-rejecting  \n",
       "410           TCMR  \n",
       "\n",
       "[411 rows x 54677 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0      GSM880204\n",
       "1      GSM880205\n",
       "2      GSM880206\n",
       "3      GSM880207\n",
       "4      GSM880208\n",
       "         ...    \n",
       "406    GSM880610\n",
       "407    GSM880611\n",
       "408    GSM880612\n",
       "409    GSM880613\n",
       "410    GSM880614\n",
       "Name: Unnamed: 0, Length: 411, dtype: object"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[\"Unnamed: 0\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GSM880204</th>\n",
       "      <th>GSM880205</th>\n",
       "      <th>GSM880206</th>\n",
       "      <th>GSM880207</th>\n",
       "      <th>GSM880208</th>\n",
       "      <th>GSM880209</th>\n",
       "      <th>GSM880210</th>\n",
       "      <th>GSM880211</th>\n",
       "      <th>GSM880212</th>\n",
       "      <th>GSM880213</th>\n",
       "      <th>...</th>\n",
       "      <th>GSM880605</th>\n",
       "      <th>GSM880606</th>\n",
       "      <th>GSM880607</th>\n",
       "      <th>GSM880608</th>\n",
       "      <th>GSM880609</th>\n",
       "      <th>GSM880610</th>\n",
       "      <th>GSM880611</th>\n",
       "      <th>GSM880612</th>\n",
       "      <th>GSM880613</th>\n",
       "      <th>GSM880614</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Probe</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1007_s_at</th>\n",
       "      <td>9.808114</td>\n",
       "      <td>10.682125</td>\n",
       "      <td>9.659079</td>\n",
       "      <td>10.083274</td>\n",
       "      <td>10.460354</td>\n",
       "      <td>9.908205</td>\n",
       "      <td>9.826789</td>\n",
       "      <td>9.849271</td>\n",
       "      <td>10.167985</td>\n",
       "      <td>9.479723</td>\n",
       "      <td>...</td>\n",
       "      <td>10.045520</td>\n",
       "      <td>10.185048</td>\n",
       "      <td>10.476428</td>\n",
       "      <td>9.915969</td>\n",
       "      <td>10.165729</td>\n",
       "      <td>9.934819</td>\n",
       "      <td>10.331492</td>\n",
       "      <td>10.434603</td>\n",
       "      <td>10.172647</td>\n",
       "      <td>9.717351</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1053_at</th>\n",
       "      <td>6.039012</td>\n",
       "      <td>6.608906</td>\n",
       "      <td>6.025781</td>\n",
       "      <td>6.254128</td>\n",
       "      <td>6.304075</td>\n",
       "      <td>6.371497</td>\n",
       "      <td>6.350107</td>\n",
       "      <td>6.274171</td>\n",
       "      <td>6.061814</td>\n",
       "      <td>6.529935</td>\n",
       "      <td>...</td>\n",
       "      <td>7.250703</td>\n",
       "      <td>6.156028</td>\n",
       "      <td>6.621311</td>\n",
       "      <td>6.652392</td>\n",
       "      <td>6.324376</td>\n",
       "      <td>5.786077</td>\n",
       "      <td>6.694437</td>\n",
       "      <td>6.764096</td>\n",
       "      <td>6.226061</td>\n",
       "      <td>6.889039</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117_at</th>\n",
       "      <td>5.810846</td>\n",
       "      <td>5.847419</td>\n",
       "      <td>6.089887</td>\n",
       "      <td>5.386257</td>\n",
       "      <td>5.550745</td>\n",
       "      <td>5.461051</td>\n",
       "      <td>6.057075</td>\n",
       "      <td>5.499230</td>\n",
       "      <td>5.508480</td>\n",
       "      <td>5.989576</td>\n",
       "      <td>...</td>\n",
       "      <td>5.787078</td>\n",
       "      <td>5.822168</td>\n",
       "      <td>6.196529</td>\n",
       "      <td>5.509276</td>\n",
       "      <td>5.442660</td>\n",
       "      <td>5.567057</td>\n",
       "      <td>5.935698</td>\n",
       "      <td>6.189874</td>\n",
       "      <td>5.550959</td>\n",
       "      <td>6.058230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121_at</th>\n",
       "      <td>10.342781</td>\n",
       "      <td>10.667633</td>\n",
       "      <td>10.385927</td>\n",
       "      <td>10.383023</td>\n",
       "      <td>11.367598</td>\n",
       "      <td>10.529191</td>\n",
       "      <td>10.540292</td>\n",
       "      <td>10.331817</td>\n",
       "      <td>10.715556</td>\n",
       "      <td>9.853745</td>\n",
       "      <td>...</td>\n",
       "      <td>9.845058</td>\n",
       "      <td>10.692428</td>\n",
       "      <td>10.321942</td>\n",
       "      <td>10.380285</td>\n",
       "      <td>10.776776</td>\n",
       "      <td>10.392874</td>\n",
       "      <td>10.768941</td>\n",
       "      <td>10.814140</td>\n",
       "      <td>10.757275</td>\n",
       "      <td>10.178743</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1255_g_at</th>\n",
       "      <td>3.211123</td>\n",
       "      <td>2.924908</td>\n",
       "      <td>3.191762</td>\n",
       "      <td>3.096177</td>\n",
       "      <td>3.059629</td>\n",
       "      <td>3.138675</td>\n",
       "      <td>3.121389</td>\n",
       "      <td>3.021544</td>\n",
       "      <td>3.175132</td>\n",
       "      <td>3.229619</td>\n",
       "      <td>...</td>\n",
       "      <td>3.172923</td>\n",
       "      <td>3.017899</td>\n",
       "      <td>3.414540</td>\n",
       "      <td>3.151017</td>\n",
       "      <td>3.025866</td>\n",
       "      <td>3.156632</td>\n",
       "      <td>3.277184</td>\n",
       "      <td>3.115665</td>\n",
       "      <td>3.154218</td>\n",
       "      <td>3.126419</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AFFX-r2-Ec-bioC-5_at</th>\n",
       "      <td>9.287713</td>\n",
       "      <td>9.596678</td>\n",
       "      <td>9.831340</td>\n",
       "      <td>10.235579</td>\n",
       "      <td>9.587661</td>\n",
       "      <td>9.448879</td>\n",
       "      <td>9.741254</td>\n",
       "      <td>9.655232</td>\n",
       "      <td>9.379062</td>\n",
       "      <td>9.099620</td>\n",
       "      <td>...</td>\n",
       "      <td>9.484491</td>\n",
       "      <td>9.873524</td>\n",
       "      <td>9.488206</td>\n",
       "      <td>9.084344</td>\n",
       "      <td>9.670812</td>\n",
       "      <td>9.537927</td>\n",
       "      <td>9.397498</td>\n",
       "      <td>8.574608</td>\n",
       "      <td>9.333553</td>\n",
       "      <td>9.328132</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AFFX-r2-Ec-bioD-3_at</th>\n",
       "      <td>11.949707</td>\n",
       "      <td>12.094172</td>\n",
       "      <td>12.238378</td>\n",
       "      <td>12.551390</td>\n",
       "      <td>12.296905</td>\n",
       "      <td>11.969689</td>\n",
       "      <td>12.121603</td>\n",
       "      <td>11.954493</td>\n",
       "      <td>11.880848</td>\n",
       "      <td>11.866709</td>\n",
       "      <td>...</td>\n",
       "      <td>11.738551</td>\n",
       "      <td>12.280356</td>\n",
       "      <td>12.076868</td>\n",
       "      <td>11.555104</td>\n",
       "      <td>12.246471</td>\n",
       "      <td>11.972551</td>\n",
       "      <td>12.002003</td>\n",
       "      <td>11.305505</td>\n",
       "      <td>12.004095</td>\n",
       "      <td>11.980187</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AFFX-r2-Ec-bioD-5_at</th>\n",
       "      <td>11.562166</td>\n",
       "      <td>11.836863</td>\n",
       "      <td>11.861172</td>\n",
       "      <td>12.098305</td>\n",
       "      <td>11.817117</td>\n",
       "      <td>11.390042</td>\n",
       "      <td>11.772458</td>\n",
       "      <td>11.458731</td>\n",
       "      <td>11.361053</td>\n",
       "      <td>11.365446</td>\n",
       "      <td>...</td>\n",
       "      <td>11.094698</td>\n",
       "      <td>11.898134</td>\n",
       "      <td>11.591101</td>\n",
       "      <td>11.049741</td>\n",
       "      <td>11.864994</td>\n",
       "      <td>11.618292</td>\n",
       "      <td>11.617847</td>\n",
       "      <td>10.803184</td>\n",
       "      <td>11.629319</td>\n",
       "      <td>11.490682</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AFFX-r2-P1-cre-3_at</th>\n",
       "      <td>13.446336</td>\n",
       "      <td>13.640625</td>\n",
       "      <td>13.875352</td>\n",
       "      <td>13.951246</td>\n",
       "      <td>13.796431</td>\n",
       "      <td>13.421148</td>\n",
       "      <td>13.528259</td>\n",
       "      <td>13.420234</td>\n",
       "      <td>13.255877</td>\n",
       "      <td>13.224878</td>\n",
       "      <td>...</td>\n",
       "      <td>13.252194</td>\n",
       "      <td>13.776512</td>\n",
       "      <td>13.405598</td>\n",
       "      <td>13.223404</td>\n",
       "      <td>13.693976</td>\n",
       "      <td>13.642457</td>\n",
       "      <td>13.395456</td>\n",
       "      <td>12.971723</td>\n",
       "      <td>13.372509</td>\n",
       "      <td>13.344565</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AFFX-r2-P1-cre-5_at</th>\n",
       "      <td>13.311750</td>\n",
       "      <td>13.376220</td>\n",
       "      <td>13.715878</td>\n",
       "      <td>13.789232</td>\n",
       "      <td>13.553796</td>\n",
       "      <td>13.143874</td>\n",
       "      <td>13.343617</td>\n",
       "      <td>13.231067</td>\n",
       "      <td>13.032238</td>\n",
       "      <td>12.782139</td>\n",
       "      <td>...</td>\n",
       "      <td>12.957775</td>\n",
       "      <td>13.522547</td>\n",
       "      <td>13.150198</td>\n",
       "      <td>12.892533</td>\n",
       "      <td>13.536731</td>\n",
       "      <td>13.353197</td>\n",
       "      <td>13.256493</td>\n",
       "      <td>12.677655</td>\n",
       "      <td>13.103544</td>\n",
       "      <td>13.074462</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>54675 rows × 411 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      GSM880204  GSM880205  GSM880206  GSM880207  GSM880208  \\\n",
       "Probe                                                                         \n",
       "1007_s_at              9.808114  10.682125   9.659079  10.083274  10.460354   \n",
       "1053_at                6.039012   6.608906   6.025781   6.254128   6.304075   \n",
       "117_at                 5.810846   5.847419   6.089887   5.386257   5.550745   \n",
       "121_at                10.342781  10.667633  10.385927  10.383023  11.367598   \n",
       "1255_g_at              3.211123   2.924908   3.191762   3.096177   3.059629   \n",
       "...                         ...        ...        ...        ...        ...   \n",
       "AFFX-r2-Ec-bioC-5_at   9.287713   9.596678   9.831340  10.235579   9.587661   \n",
       "AFFX-r2-Ec-bioD-3_at  11.949707  12.094172  12.238378  12.551390  12.296905   \n",
       "AFFX-r2-Ec-bioD-5_at  11.562166  11.836863  11.861172  12.098305  11.817117   \n",
       "AFFX-r2-P1-cre-3_at   13.446336  13.640625  13.875352  13.951246  13.796431   \n",
       "AFFX-r2-P1-cre-5_at   13.311750  13.376220  13.715878  13.789232  13.553796   \n",
       "\n",
       "                      GSM880209  GSM880210  GSM880211  GSM880212  GSM880213  \\\n",
       "Probe                                                                         \n",
       "1007_s_at              9.908205   9.826789   9.849271  10.167985   9.479723   \n",
       "1053_at                6.371497   6.350107   6.274171   6.061814   6.529935   \n",
       "117_at                 5.461051   6.057075   5.499230   5.508480   5.989576   \n",
       "121_at                10.529191  10.540292  10.331817  10.715556   9.853745   \n",
       "1255_g_at              3.138675   3.121389   3.021544   3.175132   3.229619   \n",
       "...                         ...        ...        ...        ...        ...   \n",
       "AFFX-r2-Ec-bioC-5_at   9.448879   9.741254   9.655232   9.379062   9.099620   \n",
       "AFFX-r2-Ec-bioD-3_at  11.969689  12.121603  11.954493  11.880848  11.866709   \n",
       "AFFX-r2-Ec-bioD-5_at  11.390042  11.772458  11.458731  11.361053  11.365446   \n",
       "AFFX-r2-P1-cre-3_at   13.421148  13.528259  13.420234  13.255877  13.224878   \n",
       "AFFX-r2-P1-cre-5_at   13.143874  13.343617  13.231067  13.032238  12.782139   \n",
       "\n",
       "                      ...  GSM880605  GSM880606  GSM880607  GSM880608  \\\n",
       "Probe                 ...                                               \n",
       "1007_s_at             ...  10.045520  10.185048  10.476428   9.915969   \n",
       "1053_at               ...   7.250703   6.156028   6.621311   6.652392   \n",
       "117_at                ...   5.787078   5.822168   6.196529   5.509276   \n",
       "121_at                ...   9.845058  10.692428  10.321942  10.380285   \n",
       "1255_g_at             ...   3.172923   3.017899   3.414540   3.151017   \n",
       "...                   ...        ...        ...        ...        ...   \n",
       "AFFX-r2-Ec-bioC-5_at  ...   9.484491   9.873524   9.488206   9.084344   \n",
       "AFFX-r2-Ec-bioD-3_at  ...  11.738551  12.280356  12.076868  11.555104   \n",
       "AFFX-r2-Ec-bioD-5_at  ...  11.094698  11.898134  11.591101  11.049741   \n",
       "AFFX-r2-P1-cre-3_at   ...  13.252194  13.776512  13.405598  13.223404   \n",
       "AFFX-r2-P1-cre-5_at   ...  12.957775  13.522547  13.150198  12.892533   \n",
       "\n",
       "                      GSM880609  GSM880610  GSM880611  GSM880612  GSM880613  \\\n",
       "Probe                                                                         \n",
       "1007_s_at             10.165729   9.934819  10.331492  10.434603  10.172647   \n",
       "1053_at                6.324376   5.786077   6.694437   6.764096   6.226061   \n",
       "117_at                 5.442660   5.567057   5.935698   6.189874   5.550959   \n",
       "121_at                10.776776  10.392874  10.768941  10.814140  10.757275   \n",
       "1255_g_at              3.025866   3.156632   3.277184   3.115665   3.154218   \n",
       "...                         ...        ...        ...        ...        ...   \n",
       "AFFX-r2-Ec-bioC-5_at   9.670812   9.537927   9.397498   8.574608   9.333553   \n",
       "AFFX-r2-Ec-bioD-3_at  12.246471  11.972551  12.002003  11.305505  12.004095   \n",
       "AFFX-r2-Ec-bioD-5_at  11.864994  11.618292  11.617847  10.803184  11.629319   \n",
       "AFFX-r2-P1-cre-3_at   13.693976  13.642457  13.395456  12.971723  13.372509   \n",
       "AFFX-r2-P1-cre-5_at   13.536731  13.353197  13.256493  12.677655  13.103544   \n",
       "\n",
       "                      GSM880614  \n",
       "Probe                            \n",
       "1007_s_at              9.717351  \n",
       "1053_at                6.889039  \n",
       "117_at                 6.058230  \n",
       "121_at                10.178743  \n",
       "1255_g_at              3.126419  \n",
       "...                         ...  \n",
       "AFFX-r2-Ec-bioC-5_at   9.328132  \n",
       "AFFX-r2-Ec-bioD-3_at  11.980187  \n",
       "AFFX-r2-Ec-bioD-5_at  11.490682  \n",
       "AFFX-r2-P1-cre-3_at   13.344565  \n",
       "AFFX-r2-P1-cre-5_at   13.074462  \n",
       "\n",
       "[54675 rows x 411 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "expression_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "           GSM880204  GSM880205  GSM880206  GSM880207  GSM880208  GSM880209  \\\n",
      "Probe                                                                         \n",
      "1007_s_at   9.808114  10.682125   9.659079  10.083274  10.460354   9.908205   \n",
      "1053_at     6.039012   6.608906   6.025781   6.254128   6.304075   6.371497   \n",
      "117_at      5.810846   5.847419   6.089887   5.386257   5.550745   5.461051   \n",
      "121_at     10.342781  10.667633  10.385927  10.383023  11.367598  10.529191   \n",
      "1255_g_at   3.211123   2.924908   3.191762   3.096177   3.059629   3.138675   \n",
      "\n",
      "           GSM880210  GSM880211  GSM880212  GSM880213  ...  GSM880607  \\\n",
      "Probe                                                  ...              \n",
      "1007_s_at   9.826789   9.849271  10.167985   9.479723  ...  10.476428   \n",
      "1053_at     6.350107   6.274171   6.061814   6.529935  ...   6.621311   \n",
      "117_at      6.057075   5.499230   5.508480   5.989576  ...   6.196529   \n",
      "121_at     10.540292  10.331817  10.715556   9.853745  ...  10.321942   \n",
      "1255_g_at   3.121389   3.021544   3.175132   3.229619  ...   3.414540   \n",
      "\n",
      "           GSM880608  GSM880609  GSM880610  GSM880611  GSM880612  GSM880613  \\\n",
      "Probe                                                                         \n",
      "1007_s_at   9.915969  10.165729   9.934819  10.331492  10.434603  10.172647   \n",
      "1053_at     6.652392   6.324376   5.786077   6.694437   6.764096   6.226061   \n",
      "117_at      5.509276   5.442660   5.567057   5.935698   6.189874   5.550959   \n",
      "121_at     10.380285  10.776776  10.392874  10.768941  10.814140  10.757275   \n",
      "1255_g_at   3.151017   3.025866   3.156632   3.277184   3.115665   3.154218   \n",
      "\n",
      "           GSM880614         ID       Gene Symbol  \n",
      "Probe                                              \n",
      "1007_s_at   9.717351  1007_s_at  DDR1 /// MIR4640  \n",
      "1053_at     6.889039    1053_at              RFC2  \n",
      "117_at      6.058230     117_at             HSPA6  \n",
      "121_at     10.178743     121_at              PAX8  \n",
      "1255_g_at   3.126419  1255_g_at            GUCA1A  \n",
      "\n",
      "[5 rows x 413 columns]\n"
     ]
    }
   ],
   "source": [
    "# Read the platform annotation table and correspond the expression matrix to the gene annotation information\n",
    "if len(gse.gpls) > 0:\n",
    "    gpl_name = list(gse.gpls.keys())[0]  # If there is only one platform, it can be used directly\n",
    "    gpl = gse.gpls[gpl_name]\n",
    "    gpl_table = gpl.table.copy()\n",
    "\n",
    "    relevant_cols = [\"ID\", \"Gene Symbol\"]\n",
    "    gpl_table = gpl_table[[col for col in relevant_cols if col in gpl_table.columns]]\n",
    "\n",
    "    annotated_data = pd.merge(\n",
    "        expression_data.reset_index(),\n",
    "        gpl_table,\n",
    "        left_on=\"Probe\", \n",
    "        right_on=\"ID\",\n",
    "        how=\"left\"\n",
    "    )\n",
    "    annotated_data.set_index(\"Probe\", inplace=True)\n",
    "    print(annotated_data.head())\n",
    "else:\n",
    "    print(\"No platform info found. expression_data is ready without annotation.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: Null values exist in the 'Gene Symbol' column and need to be handled first.\n"
     ]
    }
   ],
   "source": [
    "if annotated_data['Gene Symbol'].isnull().any():\n",
    "    print(\"Warning: Null values exist in the 'Gene Symbol' column and need to be handled first.\")\n",
    "    # 可以选择填充空值，例如使用空字符串，或者特定的填充值\n",
    "    # data['Gene Symbol'].fillna('No Symbol', inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                  GSM880204  GSM880205  GSM880206  GSM880207  GSM880208  \\\n",
      "DDR1 /// MIR4640   9.808114  10.682125   9.659079  10.083274  10.460354   \n",
      "RFC2               6.039012   6.608906   6.025781   6.254128   6.304075   \n",
      "HSPA6              5.810846   5.847419   6.089887   5.386257   5.550745   \n",
      "PAX8              10.342781  10.667633  10.385927  10.383023  11.367598   \n",
      "GUCA1A             3.211123   2.924908   3.191762   3.096177   3.059629   \n",
      "\n",
      "                  GSM880209  GSM880210  GSM880211  GSM880212  GSM880213  ...  \\\n",
      "DDR1 /// MIR4640   9.908205   9.826789   9.849271  10.167985   9.479723  ...   \n",
      "RFC2               6.371497   6.350107   6.274171   6.061814   6.529935  ...   \n",
      "HSPA6              5.461051   6.057075   5.499230   5.508480   5.989576  ...   \n",
      "PAX8              10.529191  10.540292  10.331817  10.715556   9.853745  ...   \n",
      "GUCA1A             3.138675   3.121389   3.021544   3.175132   3.229619  ...   \n",
      "\n",
      "                  GSM880607  GSM880608  GSM880609  GSM880610  GSM880611  \\\n",
      "DDR1 /// MIR4640  10.476428   9.915969  10.165729   9.934819  10.331492   \n",
      "RFC2               6.621311   6.652392   6.324376   5.786077   6.694437   \n",
      "HSPA6              6.196529   5.509276   5.442660   5.567057   5.935698   \n",
      "PAX8              10.321942  10.380285  10.776776  10.392874  10.768941   \n",
      "GUCA1A             3.414540   3.151017   3.025866   3.156632   3.277184   \n",
      "\n",
      "                  GSM880612  GSM880613  GSM880614         ID       Gene Symbol  \n",
      "DDR1 /// MIR4640  10.434603  10.172647   9.717351  1007_s_at  DDR1 /// MIR4640  \n",
      "RFC2               6.764096   6.226061   6.889039    1053_at              RFC2  \n",
      "HSPA6              6.189874   5.550959   6.058230     117_at             HSPA6  \n",
      "PAX8              10.814140  10.757275  10.178743     121_at              PAX8  \n",
      "GUCA1A             3.115665   3.154218   3.126419  1255_g_at            GUCA1A  \n",
      "\n",
      "[5 rows x 413 columns]\n"
     ]
    }
   ],
   "source": [
    "annotated_data['Original_Probe'] = annotated_data.index\n",
    "\n",
    "# 更新行名，只在 'Gene Symbol' 不为空（不是 'NaN'）时使用它\n",
    "annotated_data.index = annotated_data.apply(lambda x: x['Gene Symbol'] if pd.notna(x['Gene Symbol']) else x['Original_Probe'], axis=1)\n",
    "\n",
    "# 现在，你可以选择删除 'Original_Probe' 列\n",
    "annotated_data.drop(columns='Original_Probe', inplace=True)\n",
    "\n",
    "# 打印更新后的 DataFrame 以确认更改\n",
    "print(annotated_data.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sample: GSM880204\n",
      "  title: ['non-rejecting146']\n",
      "  geo_accession: ['GSM880204']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880204/suppl/GSM880204_non-rejecting146.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880205\n",
      "  title: ['non-rejecting63']\n",
      "  geo_accession: ['GSM880205']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880205/suppl/GSM880205_non-rejecting63.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880206\n",
      "  title: ['non-rejecting252']\n",
      "  geo_accession: ['GSM880206']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880206/suppl/GSM880206_non-rejecting252.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880207\n",
      "  title: ['non-rejecting180']\n",
      "  geo_accession: ['GSM880207']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880207/suppl/GSM880207_non-rejecting180.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880208\n",
      "  title: ['non-rejecting181']\n",
      "  geo_accession: ['GSM880208']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880208/suppl/GSM880208_non-rejecting181.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880209\n",
      "  title: ['non-rejecting182']\n",
      "  geo_accession: ['GSM880209']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880209/suppl/GSM880209_non-rejecting182.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880210\n",
      "  title: ['non-rejecting147']\n",
      "  geo_accession: ['GSM880210']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880210/suppl/GSM880210_non-rejecting147.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880211\n",
      "  title: ['non-rejecting183']\n",
      "  geo_accession: ['GSM880211']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880211/suppl/GSM880211_non-rejecting183.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880212\n",
      "  title: ['ABMR18']\n",
      "  geo_accession: ['GSM880212']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880212/suppl/GSM880212_ABMR18.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880213\n",
      "  title: ['non-rejecting1']\n",
      "  geo_accession: ['GSM880213']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880213/suppl/GSM880213_non-rejecting1.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880214\n",
      "  title: ['ABMR19']\n",
      "  geo_accession: ['GSM880214']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880214/suppl/GSM880214_ABMR19.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880215\n",
      "  title: ['TCMR1']\n",
      "  geo_accession: ['GSM880215']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880215/suppl/GSM880215_TCMR1.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880216\n",
      "  title: ['ABMR20']\n",
      "  geo_accession: ['GSM880216']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880216/suppl/GSM880216_ABMR20.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880217\n",
      "  title: ['ABMR21']\n",
      "  geo_accession: ['GSM880217']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880217/suppl/GSM880217_ABMR21.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880218\n",
      "  title: ['ABMR22']\n",
      "  geo_accession: ['GSM880218']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880218/suppl/GSM880218_ABMR22.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880219\n",
      "  title: ['non-rejecting148']\n",
      "  geo_accession: ['GSM880219']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880219/suppl/GSM880219_non-rejecting148.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880220\n",
      "  title: ['non-rejecting2']\n",
      "  geo_accession: ['GSM880220']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880220/suppl/GSM880220_non-rejecting2.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880221\n",
      "  title: ['non-rejecting3']\n",
      "  geo_accession: ['GSM880221']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880221/suppl/GSM880221_non-rejecting3.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880222\n",
      "  title: ['non-rejecting64']\n",
      "  geo_accession: ['GSM880222']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880222/suppl/GSM880222_non-rejecting64.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880223\n",
      "  title: ['non-rejecting184']\n",
      "  geo_accession: ['GSM880223']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880223/suppl/GSM880223_non-rejecting184.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880224\n",
      "  title: ['non-rejecting105']\n",
      "  geo_accession: ['GSM880224']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880224/suppl/GSM880224_non-rejecting105.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880225\n",
      "  title: ['non-rejecting4']\n",
      "  geo_accession: ['GSM880225']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880225/suppl/GSM880225_non-rejecting4.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880226\n",
      "  title: ['non-rejecting185']\n",
      "  geo_accession: ['GSM880226']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880226/suppl/GSM880226_non-rejecting185.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880227\n",
      "  title: ['non-rejecting5']\n",
      "  geo_accession: ['GSM880227']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880227/suppl/GSM880227_non-rejecting5.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880228\n",
      "  title: ['MIXED1']\n",
      "  geo_accession: ['GSM880228']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880228/suppl/GSM880228_MIXED1.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880229\n",
      "  title: ['ABMR23']\n",
      "  geo_accession: ['GSM880229']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880229/suppl/GSM880229_ABMR23.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880230\n",
      "  title: ['MIXED2']\n",
      "  geo_accession: ['GSM880230']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880230/suppl/GSM880230_MIXED2.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880231\n",
      "  title: ['non-rejecting6']\n",
      "  geo_accession: ['GSM880231']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880231/suppl/GSM880231_non-rejecting6.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880232\n",
      "  title: ['non-rejecting65']\n",
      "  geo_accession: ['GSM880232']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880232/suppl/GSM880232_non-rejecting65.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880233\n",
      "  title: ['ABMR1']\n",
      "  geo_accession: ['GSM880233']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880233/suppl/GSM880233_ABMR1.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880234\n",
      "  title: ['TCMR2']\n",
      "  geo_accession: ['GSM880234']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880234/suppl/GSM880234_TCMR2.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880235\n",
      "  title: ['non-rejecting186']\n",
      "  geo_accession: ['GSM880235']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880235/suppl/GSM880235_non-rejecting186.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880236\n",
      "  title: ['ABMR24']\n",
      "  geo_accession: ['GSM880236']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880236/suppl/GSM880236_ABMR24.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880237\n",
      "  title: ['ABMR25']\n",
      "  geo_accession: ['GSM880237']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880237/suppl/GSM880237_ABMR25.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880238\n",
      "  title: ['non-rejecting149']\n",
      "  geo_accession: ['GSM880238']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880238/suppl/GSM880238_non-rejecting149.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880239\n",
      "  title: ['ABMR26']\n",
      "  geo_accession: ['GSM880239']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880239/suppl/GSM880239_ABMR26.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880240\n",
      "  title: ['TCMR3']\n",
      "  geo_accession: ['GSM880240']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880240/suppl/GSM880240_TCMR3.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880241\n",
      "  title: ['non-rejecting150']\n",
      "  geo_accession: ['GSM880241']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880241/suppl/GSM880241_non-rejecting150.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880242\n",
      "  title: ['non-rejecting151']\n",
      "  geo_accession: ['GSM880242']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880242/suppl/GSM880242_non-rejecting151.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880243\n",
      "  title: ['non-rejecting152']\n",
      "  geo_accession: ['GSM880243']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880243/suppl/GSM880243_non-rejecting152.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880244\n",
      "  title: ['MIXED3']\n",
      "  geo_accession: ['GSM880244']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880244/suppl/GSM880244_MIXED3.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880245\n",
      "  title: ['Nephrectomy1']\n",
      "  geo_accession: ['GSM880245']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880245/suppl/GSM880245_Nephrectomy1.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928885']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880246\n",
      "  title: ['Nephrectomy2']\n",
      "  geo_accession: ['GSM880246']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880246/suppl/GSM880246_Nephrectomy2.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928886']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880247\n",
      "  title: ['non-rejecting106']\n",
      "  geo_accession: ['GSM880247']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880247/suppl/GSM880247_non-rejecting106.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880248\n",
      "  title: ['ABMR27']\n",
      "  geo_accession: ['GSM880248']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880248/suppl/GSM880248_ABMR27.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880249\n",
      "  title: ['ABMR28']\n",
      "  geo_accession: ['GSM880249']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880249/suppl/GSM880249_ABMR28.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880250\n",
      "  title: ['MIXED4']\n",
      "  geo_accession: ['GSM880250']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880250/suppl/GSM880250_MIXED4.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880251\n",
      "  title: ['ABMR29']\n",
      "  geo_accession: ['GSM880251']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880251/suppl/GSM880251_ABMR29.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880252\n",
      "  title: ['non-rejecting7']\n",
      "  geo_accession: ['GSM880252']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880252/suppl/GSM880252_non-rejecting7.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880253\n",
      "  title: ['non-rejecting66']\n",
      "  geo_accession: ['GSM880253']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880253/suppl/GSM880253_non-rejecting66.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880254\n",
      "  title: ['non-rejecting8']\n",
      "  geo_accession: ['GSM880254']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880254/suppl/GSM880254_non-rejecting8.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880255\n",
      "  title: ['non-rejecting107']\n",
      "  geo_accession: ['GSM880255']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880255/suppl/GSM880255_non-rejecting107.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880256\n",
      "  title: ['non-rejecting108']\n",
      "  geo_accession: ['GSM880256']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880256/suppl/GSM880256_non-rejecting108.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880257\n",
      "  title: ['non-rejecting109']\n",
      "  geo_accession: ['GSM880257']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880257/suppl/GSM880257_non-rejecting109.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880258\n",
      "  title: ['ABMR2']\n",
      "  geo_accession: ['GSM880258']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880258/suppl/GSM880258_ABMR2.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880259\n",
      "  title: ['ABMR30']\n",
      "  geo_accession: ['GSM880259']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880259/suppl/GSM880259_ABMR30.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880260\n",
      "  title: ['MIXED5']\n",
      "  geo_accession: ['GSM880260']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880260/suppl/GSM880260_MIXED5.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880261\n",
      "  title: ['MIXED6']\n",
      "  geo_accession: ['GSM880261']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880261/suppl/GSM880261_MIXED6.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880262\n",
      "  title: ['non-rejecting110']\n",
      "  geo_accession: ['GSM880262']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880262/suppl/GSM880262_non-rejecting110.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880263\n",
      "  title: ['non-rejecting67']\n",
      "  geo_accession: ['GSM880263']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880263/suppl/GSM880263_non-rejecting67.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880264\n",
      "  title: ['ABMR3']\n",
      "  geo_accession: ['GSM880264']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880264/suppl/GSM880264_ABMR3.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880265\n",
      "  title: ['non-rejecting153']\n",
      "  geo_accession: ['GSM880265']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880265/suppl/GSM880265_non-rejecting153.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880266\n",
      "  title: ['non-rejecting187']\n",
      "  geo_accession: ['GSM880266']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880266/suppl/GSM880266_non-rejecting187.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880267\n",
      "  title: ['non-rejecting154']\n",
      "  geo_accession: ['GSM880267']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880267/suppl/GSM880267_non-rejecting154.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880268\n",
      "  title: ['non-rejecting111']\n",
      "  geo_accession: ['GSM880268']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880268/suppl/GSM880268_non-rejecting111.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880269\n",
      "  title: ['non-rejecting253']\n",
      "  geo_accession: ['GSM880269']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880269/suppl/GSM880269_non-rejecting253.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880270\n",
      "  title: ['ABMR31']\n",
      "  geo_accession: ['GSM880270']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880270/suppl/GSM880270_ABMR31.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880271\n",
      "  title: ['MIXED7']\n",
      "  geo_accession: ['GSM880271']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880271/suppl/GSM880271_MIXED7.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880272\n",
      "  title: ['non-rejecting272']\n",
      "  geo_accession: ['GSM880272']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880272/suppl/GSM880272_non-rejecting272.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880273\n",
      "  title: ['MIXED8']\n",
      "  geo_accession: ['GSM880273']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880273/suppl/GSM880273_MIXED8.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880274\n",
      "  title: ['non-rejecting188']\n",
      "  geo_accession: ['GSM880274']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880274/suppl/GSM880274_non-rejecting188.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880275\n",
      "  title: ['ABMR4']\n",
      "  geo_accession: ['GSM880275']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880275/suppl/GSM880275_ABMR4.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880276\n",
      "  title: ['ABMR32']\n",
      "  geo_accession: ['GSM880276']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880276/suppl/GSM880276_ABMR32.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880277\n",
      "  title: ['non-rejecting155']\n",
      "  geo_accession: ['GSM880277']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880277/suppl/GSM880277_non-rejecting155.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880278\n",
      "  title: ['non-rejecting9']\n",
      "  geo_accession: ['GSM880278']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880278/suppl/GSM880278_non-rejecting9.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880279\n",
      "  title: ['non-rejecting10']\n",
      "  geo_accession: ['GSM880279']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880279/suppl/GSM880279_non-rejecting10.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880280\n",
      "  title: ['TCMR4']\n",
      "  geo_accession: ['GSM880280']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880280/suppl/GSM880280_TCMR4.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880281\n",
      "  title: ['non-rejecting273']\n",
      "  geo_accession: ['GSM880281']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880281/suppl/GSM880281_non-rejecting273.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880282\n",
      "  title: ['non-rejecting112']\n",
      "  geo_accession: ['GSM880282']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880282/suppl/GSM880282_non-rejecting112.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880283\n",
      "  title: ['non-rejecting11']\n",
      "  geo_accession: ['GSM880283']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880283/suppl/GSM880283_non-rejecting11.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880284\n",
      "  title: ['non-rejecting189']\n",
      "  geo_accession: ['GSM880284']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880284/suppl/GSM880284_non-rejecting189.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880285\n",
      "  title: ['non-rejecting190']\n",
      "  geo_accession: ['GSM880285']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880285/suppl/GSM880285_non-rejecting190.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880286\n",
      "  title: ['ABMR33']\n",
      "  geo_accession: ['GSM880286']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880286/suppl/GSM880286_ABMR33.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880287\n",
      "  title: ['ABMR34']\n",
      "  geo_accession: ['GSM880287']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880287/suppl/GSM880287_ABMR34.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880288\n",
      "  title: ['ABMR35']\n",
      "  geo_accession: ['GSM880288']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880288/suppl/GSM880288_ABMR35.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880289\n",
      "  title: ['ABMR36']\n",
      "  geo_accession: ['GSM880289']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880289/suppl/GSM880289_ABMR36.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880290\n",
      "  title: ['non-rejecting191']\n",
      "  geo_accession: ['GSM880290']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880290/suppl/GSM880290_non-rejecting191.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880291\n",
      "  title: ['ABMR37']\n",
      "  geo_accession: ['GSM880291']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880291/suppl/GSM880291_ABMR37.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880292\n",
      "  title: ['non-rejecting156']\n",
      "  geo_accession: ['GSM880292']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880292/suppl/GSM880292_non-rejecting156.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880293\n",
      "  title: ['non-rejecting68']\n",
      "  geo_accession: ['GSM880293']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880293/suppl/GSM880293_non-rejecting68.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880294\n",
      "  title: ['ABMR38']\n",
      "  geo_accession: ['GSM880294']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880294/suppl/GSM880294_ABMR38.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880295\n",
      "  title: ['non-rejecting113']\n",
      "  geo_accession: ['GSM880295']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880295/suppl/GSM880295_non-rejecting113.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880296\n",
      "  title: ['non-rejecting114']\n",
      "  geo_accession: ['GSM880296']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880296/suppl/GSM880296_non-rejecting114.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880297\n",
      "  title: ['ABMR39']\n",
      "  geo_accession: ['GSM880297']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880297/suppl/GSM880297_ABMR39.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880298\n",
      "  title: ['non-rejecting69']\n",
      "  geo_accession: ['GSM880298']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880298/suppl/GSM880298_non-rejecting69.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880299\n",
      "  title: ['Nephrectomy3']\n",
      "  geo_accession: ['GSM880299']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880299/suppl/GSM880299_Nephrectomy3.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928887']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880300\n",
      "  title: ['non-rejecting192']\n",
      "  geo_accession: ['GSM880300']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880300/suppl/GSM880300_non-rejecting192.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880301\n",
      "  title: ['non-rejecting274']\n",
      "  geo_accession: ['GSM880301']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880301/suppl/GSM880301_non-rejecting274.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880302\n",
      "  title: ['MIXED9']\n",
      "  geo_accession: ['GSM880302']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880302/suppl/GSM880302_MIXED9.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880303\n",
      "  title: ['non-rejecting157']\n",
      "  geo_accession: ['GSM880303']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880303/suppl/GSM880303_non-rejecting157.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880304\n",
      "  title: ['ABMR40']\n",
      "  geo_accession: ['GSM880304']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880304/suppl/GSM880304_ABMR40.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880305\n",
      "  title: ['ABMR41']\n",
      "  geo_accession: ['GSM880305']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880305/suppl/GSM880305_ABMR41.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880306\n",
      "  title: ['ABMR42']\n",
      "  geo_accession: ['GSM880306']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880306/suppl/GSM880306_ABMR42.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880307\n",
      "  title: ['TCMR5']\n",
      "  geo_accession: ['GSM880307']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880307/suppl/GSM880307_TCMR5.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880308\n",
      "  title: ['non-rejecting70']\n",
      "  geo_accession: ['GSM880308']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880308/suppl/GSM880308_non-rejecting70.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880309\n",
      "  title: ['TCMR6']\n",
      "  geo_accession: ['GSM880309']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880309/suppl/GSM880309_TCMR6.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880310\n",
      "  title: ['non-rejecting193']\n",
      "  geo_accession: ['GSM880310']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880310/suppl/GSM880310_non-rejecting193.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880311\n",
      "  title: ['Nephrectomy4']\n",
      "  geo_accession: ['GSM880311']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880311/suppl/GSM880311_Nephrectomy4.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928888']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880312\n",
      "  title: ['non-rejecting12']\n",
      "  geo_accession: ['GSM880312']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880312/suppl/GSM880312_non-rejecting12.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880313\n",
      "  title: ['ABMR43']\n",
      "  geo_accession: ['GSM880313']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880313/suppl/GSM880313_ABMR43.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880314\n",
      "  title: ['MIXED10']\n",
      "  geo_accession: ['GSM880314']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880314/suppl/GSM880314_MIXED10.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880315\n",
      "  title: ['non-rejecting71']\n",
      "  geo_accession: ['GSM880315']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880315/suppl/GSM880315_non-rejecting71.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880316\n",
      "  title: ['non-rejecting72']\n",
      "  geo_accession: ['GSM880316']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880316/suppl/GSM880316_non-rejecting72.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880317\n",
      "  title: ['ABMR44']\n",
      "  geo_accession: ['GSM880317']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880317/suppl/GSM880317_ABMR44.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880318\n",
      "  title: ['Nephrectomy5']\n",
      "  geo_accession: ['GSM880318']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880318/suppl/GSM880318_Nephrectomy5.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928889']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880319\n",
      "  title: ['non-rejecting13']\n",
      "  geo_accession: ['GSM880319']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880319/suppl/GSM880319_non-rejecting13.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880320\n",
      "  title: ['non-rejecting51']\n",
      "  geo_accession: ['GSM880320']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880320/suppl/GSM880320_non-rejecting51.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880321\n",
      "  title: ['non-rejecting52']\n",
      "  geo_accession: ['GSM880321']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880321/suppl/GSM880321_non-rejecting52.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880322\n",
      "  title: ['non-rejecting14']\n",
      "  geo_accession: ['GSM880322']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880322/suppl/GSM880322_non-rejecting14.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880323\n",
      "  title: ['non-rejecting194']\n",
      "  geo_accession: ['GSM880323']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880323/suppl/GSM880323_non-rejecting194.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880324\n",
      "  title: ['non-rejecting158']\n",
      "  geo_accession: ['GSM880324']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880324/suppl/GSM880324_non-rejecting158.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880325\n",
      "  title: ['non-rejecting73']\n",
      "  geo_accession: ['GSM880325']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880325/suppl/GSM880325_non-rejecting73.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880326\n",
      "  title: ['non-rejecting254']\n",
      "  geo_accession: ['GSM880326']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880326/suppl/GSM880326_non-rejecting254.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880327\n",
      "  title: ['non-rejecting74']\n",
      "  geo_accession: ['GSM880327']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880327/suppl/GSM880327_non-rejecting74.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880328\n",
      "  title: ['non-rejecting195']\n",
      "  geo_accession: ['GSM880328']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880328/suppl/GSM880328_non-rejecting195.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880329\n",
      "  title: ['non-rejecting75']\n",
      "  geo_accession: ['GSM880329']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880329/suppl/GSM880329_non-rejecting75.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880330\n",
      "  title: ['non-rejecting196']\n",
      "  geo_accession: ['GSM880330']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880330/suppl/GSM880330_non-rejecting196.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880331\n",
      "  title: ['TCMR7']\n",
      "  geo_accession: ['GSM880331']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880331/suppl/GSM880331_TCMR7.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880332\n",
      "  title: ['non-rejecting159']\n",
      "  geo_accession: ['GSM880332']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880332/suppl/GSM880332_non-rejecting159.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880333\n",
      "  title: ['ABMR45']\n",
      "  geo_accession: ['GSM880333']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880333/suppl/GSM880333_ABMR45.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880334\n",
      "  title: ['ABMR46']\n",
      "  geo_accession: ['GSM880334']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880334/suppl/GSM880334_ABMR46.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880335\n",
      "  title: ['ABMR47']\n",
      "  geo_accession: ['GSM880335']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880335/suppl/GSM880335_ABMR47.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880336\n",
      "  title: ['non-rejecting197']\n",
      "  geo_accession: ['GSM880336']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880336/suppl/GSM880336_non-rejecting197.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880337\n",
      "  title: ['non-rejecting198']\n",
      "  geo_accession: ['GSM880337']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880337/suppl/GSM880337_non-rejecting198.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880338\n",
      "  title: ['non-rejecting160']\n",
      "  geo_accession: ['GSM880338']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880338/suppl/GSM880338_non-rejecting160.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880339\n",
      "  title: ['non-rejecting255']\n",
      "  geo_accession: ['GSM880339']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880339/suppl/GSM880339_non-rejecting255.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880340\n",
      "  title: ['non-rejecting199']\n",
      "  geo_accession: ['GSM880340']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880340/suppl/GSM880340_non-rejecting199.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880341\n",
      "  title: ['TCMR8']\n",
      "  geo_accession: ['GSM880341']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880341/suppl/GSM880341_TCMR8.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880342\n",
      "  title: ['non-rejecting161']\n",
      "  geo_accession: ['GSM880342']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880342/suppl/GSM880342_non-rejecting161.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880343\n",
      "  title: ['non-rejecting15']\n",
      "  geo_accession: ['GSM880343']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880343/suppl/GSM880343_non-rejecting15.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880344\n",
      "  title: ['non-rejecting200']\n",
      "  geo_accession: ['GSM880344']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880344/suppl/GSM880344_non-rejecting200.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880345\n",
      "  title: ['TCMR9']\n",
      "  geo_accession: ['GSM880345']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880345/suppl/GSM880345_TCMR9.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880346\n",
      "  title: ['TCMR10']\n",
      "  geo_accession: ['GSM880346']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880346/suppl/GSM880346_TCMR10.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880347\n",
      "  title: ['non-rejecting201']\n",
      "  geo_accession: ['GSM880347']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880347/suppl/GSM880347_non-rejecting201.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880348\n",
      "  title: ['non-rejecting76']\n",
      "  geo_accession: ['GSM880348']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880348/suppl/GSM880348_non-rejecting76.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880349\n",
      "  title: ['non-rejecting115']\n",
      "  geo_accession: ['GSM880349']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880349/suppl/GSM880349_non-rejecting115.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880350\n",
      "  title: ['non-rejecting16']\n",
      "  geo_accession: ['GSM880350']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880350/suppl/GSM880350_non-rejecting16.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880351\n",
      "  title: ['non-rejecting116']\n",
      "  geo_accession: ['GSM880351']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880351/suppl/GSM880351_non-rejecting116.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880352\n",
      "  title: ['MIXED11']\n",
      "  geo_accession: ['GSM880352']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880352/suppl/GSM880352_MIXED11.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880353\n",
      "  title: ['ABMR48']\n",
      "  geo_accession: ['GSM880353']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880353/suppl/GSM880353_ABMR48.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880354\n",
      "  title: ['non-rejecting77']\n",
      "  geo_accession: ['GSM880354']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880354/suppl/GSM880354_non-rejecting77.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880355\n",
      "  title: ['non-rejecting117']\n",
      "  geo_accession: ['GSM880355']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880355/suppl/GSM880355_non-rejecting117.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880356\n",
      "  title: ['non-rejecting162']\n",
      "  geo_accession: ['GSM880356']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880356/suppl/GSM880356_non-rejecting162.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880357\n",
      "  title: ['non-rejecting118']\n",
      "  geo_accession: ['GSM880357']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880357/suppl/GSM880357_non-rejecting118.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880358\n",
      "  title: ['non-rejecting119']\n",
      "  geo_accession: ['GSM880358']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880358/suppl/GSM880358_non-rejecting119.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880359\n",
      "  title: ['non-rejecting256']\n",
      "  geo_accession: ['GSM880359']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880359/suppl/GSM880359_non-rejecting256.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880360\n",
      "  title: ['non-rejecting257']\n",
      "  geo_accession: ['GSM880360']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880360/suppl/GSM880360_non-rejecting257.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880361\n",
      "  title: ['non-rejecting17']\n",
      "  geo_accession: ['GSM880361']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880361/suppl/GSM880361_non-rejecting17.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880362\n",
      "  title: ['non-rejecting258']\n",
      "  geo_accession: ['GSM880362']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880362/suppl/GSM880362_non-rejecting258.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880363\n",
      "  title: ['non-rejecting202']\n",
      "  geo_accession: ['GSM880363']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880363/suppl/GSM880363_non-rejecting202.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880364\n",
      "  title: ['non-rejecting120']\n",
      "  geo_accession: ['GSM880364']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880364/suppl/GSM880364_non-rejecting120.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880365\n",
      "  title: ['non-rejecting18']\n",
      "  geo_accession: ['GSM880365']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880365/suppl/GSM880365_non-rejecting18.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880366\n",
      "  title: ['non-rejecting259']\n",
      "  geo_accession: ['GSM880366']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880366/suppl/GSM880366_non-rejecting259.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880367\n",
      "  title: ['non-rejecting260']\n",
      "  geo_accession: ['GSM880367']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880367/suppl/GSM880367_non-rejecting260.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880368\n",
      "  title: ['non-rejecting19']\n",
      "  geo_accession: ['GSM880368']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880368/suppl/GSM880368_non-rejecting19.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880369\n",
      "  title: ['non-rejecting203']\n",
      "  geo_accession: ['GSM880369']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880369/suppl/GSM880369_non-rejecting203.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880370\n",
      "  title: ['ABMR5']\n",
      "  geo_accession: ['GSM880370']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880370/suppl/GSM880370_ABMR5.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880371\n",
      "  title: ['non-rejecting20']\n",
      "  geo_accession: ['GSM880371']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880371/suppl/GSM880371_non-rejecting20.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880372\n",
      "  title: ['TCMR11']\n",
      "  geo_accession: ['GSM880372']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880372/suppl/GSM880372_TCMR11.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880373\n",
      "  title: ['non-rejecting21']\n",
      "  geo_accession: ['GSM880373']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880373/suppl/GSM880373_non-rejecting21.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880374\n",
      "  title: ['ABMR6']\n",
      "  geo_accession: ['GSM880374']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880374/suppl/GSM880374_ABMR6.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880375\n",
      "  title: ['non-rejecting22']\n",
      "  geo_accession: ['GSM880375']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880375/suppl/GSM880375_non-rejecting22.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880376\n",
      "  title: ['non-rejecting204']\n",
      "  geo_accession: ['GSM880376']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880376/suppl/GSM880376_non-rejecting204.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880377\n",
      "  title: ['non-rejecting23']\n",
      "  geo_accession: ['GSM880377']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880377/suppl/GSM880377_non-rejecting23.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880378\n",
      "  title: ['non-rejecting261']\n",
      "  geo_accession: ['GSM880378']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880378/suppl/GSM880378_non-rejecting261.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880379\n",
      "  title: ['non-rejecting205']\n",
      "  geo_accession: ['GSM880379']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880379/suppl/GSM880379_non-rejecting205.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880380\n",
      "  title: ['non-rejecting53']\n",
      "  geo_accession: ['GSM880380']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880380/suppl/GSM880380_non-rejecting53.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880381\n",
      "  title: ['non-rejecting24']\n",
      "  geo_accession: ['GSM880381']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880381/suppl/GSM880381_non-rejecting24.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880382\n",
      "  title: ['non-rejecting25']\n",
      "  geo_accession: ['GSM880382']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880382/suppl/GSM880382_non-rejecting25.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880383\n",
      "  title: ['non-rejecting26']\n",
      "  geo_accession: ['GSM880383']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880383/suppl/GSM880383_non-rejecting26.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880384\n",
      "  title: ['non-rejecting262']\n",
      "  geo_accession: ['GSM880384']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880384/suppl/GSM880384_non-rejecting262.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880385\n",
      "  title: ['non-rejecting206']\n",
      "  geo_accession: ['GSM880385']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880385/suppl/GSM880385_non-rejecting206.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880386\n",
      "  title: ['non-rejecting121']\n",
      "  geo_accession: ['GSM880386']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880386/suppl/GSM880386_non-rejecting121.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880387\n",
      "  title: ['non-rejecting207']\n",
      "  geo_accession: ['GSM880387']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880387/suppl/GSM880387_non-rejecting207.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880388\n",
      "  title: ['TCMR12']\n",
      "  geo_accession: ['GSM880388']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880388/suppl/GSM880388_TCMR12.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880389\n",
      "  title: ['non-rejecting78']\n",
      "  geo_accession: ['GSM880389']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880389/suppl/GSM880389_non-rejecting78.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880390\n",
      "  title: ['non-rejecting208']\n",
      "  geo_accession: ['GSM880390']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880390/suppl/GSM880390_non-rejecting208.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880391\n",
      "  title: ['MIXED12']\n",
      "  geo_accession: ['GSM880391']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880391/suppl/GSM880391_MIXED12.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880392\n",
      "  title: ['non-rejecting209']\n",
      "  geo_accession: ['GSM880392']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880392/suppl/GSM880392_non-rejecting209.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880393\n",
      "  title: ['non-rejecting27']\n",
      "  geo_accession: ['GSM880393']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880393/suppl/GSM880393_non-rejecting27.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880394\n",
      "  title: ['TCMR13']\n",
      "  geo_accession: ['GSM880394']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880394/suppl/GSM880394_TCMR13.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880395\n",
      "  title: ['MIXED13']\n",
      "  geo_accession: ['GSM880395']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880395/suppl/GSM880395_MIXED13.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880396\n",
      "  title: ['non-rejecting122']\n",
      "  geo_accession: ['GSM880396']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880396/suppl/GSM880396_non-rejecting122.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880397\n",
      "  title: ['ABMR49']\n",
      "  geo_accession: ['GSM880397']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880397/suppl/GSM880397_ABMR49.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880398\n",
      "  title: ['non-rejecting123']\n",
      "  geo_accession: ['GSM880398']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880398/suppl/GSM880398_non-rejecting123.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880399\n",
      "  title: ['TCMR14']\n",
      "  geo_accession: ['GSM880399']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880399/suppl/GSM880399_TCMR14.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880400\n",
      "  title: ['non-rejecting124']\n",
      "  geo_accession: ['GSM880400']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880400/suppl/GSM880400_non-rejecting124.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880401\n",
      "  title: ['non-rejecting79']\n",
      "  geo_accession: ['GSM880401']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880401/suppl/GSM880401_non-rejecting79.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880402\n",
      "  title: ['non-rejecting275']\n",
      "  geo_accession: ['GSM880402']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880402/suppl/GSM880402_non-rejecting275.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880403\n",
      "  title: ['non-rejecting210']\n",
      "  geo_accession: ['GSM880403']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880403/suppl/GSM880403_non-rejecting210.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880404\n",
      "  title: ['non-rejecting28']\n",
      "  geo_accession: ['GSM880404']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880404/suppl/GSM880404_non-rejecting28.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880405\n",
      "  title: ['ABMR50']\n",
      "  geo_accession: ['GSM880405']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880405/suppl/GSM880405_ABMR50.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880406\n",
      "  title: ['non-rejecting276']\n",
      "  geo_accession: ['GSM880406']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880406/suppl/GSM880406_non-rejecting276.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880407\n",
      "  title: ['MIXED14']\n",
      "  geo_accession: ['GSM880407']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880407/suppl/GSM880407_MIXED14.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880408\n",
      "  title: ['ABMR7']\n",
      "  geo_accession: ['GSM880408']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880408/suppl/GSM880408_ABMR7.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880409\n",
      "  title: ['non-rejecting277']\n",
      "  geo_accession: ['GSM880409']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880409/suppl/GSM880409_non-rejecting277.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880410\n",
      "  title: ['non-rejecting278']\n",
      "  geo_accession: ['GSM880410']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880410/suppl/GSM880410_non-rejecting278.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880411\n",
      "  title: ['TCMR15']\n",
      "  geo_accession: ['GSM880411']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880411/suppl/GSM880411_TCMR15.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880412\n",
      "  title: ['non-rejecting211']\n",
      "  geo_accession: ['GSM880412']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880412/suppl/GSM880412_non-rejecting211.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880413\n",
      "  title: ['non-rejecting125']\n",
      "  geo_accession: ['GSM880413']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880413/suppl/GSM880413_non-rejecting125.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880414\n",
      "  title: ['non-rejecting212']\n",
      "  geo_accession: ['GSM880414']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880414/suppl/GSM880414_non-rejecting212.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880415\n",
      "  title: ['non-rejecting126']\n",
      "  geo_accession: ['GSM880415']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880415/suppl/GSM880415_non-rejecting126.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880416\n",
      "  title: ['TCMR16']\n",
      "  geo_accession: ['GSM880416']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880416/suppl/GSM880416_TCMR16.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880417\n",
      "  title: ['non-rejecting163']\n",
      "  geo_accession: ['GSM880417']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880417/suppl/GSM880417_non-rejecting163.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880418\n",
      "  title: ['non-rejecting127']\n",
      "  geo_accession: ['GSM880418']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880418/suppl/GSM880418_non-rejecting127.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880419\n",
      "  title: ['non-rejecting80']\n",
      "  geo_accession: ['GSM880419']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880419/suppl/GSM880419_non-rejecting80.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880420\n",
      "  title: ['ABMR51']\n",
      "  geo_accession: ['GSM880420']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880420/suppl/GSM880420_ABMR51.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880421\n",
      "  title: ['non-rejecting81']\n",
      "  geo_accession: ['GSM880421']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880421/suppl/GSM880421_non-rejecting81.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880422\n",
      "  title: ['non-rejecting29']\n",
      "  geo_accession: ['GSM880422']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880422/suppl/GSM880422_non-rejecting29.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880423\n",
      "  title: ['non-rejecting82']\n",
      "  geo_accession: ['GSM880423']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880423/suppl/GSM880423_non-rejecting82.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880424\n",
      "  title: ['ABMR52']\n",
      "  geo_accession: ['GSM880424']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880424/suppl/GSM880424_ABMR52.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880425\n",
      "  title: ['non-rejecting30']\n",
      "  geo_accession: ['GSM880425']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880425/suppl/GSM880425_non-rejecting30.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880426\n",
      "  title: ['TCMR17']\n",
      "  geo_accession: ['GSM880426']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880426/suppl/GSM880426_TCMR17.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880427\n",
      "  title: ['non-rejecting213']\n",
      "  geo_accession: ['GSM880427']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880427/suppl/GSM880427_non-rejecting213.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880428\n",
      "  title: ['ABMR53']\n",
      "  geo_accession: ['GSM880428']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880428/suppl/GSM880428_ABMR53.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880429\n",
      "  title: ['ABMR54']\n",
      "  geo_accession: ['GSM880429']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880429/suppl/GSM880429_ABMR54.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880430\n",
      "  title: ['Nephrectomy6']\n",
      "  geo_accession: ['GSM880430']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880430/suppl/GSM880430_Nephrectomy6.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928890']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880431\n",
      "  title: ['non-rejecting54']\n",
      "  geo_accession: ['GSM880431']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880431/suppl/GSM880431_non-rejecting54.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880432\n",
      "  title: ['non-rejecting214']\n",
      "  geo_accession: ['GSM880432']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880432/suppl/GSM880432_non-rejecting214.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880433\n",
      "  title: ['non-rejecting215']\n",
      "  geo_accession: ['GSM880433']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880433/suppl/GSM880433_non-rejecting215.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880434\n",
      "  title: ['TCMR18']\n",
      "  geo_accession: ['GSM880434']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880434/suppl/GSM880434_TCMR18.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880435\n",
      "  title: ['non-rejecting216']\n",
      "  geo_accession: ['GSM880435']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880435/suppl/GSM880435_non-rejecting216.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880436\n",
      "  title: ['ABMR8']\n",
      "  geo_accession: ['GSM880436']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880436/suppl/GSM880436_ABMR8.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880437\n",
      "  title: ['MIXED15']\n",
      "  geo_accession: ['GSM880437']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880437/suppl/GSM880437_MIXED15.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880438\n",
      "  title: ['non-rejecting128']\n",
      "  geo_accession: ['GSM880438']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880438/suppl/GSM880438_non-rejecting128.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880439\n",
      "  title: ['non-rejecting129']\n",
      "  geo_accession: ['GSM880439']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880439/suppl/GSM880439_non-rejecting129.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880440\n",
      "  title: ['TCMR19']\n",
      "  geo_accession: ['GSM880440']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880440/suppl/GSM880440_TCMR19.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880441\n",
      "  title: ['non-rejecting130']\n",
      "  geo_accession: ['GSM880441']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880441/suppl/GSM880441_non-rejecting130.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880442\n",
      "  title: ['ABMR55']\n",
      "  geo_accession: ['GSM880442']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880442/suppl/GSM880442_ABMR55.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880443\n",
      "  title: ['non-rejecting131']\n",
      "  geo_accession: ['GSM880443']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880443/suppl/GSM880443_non-rejecting131.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880444\n",
      "  title: ['non-rejecting132']\n",
      "  geo_accession: ['GSM880444']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880444/suppl/GSM880444_non-rejecting132.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880445\n",
      "  title: ['TCMR20']\n",
      "  geo_accession: ['GSM880445']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880445/suppl/GSM880445_TCMR20.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880446\n",
      "  title: ['non-rejecting83']\n",
      "  geo_accession: ['GSM880446']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880446/suppl/GSM880446_non-rejecting83.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880447\n",
      "  title: ['Nephrectomy7']\n",
      "  geo_accession: ['GSM880447']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880447/suppl/GSM880447_Nephrectomy7.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928891']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880448\n",
      "  title: ['non-rejecting84']\n",
      "  geo_accession: ['GSM880448']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880448/suppl/GSM880448_non-rejecting84.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880449\n",
      "  title: ['ABMR56']\n",
      "  geo_accession: ['GSM880449']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880449/suppl/GSM880449_ABMR56.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880450\n",
      "  title: ['non-rejecting164']\n",
      "  geo_accession: ['GSM880450']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880450/suppl/GSM880450_non-rejecting164.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880451\n",
      "  title: ['non-rejecting279']\n",
      "  geo_accession: ['GSM880451']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880451/suppl/GSM880451_non-rejecting279.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880452\n",
      "  title: ['non-rejecting133']\n",
      "  geo_accession: ['GSM880452']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880452/suppl/GSM880452_non-rejecting133.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880453\n",
      "  title: ['ABMR57']\n",
      "  geo_accession: ['GSM880453']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880453/suppl/GSM880453_ABMR57.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880454\n",
      "  title: ['non-rejecting31']\n",
      "  geo_accession: ['GSM880454']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880454/suppl/GSM880454_non-rejecting31.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880455\n",
      "  title: ['non-rejecting263']\n",
      "  geo_accession: ['GSM880455']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880455/suppl/GSM880455_non-rejecting263.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880456\n",
      "  title: ['TCMR21']\n",
      "  geo_accession: ['GSM880456']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880456/suppl/GSM880456_TCMR21.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880457\n",
      "  title: ['non-rejecting134']\n",
      "  geo_accession: ['GSM880457']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880457/suppl/GSM880457_non-rejecting134.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880458\n",
      "  title: ['non-rejecting165']\n",
      "  geo_accession: ['GSM880458']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880458/suppl/GSM880458_non-rejecting165.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880459\n",
      "  title: ['MIXED16']\n",
      "  geo_accession: ['GSM880459']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880459/suppl/GSM880459_MIXED16.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880460\n",
      "  title: ['non-rejecting135']\n",
      "  geo_accession: ['GSM880460']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880460/suppl/GSM880460_non-rejecting135.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880461\n",
      "  title: ['non-rejecting32']\n",
      "  geo_accession: ['GSM880461']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880461/suppl/GSM880461_non-rejecting32.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880462\n",
      "  title: ['non-rejecting33']\n",
      "  geo_accession: ['GSM880462']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880462/suppl/GSM880462_non-rejecting33.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880463\n",
      "  title: ['non-rejecting85']\n",
      "  geo_accession: ['GSM880463']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880463/suppl/GSM880463_non-rejecting85.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880464\n",
      "  title: ['TCMR22']\n",
      "  geo_accession: ['GSM880464']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880464/suppl/GSM880464_TCMR22.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880465\n",
      "  title: ['TCMR23']\n",
      "  geo_accession: ['GSM880465']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880465/suppl/GSM880465_TCMR23.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880466\n",
      "  title: ['ABMR9']\n",
      "  geo_accession: ['GSM880466']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880466/suppl/GSM880466_ABMR9.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880467\n",
      "  title: ['non-rejecting280']\n",
      "  geo_accession: ['GSM880467']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880467/suppl/GSM880467_non-rejecting280.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880468\n",
      "  title: ['non-rejecting217']\n",
      "  geo_accession: ['GSM880468']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880468/suppl/GSM880468_non-rejecting217.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880469\n",
      "  title: ['non-rejecting86']\n",
      "  geo_accession: ['GSM880469']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880469/suppl/GSM880469_non-rejecting86.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880470\n",
      "  title: ['non-rejecting87']\n",
      "  geo_accession: ['GSM880470']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880470/suppl/GSM880470_non-rejecting87.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880471\n",
      "  title: ['non-rejecting218']\n",
      "  geo_accession: ['GSM880471']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880471/suppl/GSM880471_non-rejecting218.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880472\n",
      "  title: ['non-rejecting55']\n",
      "  geo_accession: ['GSM880472']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880472/suppl/GSM880472_non-rejecting55.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880473\n",
      "  title: ['non-rejecting219']\n",
      "  geo_accession: ['GSM880473']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880473/suppl/GSM880473_non-rejecting219.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880474\n",
      "  title: ['non-rejecting136']\n",
      "  geo_accession: ['GSM880474']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880474/suppl/GSM880474_non-rejecting136.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880475\n",
      "  title: ['non-rejecting137']\n",
      "  geo_accession: ['GSM880475']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880475/suppl/GSM880475_non-rejecting137.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880476\n",
      "  title: ['non-rejecting220']\n",
      "  geo_accession: ['GSM880476']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880476/suppl/GSM880476_non-rejecting220.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880477\n",
      "  title: ['ABMR10']\n",
      "  geo_accession: ['GSM880477']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880477/suppl/GSM880477_ABMR10.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880478\n",
      "  title: ['non-rejecting221']\n",
      "  geo_accession: ['GSM880478']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880478/suppl/GSM880478_non-rejecting221.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880479\n",
      "  title: ['non-rejecting88']\n",
      "  geo_accession: ['GSM880479']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880479/suppl/GSM880479_non-rejecting88.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880480\n",
      "  title: ['non-rejecting222']\n",
      "  geo_accession: ['GSM880480']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880480/suppl/GSM880480_non-rejecting222.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880481\n",
      "  title: ['non-rejecting223']\n",
      "  geo_accession: ['GSM880481']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880481/suppl/GSM880481_non-rejecting223.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880482\n",
      "  title: ['non-rejecting56']\n",
      "  geo_accession: ['GSM880482']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880482/suppl/GSM880482_non-rejecting56.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880483\n",
      "  title: ['non-rejecting57']\n",
      "  geo_accession: ['GSM880483']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880483/suppl/GSM880483_non-rejecting57.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880484\n",
      "  title: ['TCMR24']\n",
      "  geo_accession: ['GSM880484']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880484/suppl/GSM880484_TCMR24.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880485\n",
      "  title: ['TCMR25']\n",
      "  geo_accession: ['GSM880485']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880485/suppl/GSM880485_TCMR25.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880486\n",
      "  title: ['non-rejecting89']\n",
      "  geo_accession: ['GSM880486']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880486/suppl/GSM880486_non-rejecting89.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880487\n",
      "  title: ['non-rejecting90']\n",
      "  geo_accession: ['GSM880487']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880487/suppl/GSM880487_non-rejecting90.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880488\n",
      "  title: ['non-rejecting91']\n",
      "  geo_accession: ['GSM880488']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880488/suppl/GSM880488_non-rejecting91.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880489\n",
      "  title: ['non-rejecting92']\n",
      "  geo_accession: ['GSM880489']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880489/suppl/GSM880489_non-rejecting92.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880490\n",
      "  title: ['TCMR26']\n",
      "  geo_accession: ['GSM880490']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880490/suppl/GSM880490_TCMR26.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880491\n",
      "  title: ['non-rejecting264']\n",
      "  geo_accession: ['GSM880491']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880491/suppl/GSM880491_non-rejecting264.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880492\n",
      "  title: ['TCMR27']\n",
      "  geo_accession: ['GSM880492']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880492/suppl/GSM880492_TCMR27.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880493\n",
      "  title: ['TCMR28']\n",
      "  geo_accession: ['GSM880493']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880493/suppl/GSM880493_TCMR28.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880494\n",
      "  title: ['non-rejecting166']\n",
      "  geo_accession: ['GSM880494']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880494/suppl/GSM880494_non-rejecting166.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880495\n",
      "  title: ['non-rejecting167']\n",
      "  geo_accession: ['GSM880495']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880495/suppl/GSM880495_non-rejecting167.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880496\n",
      "  title: ['non-rejecting224']\n",
      "  geo_accession: ['GSM880496']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880496/suppl/GSM880496_non-rejecting224.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880497\n",
      "  title: ['non-rejecting138']\n",
      "  geo_accession: ['GSM880497']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880497/suppl/GSM880497_non-rejecting138.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880498\n",
      "  title: ['non-rejecting93']\n",
      "  geo_accession: ['GSM880498']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880498/suppl/GSM880498_non-rejecting93.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880499\n",
      "  title: ['non-rejecting34']\n",
      "  geo_accession: ['GSM880499']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880499/suppl/GSM880499_non-rejecting34.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880500\n",
      "  title: ['Nephrectomy8']\n",
      "  geo_accession: ['GSM880500']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['nephrectomy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): Nephrectomy', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880500/suppl/GSM880500_Nephrectomy8.CEL.gz']\n",
      "  relation: ['Reanalyzed by: GSM928892']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880501\n",
      "  title: ['non-rejecting225']\n",
      "  geo_accession: ['GSM880501']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880501/suppl/GSM880501_non-rejecting225.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880502\n",
      "  title: ['MIXED17']\n",
      "  geo_accession: ['GSM880502']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880502/suppl/GSM880502_MIXED17.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880503\n",
      "  title: ['ABMR11']\n",
      "  geo_accession: ['GSM880503']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880503/suppl/GSM880503_ABMR11.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880504\n",
      "  title: ['non-rejecting35']\n",
      "  geo_accession: ['GSM880504']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880504/suppl/GSM880504_non-rejecting35.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880505\n",
      "  title: ['MIXED18']\n",
      "  geo_accession: ['GSM880505']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880505/suppl/GSM880505_MIXED18.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880506\n",
      "  title: ['MIXED19']\n",
      "  geo_accession: ['GSM880506']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880506/suppl/GSM880506_MIXED19.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880507\n",
      "  title: ['non-rejecting168']\n",
      "  geo_accession: ['GSM880507']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880507/suppl/GSM880507_non-rejecting168.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880508\n",
      "  title: ['non-rejecting226']\n",
      "  geo_accession: ['GSM880508']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880508/suppl/GSM880508_non-rejecting226.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880509\n",
      "  title: ['non-rejecting36']\n",
      "  geo_accession: ['GSM880509']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880509/suppl/GSM880509_non-rejecting36.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880510\n",
      "  title: ['non-rejecting227']\n",
      "  geo_accession: ['GSM880510']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880510/suppl/GSM880510_non-rejecting227.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880511\n",
      "  title: ['non-rejecting169']\n",
      "  geo_accession: ['GSM880511']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880511/suppl/GSM880511_non-rejecting169.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880512\n",
      "  title: ['MIXED20']\n",
      "  geo_accession: ['GSM880512']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880512/suppl/GSM880512_MIXED20.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880513\n",
      "  title: ['non-rejecting37']\n",
      "  geo_accession: ['GSM880513']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880513/suppl/GSM880513_non-rejecting37.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880514\n",
      "  title: ['non-rejecting94']\n",
      "  geo_accession: ['GSM880514']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880514/suppl/GSM880514_non-rejecting94.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880515\n",
      "  title: ['non-rejecting170']\n",
      "  geo_accession: ['GSM880515']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880515/suppl/GSM880515_non-rejecting170.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880516\n",
      "  title: ['non-rejecting171']\n",
      "  geo_accession: ['GSM880516']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880516/suppl/GSM880516_non-rejecting171.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880517\n",
      "  title: ['ABMR12']\n",
      "  geo_accession: ['GSM880517']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880517/suppl/GSM880517_ABMR12.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880518\n",
      "  title: ['non-rejecting228']\n",
      "  geo_accession: ['GSM880518']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880518/suppl/GSM880518_non-rejecting228.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880519\n",
      "  title: ['ABMR58']\n",
      "  geo_accession: ['GSM880519']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880519/suppl/GSM880519_ABMR58.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880520\n",
      "  title: ['non-rejecting229']\n",
      "  geo_accession: ['GSM880520']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880520/suppl/GSM880520_non-rejecting229.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880521\n",
      "  title: ['ABMR59']\n",
      "  geo_accession: ['GSM880521']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880521/suppl/GSM880521_ABMR59.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880522\n",
      "  title: ['non-rejecting172']\n",
      "  geo_accession: ['GSM880522']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880522/suppl/GSM880522_non-rejecting172.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880523\n",
      "  title: ['non-rejecting230']\n",
      "  geo_accession: ['GSM880523']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880523/suppl/GSM880523_non-rejecting230.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880524\n",
      "  title: ['non-rejecting95']\n",
      "  geo_accession: ['GSM880524']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880524/suppl/GSM880524_non-rejecting95.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880525\n",
      "  title: ['non-rejecting231']\n",
      "  geo_accession: ['GSM880525']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880525/suppl/GSM880525_non-rejecting231.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880526\n",
      "  title: ['non-rejecting281']\n",
      "  geo_accession: ['GSM880526']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880526/suppl/GSM880526_non-rejecting281.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880527\n",
      "  title: ['non-rejecting232']\n",
      "  geo_accession: ['GSM880527']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880527/suppl/GSM880527_non-rejecting232.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880528\n",
      "  title: ['non-rejecting233']\n",
      "  geo_accession: ['GSM880528']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880528/suppl/GSM880528_non-rejecting233.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880529\n",
      "  title: ['ABMR60']\n",
      "  geo_accession: ['GSM880529']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880529/suppl/GSM880529_ABMR60.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880530\n",
      "  title: ['TCMR29']\n",
      "  geo_accession: ['GSM880530']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880530/suppl/GSM880530_TCMR29.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880531\n",
      "  title: ['ABMR13']\n",
      "  geo_accession: ['GSM880531']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880531/suppl/GSM880531_ABMR13.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880532\n",
      "  title: ['non-rejecting38']\n",
      "  geo_accession: ['GSM880532']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880532/suppl/GSM880532_non-rejecting38.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880533\n",
      "  title: ['non-rejecting234']\n",
      "  geo_accession: ['GSM880533']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880533/suppl/GSM880533_non-rejecting234.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880534\n",
      "  title: ['non-rejecting139']\n",
      "  geo_accession: ['GSM880534']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880534/suppl/GSM880534_non-rejecting139.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880535\n",
      "  title: ['ABMR61']\n",
      "  geo_accession: ['GSM880535']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880535/suppl/GSM880535_ABMR61.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880536\n",
      "  title: ['non-rejecting235']\n",
      "  geo_accession: ['GSM880536']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880536/suppl/GSM880536_non-rejecting235.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880537\n",
      "  title: ['non-rejecting58']\n",
      "  geo_accession: ['GSM880537']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880537/suppl/GSM880537_non-rejecting58.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880538\n",
      "  title: ['non-rejecting59']\n",
      "  geo_accession: ['GSM880538']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880538/suppl/GSM880538_non-rejecting59.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880539\n",
      "  title: ['non-rejecting60']\n",
      "  geo_accession: ['GSM880539']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880539/suppl/GSM880539_non-rejecting60.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880540\n",
      "  title: ['non-rejecting265']\n",
      "  geo_accession: ['GSM880540']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880540/suppl/GSM880540_non-rejecting265.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880541\n",
      "  title: ['non-rejecting236']\n",
      "  geo_accession: ['GSM880541']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880541/suppl/GSM880541_non-rejecting236.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880542\n",
      "  title: ['non-rejecting237']\n",
      "  geo_accession: ['GSM880542']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880542/suppl/GSM880542_non-rejecting237.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880543\n",
      "  title: ['non-rejecting96']\n",
      "  geo_accession: ['GSM880543']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880543/suppl/GSM880543_non-rejecting96.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880544\n",
      "  title: ['non-rejecting238']\n",
      "  geo_accession: ['GSM880544']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880544/suppl/GSM880544_non-rejecting238.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880545\n",
      "  title: ['non-rejecting61']\n",
      "  geo_accession: ['GSM880545']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880545/suppl/GSM880545_non-rejecting61.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880546\n",
      "  title: ['non-rejecting97']\n",
      "  geo_accession: ['GSM880546']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880546/suppl/GSM880546_non-rejecting97.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880547\n",
      "  title: ['non-rejecting140']\n",
      "  geo_accession: ['GSM880547']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880547/suppl/GSM880547_non-rejecting140.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880548\n",
      "  title: ['non-rejecting141']\n",
      "  geo_accession: ['GSM880548']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880548/suppl/GSM880548_non-rejecting141.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880549\n",
      "  title: ['non-rejecting98']\n",
      "  geo_accession: ['GSM880549']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880549/suppl/GSM880549_non-rejecting98.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880550\n",
      "  title: ['non-rejecting239']\n",
      "  geo_accession: ['GSM880550']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880550/suppl/GSM880550_non-rejecting239.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880551\n",
      "  title: ['non-rejecting142']\n",
      "  geo_accession: ['GSM880551']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880551/suppl/GSM880551_non-rejecting142.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880552\n",
      "  title: ['non-rejecting62']\n",
      "  geo_accession: ['GSM880552']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880552/suppl/GSM880552_non-rejecting62.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880553\n",
      "  title: ['non-rejecting266']\n",
      "  geo_accession: ['GSM880553']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880553/suppl/GSM880553_non-rejecting266.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880554\n",
      "  title: ['non-rejecting267']\n",
      "  geo_accession: ['GSM880554']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880554/suppl/GSM880554_non-rejecting267.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880555\n",
      "  title: ['non-rejecting39']\n",
      "  geo_accession: ['GSM880555']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880555/suppl/GSM880555_non-rejecting39.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880556\n",
      "  title: ['non-rejecting40']\n",
      "  geo_accession: ['GSM880556']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880556/suppl/GSM880556_non-rejecting40.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880557\n",
      "  title: ['non-rejecting41']\n",
      "  geo_accession: ['GSM880557']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880557/suppl/GSM880557_non-rejecting41.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880558\n",
      "  title: ['non-rejecting240']\n",
      "  geo_accession: ['GSM880558']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880558/suppl/GSM880558_non-rejecting240.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880559\n",
      "  title: ['non-rejecting99']\n",
      "  geo_accession: ['GSM880559']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880559/suppl/GSM880559_non-rejecting99.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880560\n",
      "  title: ['non-rejecting173']\n",
      "  geo_accession: ['GSM880560']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880560/suppl/GSM880560_non-rejecting173.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880561\n",
      "  title: ['non-rejecting42']\n",
      "  geo_accession: ['GSM880561']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880561/suppl/GSM880561_non-rejecting42.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880562\n",
      "  title: ['non-rejecting43']\n",
      "  geo_accession: ['GSM880562']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880562/suppl/GSM880562_non-rejecting43.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880563\n",
      "  title: ['non-rejecting44']\n",
      "  geo_accession: ['GSM880563']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880563/suppl/GSM880563_non-rejecting44.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880564\n",
      "  title: ['TCMR30']\n",
      "  geo_accession: ['GSM880564']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880564/suppl/GSM880564_TCMR30.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880565\n",
      "  title: ['ABMR62']\n",
      "  geo_accession: ['GSM880565']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880565/suppl/GSM880565_ABMR62.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880566\n",
      "  title: ['non-rejecting174']\n",
      "  geo_accession: ['GSM880566']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880566/suppl/GSM880566_non-rejecting174.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880567\n",
      "  title: ['non-rejecting241']\n",
      "  geo_accession: ['GSM880567']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880567/suppl/GSM880567_non-rejecting241.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880568\n",
      "  title: ['non-rejecting45']\n",
      "  geo_accession: ['GSM880568']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880568/suppl/GSM880568_non-rejecting45.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880569\n",
      "  title: ['non-rejecting242']\n",
      "  geo_accession: ['GSM880569']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880569/suppl/GSM880569_non-rejecting242.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880570\n",
      "  title: ['TCMR31']\n",
      "  geo_accession: ['GSM880570']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880570/suppl/GSM880570_TCMR31.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880571\n",
      "  title: ['non-rejecting243']\n",
      "  geo_accession: ['GSM880571']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880571/suppl/GSM880571_non-rejecting243.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880572\n",
      "  title: ['non-rejecting244']\n",
      "  geo_accession: ['GSM880572']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880572/suppl/GSM880572_non-rejecting244.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880573\n",
      "  title: ['ABMR63']\n",
      "  geo_accession: ['GSM880573']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880573/suppl/GSM880573_ABMR63.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880574\n",
      "  title: ['non-rejecting245']\n",
      "  geo_accession: ['GSM880574']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880574/suppl/GSM880574_non-rejecting245.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880575\n",
      "  title: ['non-rejecting246']\n",
      "  geo_accession: ['GSM880575']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880575/suppl/GSM880575_non-rejecting246.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880576\n",
      "  title: ['TCMR32']\n",
      "  geo_accession: ['GSM880576']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880576/suppl/GSM880576_TCMR32.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880577\n",
      "  title: ['non-rejecting268']\n",
      "  geo_accession: ['GSM880577']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880577/suppl/GSM880577_non-rejecting268.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880578\n",
      "  title: ['non-rejecting46']\n",
      "  geo_accession: ['GSM880578']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880578/suppl/GSM880578_non-rejecting46.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880579\n",
      "  title: ['non-rejecting47']\n",
      "  geo_accession: ['GSM880579']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880579/suppl/GSM880579_non-rejecting47.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880580\n",
      "  title: ['non-rejecting175']\n",
      "  geo_accession: ['GSM880580']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880580/suppl/GSM880580_non-rejecting175.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880581\n",
      "  title: ['TCMR33']\n",
      "  geo_accession: ['GSM880581']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880581/suppl/GSM880581_TCMR33.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880582\n",
      "  title: ['non-rejecting48']\n",
      "  geo_accession: ['GSM880582']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880582/suppl/GSM880582_non-rejecting48.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880583\n",
      "  title: ['non-rejecting247']\n",
      "  geo_accession: ['GSM880583']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880583/suppl/GSM880583_non-rejecting247.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880584\n",
      "  title: ['TCMR34']\n",
      "  geo_accession: ['GSM880584']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): TCMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880584/suppl/GSM880584_TCMR34.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880585\n",
      "  title: ['non-rejecting269']\n",
      "  geo_accession: ['GSM880585']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880585/suppl/GSM880585_non-rejecting269.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880586\n",
      "  title: ['non-rejecting100']\n",
      "  geo_accession: ['GSM880586']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880586/suppl/GSM880586_non-rejecting100.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880587\n",
      "  title: ['non-rejecting248']\n",
      "  geo_accession: ['GSM880587']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880587/suppl/GSM880587_non-rejecting248.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880588\n",
      "  title: ['non-rejecting176']\n",
      "  geo_accession: ['GSM880588']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880588/suppl/GSM880588_non-rejecting176.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880589\n",
      "  title: ['non-rejecting177']\n",
      "  geo_accession: ['GSM880589']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880589/suppl/GSM880589_non-rejecting177.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880590\n",
      "  title: ['non-rejecting178']\n",
      "  geo_accession: ['GSM880590']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880590/suppl/GSM880590_non-rejecting178.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880591\n",
      "  title: ['ABMR14']\n",
      "  geo_accession: ['GSM880591']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880591/suppl/GSM880591_ABMR14.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880592\n",
      "  title: ['non-rejecting270']\n",
      "  geo_accession: ['GSM880592']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880592/suppl/GSM880592_non-rejecting270.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880593\n",
      "  title: ['non-rejecting249']\n",
      "  geo_accession: ['GSM880593']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880593/suppl/GSM880593_non-rejecting249.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880594\n",
      "  title: ['non-rejecting250']\n",
      "  geo_accession: ['GSM880594']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880594/suppl/GSM880594_non-rejecting250.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880595\n",
      "  title: ['non-rejecting49']\n",
      "  geo_accession: ['GSM880595']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880595/suppl/GSM880595_non-rejecting49.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880596\n",
      "  title: ['ABMR15']\n",
      "  geo_accession: ['GSM880596']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880596/suppl/GSM880596_ABMR15.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880597\n",
      "  title: ['non-rejecting179']\n",
      "  geo_accession: ['GSM880597']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880597/suppl/GSM880597_non-rejecting179.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880598\n",
      "  title: ['non-rejecting50']\n",
      "  geo_accession: ['GSM880598']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880598/suppl/GSM880598_non-rejecting50.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880599\n",
      "  title: ['non-rejecting251']\n",
      "  geo_accession: ['GSM880599']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880599/suppl/GSM880599_non-rejecting251.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880600\n",
      "  title: ['non-rejecting143']\n",
      "  geo_accession: ['GSM880600']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880600/suppl/GSM880600_non-rejecting143.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880601\n",
      "  title: ['MIXED21']\n",
      "  geo_accession: ['GSM880601']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880601/suppl/GSM880601_MIXED21.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880602\n",
      "  title: ['ABMR64']\n",
      "  geo_accession: ['GSM880602']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880602/suppl/GSM880602_ABMR64.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880603\n",
      "  title: ['non-rejecting101']\n",
      "  geo_accession: ['GSM880603']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880603/suppl/GSM880603_non-rejecting101.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880604\n",
      "  title: ['non-rejecting102']\n",
      "  geo_accession: ['GSM880604']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880604/suppl/GSM880604_non-rejecting102.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880605\n",
      "  title: ['ABMR16']\n",
      "  geo_accession: ['GSM880605']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): ABMR', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880605/suppl/GSM880605_ABMR16.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880606\n",
      "  title: ['ABMR17']\n",
      "  geo_accession: ['GSM880606']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880606/suppl/GSM880606_ABMR17.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880607\n",
      "  title: ['non-rejecting144']\n",
      "  geo_accession: ['GSM880607']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880607/suppl/GSM880607_non-rejecting144.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880608\n",
      "  title: ['non-rejecting145']\n",
      "  geo_accession: ['GSM880608']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880608/suppl/GSM880608_non-rejecting145.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880609\n",
      "  title: ['non-rejecting271']\n",
      "  geo_accession: ['GSM880609']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Others', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880609/suppl/GSM880609_non-rejecting271.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880610\n",
      "  title: ['non-rejecting103']\n",
      "  geo_accession: ['GSM880610']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): 0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880610/suppl/GSM880610_non-rejecting103.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880611\n",
      "  title: ['MIXED22']\n",
      "  geo_accession: ['GSM880611']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): MIXED', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Mixed', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880611/suppl/GSM880611_MIXED22.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880612\n",
      "  title: ['ABMR65']\n",
      "  geo_accession: ['GSM880612']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): ABMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880612/suppl/GSM880612_ABMR65.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880613\n",
      "  title: ['non-rejecting104']\n",
      "  geo_accession: ['GSM880613']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): non-rejecting', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): Borderline', 'ah-lesion score (0, >0): >0']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880613/suppl/GSM880613_non-rejecting104.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n",
      "Sample: GSM880614\n",
      "  title: ['TCMR35']\n",
      "  geo_accession: ['GSM880614']\n",
      "  status: ['Public on Feb 13 2013']\n",
      "  submission_date: ['Feb 24 2012']\n",
      "  last_update_date: ['Jun 11 2022']\n",
      "  type: ['RNA']\n",
      "  channel_count: ['1']\n",
      "  source_name_ch1: ['renal allograft biopsy']\n",
      "  organism_ch1: ['Homo sapiens']\n",
      "  taxid_ch1: ['9606']\n",
      "  characteristics_ch1: ['diagnosis (tcmr,abmr,mixed,non-rejecting): TCMR', 'histologic diagnosis (abmr, tcmr, bdl, mixed, others): NA', 'ah-lesion score (0, >0): NA']\n",
      "  treatment_protocol_ch1: ['The immunosuppressive treatment of the patients before the biopsy was based on individual treatment regiments; the treatment after the biopsy was adjusted based on the histopathological diagnosis.']\n",
      "  growth_protocol_ch1: ['All human renal allograft biopsies taken for clinical indication during the period specified above were included. Tissue was immediately placed in RNA later.']\n",
      "  molecule_ch1: ['total RNA']\n",
      "  extract_protocol_ch1: ['Following homogenization of the tissue in 0.5ml of Trizol reagent (Invitrogen, Carlsbad, CA), total RNA was extracted and purified using the Rneasy Micro Kit (Quiagen, Ontario, Canada) (average yield 4Âµg/core, (RNA integrity number >7).']\n",
      "  label_ch1: ['biotin']\n",
      "  label_protocol_ch1: ['RNA (1-2Âµg) was labeled using GeneChip HT One-Cycle Target Labeling and Control kit. Quality of labeled cRNA was assessed on an Agilent 2100 Bioanalyzer (Agilent, Palo Alto, CA) before hybridization to HGU133Plus2.0. GeneChip (Affymetrix, Santa Clara, CA).']\n",
      "  hyb_protocol: ['Following fragmentation, 15 ug of cRNA were hybridized for 16 hr at 45C on GeneChip HGU133 Plus 2.0 Human Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.']\n",
      "  scan_protocol: ['GeneChips were scanned using Gene Array Scanner (Affymetrix) and processed with GeneChip Operating Software Version 1.4.0 (Affynetrix).']\n",
      "  data_processing: ['Microarray data files were preprocessed using robust multi-chip averaging (RMA) implemented in Bioconductor.']\n",
      "  platform_id: ['GPL570']\n",
      "  contact_name: ['Jessica,,Chang']\n",
      "  contact_email: ['jjchang@ualberta.ca']\n",
      "  contact_laboratory: ['ATAGC']\n",
      "  contact_department: ['Medicine']\n",
      "  contact_institute: ['University of Alberta']\n",
      "  contact_address: ['250 Heritage Medical Research Centre']\n",
      "  contact_city: ['Edmonton']\n",
      "  contact_zip/postal_code: ['T6G 2S2']\n",
      "  contact_country: ['Canada']\n",
      "  supplementary_file: ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM880nnn/GSM880614/suppl/GSM880614_TCMR35.CEL.gz']\n",
      "  series_id: ['GSE36059']\n",
      "  data_row_count: ['54675']\n",
      "--------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    print(\"Sample:\", gsm_name)\n",
    "    for k, v in gsm.metadata.items():\n",
    "        print(f\"  {k}: {v}\")\n",
    "    print(\"-\" * 50)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "               Diagnosis\n",
      "GSM880204  Non-rejecting\n",
      "GSM880205  Non-rejecting\n",
      "GSM880206  Non-rejecting\n",
      "GSM880207  Non-rejecting\n",
      "GSM880208  Non-rejecting\n"
     ]
    }
   ],
   "source": [
    "import GEOparse\n",
    "import pandas as pd\n",
    "\n",
    "# 假设你已经通过下述方式获取到 GSE 对象\n",
    "# gse = GEOparse.get_GEO(\"GSEXXXXXX\", destdir=\"data\")\n",
    "\n",
    "# 用于存储 {样本名: 诊断类型} 的字典\n",
    "diagnosis_dict = {}\n",
    "\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    # 获取每个样本的 characteristics_ch1 列表\n",
    "    char_list = gsm.metadata.get(\"characteristics_ch1\", [])\n",
    "\n",
    "    # 先给诊断类型一个默认值\n",
    "    rejection_type = \"Unknown\"\n",
    "\n",
    "    # 遍历每个 characteristics_ch1 字符串\n",
    "    for line in char_list:\n",
    "        # 检查是否有以 \"diagnosis (tcmr,abmr,mixed,non-rejecting):\" 开头的行\n",
    "        # 有些数据集大小写不一致，为避免报错可全部转为小写再比较\n",
    "        if line.lower().startswith(\"diagnosis (tcmr,abmr,mixed,non-rejecting):\"):\n",
    "            # 取冒号后面的部分，并去掉前后空格\n",
    "            status = line.split(\":\", 1)[1].strip()\n",
    "            # 用小写来比较，避免大小写问题\n",
    "            status_lower = status.lower()\n",
    "\n",
    "            if status_lower == \"tcmr\":\n",
    "                rejection_type = \"TCMR\"\n",
    "            elif status_lower == \"abmr\":\n",
    "                rejection_type = \"ABMR\"\n",
    "            elif status_lower == \"mixed\":\n",
    "                rejection_type = \"Mixed\"\n",
    "            elif status_lower == \"non-rejecting\":\n",
    "                rejection_type = \"Non-rejecting\"\n",
    "            else:\n",
    "                rejection_type = f\"Unrecognized: {status}\"\n",
    "            break  # 找到后即可跳出循环（假设只需要这一条）\n",
    "\n",
    "    # 把结果存入字典\n",
    "    diagnosis_dict[gsm_name] = rejection_type\n",
    "\n",
    "# 将结果转为 pandas DataFrame，方便后续分析或合并\n",
    "df_diagnosis = pd.DataFrame.from_dict(diagnosis_dict, orient=\"index\", columns=[\"Diagnosis\"])\n",
    "\n",
    "# 查看结果\n",
    "print(df_diagnosis.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "          DDR1 /// MIR4640      RFC2     HSPA6       PAX8    GUCA1A  \\\n",
      "GSM880204         9.808114  6.039012  5.810846  10.342781  3.211123   \n",
      "GSM880205        10.682125  6.608906  5.847419  10.667633  2.924908   \n",
      "GSM880206         9.659079  6.025781  6.089887  10.385927  3.191762   \n",
      "GSM880207        10.083274  6.254128  5.386257  10.383023  3.096177   \n",
      "GSM880208        10.460354  6.304075  5.550745  11.367598  3.059629   \n",
      "\n",
      "          MIR5193 /// UBA7      THRA    PTPN21      CCL5    CYP2E1  ...  \\\n",
      "GSM880204         7.615631  5.473264  5.130847  5.967407  3.806599  ...   \n",
      "GSM880205         8.282238  5.222516  4.620912  8.913122  4.053772  ...   \n",
      "GSM880206         7.013927  5.483255  5.346077  5.872588  4.038955  ...   \n",
      "GSM880207          7.21746  5.364841  4.968738  5.622734  7.228756  ...   \n",
      "GSM880208         7.658014  5.402205  5.469187  4.728435  3.733386  ...   \n",
      "\n",
      "          AFFX-r2-Ec-bioB-3_at AFFX-r2-Ec-bioB-5_at AFFX-r2-Ec-bioB-M_at  \\\n",
      "GSM880204              8.40538             7.976476             7.577744   \n",
      "GSM880205             8.385829             8.377783             8.108189   \n",
      "GSM880206             8.621434             8.669645             8.406157   \n",
      "GSM880207             9.197614             9.086772             8.752838   \n",
      "GSM880208             8.391848             8.082121             8.007603   \n",
      "\n",
      "          AFFX-r2-Ec-bioC-3_at AFFX-r2-Ec-bioC-5_at AFFX-r2-Ec-bioD-3_at  \\\n",
      "GSM880204             9.686881             9.287713            11.949707   \n",
      "GSM880205            10.007514             9.596678            12.094172   \n",
      "GSM880206            10.095641              9.83134            12.238378   \n",
      "GSM880207            10.489239            10.235579             12.55139   \n",
      "GSM880208             9.887173             9.587661            12.296905   \n",
      "\n",
      "          AFFX-r2-Ec-bioD-5_at AFFX-r2-P1-cre-3_at AFFX-r2-P1-cre-5_at  \\\n",
      "GSM880204            11.562166           13.446336            13.31175   \n",
      "GSM880205            11.836863           13.640625            13.37622   \n",
      "GSM880206            11.861172           13.875352           13.715878   \n",
      "GSM880207            12.098305           13.951246           13.789232   \n",
      "GSM880208            11.817117           13.796431           13.553796   \n",
      "\n",
      "               Diagnosis  \n",
      "GSM880204  Non-rejecting  \n",
      "GSM880205  Non-rejecting  \n",
      "GSM880206  Non-rejecting  \n",
      "GSM880207  Non-rejecting  \n",
      "GSM880208  Non-rejecting  \n",
      "\n",
      "[5 rows x 54676 columns]\n"
     ]
    }
   ],
   "source": [
    "# 1) 转置 annotated_data 使行=样本, 列=基因\n",
    "ann_t = annotated_data  # ann_t.index = GSM, ann_t.columns = 基因/探针\n",
    "ann_t = ann_t.T\n",
    "\n",
    "# 2) 与df_diagnosis合并\n",
    "#   df_diagnosis 也是行= GSM, 列= [Diagnosis]\n",
    "ann_merged = ann_t.merge(df_diagnosis, left_index=True, right_index=True, how=\"left\")\n",
    "# 现在 ann_t 中多了一列 \"Diagnosis\"\n",
    "\n",
    "# 观察结果\n",
    "print(ann_merged.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "ann_merged = ann_merged.drop([\"ID\", \"Gene Symbol\"], axis=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "ann_merged.to_csv(\"gse36059.csv\", index=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.model_selection import train_test_split\n",
    "# 创建一个新的二分类标签列\n",
    "df[\"Label\"] = df[\"Diagnosis\"].apply(lambda x: 1 if x in [\"TCMR\", \"ABMR\", \"Mixed\"] else 0)\n",
    "\n",
    "# 选出特征（去掉 Diagnosis 和 Label 列）\n",
    "X = df.drop(columns=[\"Diagnosis\", \"Label\", \"Unnamed: 0\"])\n",
    "y = df[\"Label\"]\n",
    "\n",
    "# 划分训练集和测试集 (80% 训练, 20% 测试)\n",
    "X_train, X_test, y_train, y_test = train_test_split(X, y,\n",
    "                                                                                                        test_size=0.3,\n",
    "                                                    random_state=42,\n",
    "                                                    stratify=y)  # 保持分类比例一致"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "随机森林模型在测试集上的准确率: 0.7177\n"
     ]
    }
   ],
   "source": [
    "from sklearn.ensemble import RandomForestClassifier\n",
    "\n",
    "# 创建随机森林分类器\n",
    "rf = RandomForestClassifier(n_estimators=500,\n",
    "                            random_state=42,\n",
    "                            n_jobs=-1)  # 使用所有CPU加速训练\n",
    "\n",
    "# 训练模型\n",
    "rf.fit(X_train, y_train)\n",
    "\n",
    "# 在测试集上评估\n",
    "accuracy = rf.score(X_test, y_test)\n",
    "print(f\"随机森林模型在测试集上的准确率: {accuracy:.4f}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.preprocessing import StandardScaler\n",
    "X = df.drop(columns=[\"Diagnosis\", \"Label\", \"Unnamed: 0\"])\n",
    "y = df[\"Label\"]\n",
    "\n",
    "# 分割数据为训练集和测试集\n",
    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=42)\n",
    "\n",
    "# 标准化数据\n",
    "scaler = StandardScaler()\n",
    "X_train_scaled = scaler.fit_transform(X_train)\n",
    "X_test_scaled = scaler.transform(X_test)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "随机森林筛选出的前 30 个重要基因:\n",
      "                                                    Gene  Importance\n",
      "10077                                             WARS.1    0.003889\n",
      "18726  HLA-DQB1 /// HLA-DRB1 /// HLA-DRB4 /// HLA-DRB...    0.002786\n",
      "13550                                               CCL4    0.002695\n",
      "10076                                               WARS    0.002423\n",
      "29061                                             GIMAP6    0.002387\n",
      "28868                                              PLA1A    0.002272\n",
      "17051                                               EMR2    0.002187\n",
      "23758                                            HLA-C.2    0.002157\n",
      "35730                                              NLRC5    0.002098\n",
      "38644                                           FAM26F.1    0.002082\n",
      "31155                                            HLA-F.1    0.002071\n",
      "18554                                            HLA-B.1    0.002063\n",
      "22191                                           CCDC69.1    0.002062\n",
      "54556                         GIMAP1-GIMAP5 /// GIMAP5.2    0.001895\n",
      "23915                                             PRF1.1    0.001803\n",
      "27607                                              ACSL5    0.001783\n",
      "2621                                           PSMB8-AS1    0.001752\n",
      "17521                                             FAM49A    0.001733\n",
      "12955                                           TNFRSF1B    0.001671\n",
      "27218                                               LAP3    0.001644\n",
      "12196                                               GBP2    0.001608\n",
      "34054                                            ASAP1.3    0.001588\n",
      "21976             HLA-DQA1 /// HLA-DQA2 /// LOC100509457    0.001571\n",
      "18144                                              HLA-B    0.001558\n",
      "18396                                           PECAM1.3    0.001547\n",
      "11859                                              IFI27    0.001540\n",
      "22716                                              CLIC2    0.001539\n",
      "21219                                            HLA-B.2    0.001503\n",
      "14124                                               CDH5    0.001501\n",
      "7239                                          1563509_at    0.001488\n",
      "已保存重要基因到 'top_30_genes_rf.csv'\n"
     ]
    }
   ],
   "source": [
    "# 获取特征重要性\n",
    "feature_importances = rf.feature_importances_\n",
    "\n",
    "# 创建 DataFrame\n",
    "importance_df = pd.DataFrame({\"Gene\": X.columns, \"Importance\": feature_importances})\n",
    "\n",
    "# 按重要性排序\n",
    "importance_df = importance_df.sort_values(by=\"Importance\", ascending=False)\n",
    "\n",
    "# 取最重要的前 20 个基因\n",
    "top_30_genes = importance_df.head(30)\n",
    "\n",
    "print(\"随机森林筛选出的前 30 个重要基因:\")\n",
    "print(top_30_genes)\n",
    "\n",
    "# 保存到 CSV\n",
    "top_30_genes.to_csv(\"36059_top_30_genes_rf.csv\", index=False)\n",
    "print(\"已保存重要基因到 'top_30_genes_rf.csv'\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Lasso 逻辑回归前 30 重要基因:\n",
      "                   Gene  Importance\n",
      "0                 MMRN1    0.164658\n",
      "1              CXCL11.1    0.150435\n",
      "2                CXCL11    0.145606\n",
      "3                  SOST    0.143730\n",
      "4                 OLFM4    0.134658\n",
      "5     FCGR3A /// FCGR3B    0.084708\n",
      "6             SLC38A4.1    0.071662\n",
      "7          ANKRD36BP2.1    0.065578\n",
      "8                 ACKR1    0.065479\n",
      "9                   REN    0.064766\n",
      "10          237625_s_at    0.058251\n",
      "11                CRHBP    0.056477\n",
      "12                 DNER    0.055811\n",
      "13                  CFD    0.049290\n",
      "14  TMSB15A /// TMSB15B    0.046109\n",
      "15                 RBP7    0.045844\n",
      "16               CXCL10    0.045730\n",
      "17                 CETP    0.045461\n",
      "18                FGB.1    0.041603\n",
      "19                CXCL9    0.040895\n",
      "20                 IDO1    0.038910\n",
      "21                 CCL4    0.038789\n",
      "22             GPR110.2    0.035484\n",
      "23              EIF5A.1    0.034955\n",
      "24           SERPINB6.2    0.033958\n",
      "25              CYP27B1    0.031355\n",
      "26             FAM26F.1    0.031183\n",
      "27              GALNT15    0.030771\n",
      "28                  VWF    0.030049\n",
      "29              THBS1.5    0.028201\n"
     ]
    }
   ],
   "source": [
    "## Lasso\n",
    "from sklearn.linear_model import LogisticRegression  # Import LogisticRegression\n",
    "lasso = LogisticRegression(penalty='l1', solver='liblinear', C=0.1, max_iter=5000) # Create a Lasso regression object\n",
    "\n",
    "lasso.fit(X, y)  # 训练模型\n",
    "coef = lasso.coef_[0]\n",
    "# 提取非零系数对应的基因\n",
    "lasso_importance = pd.Series(coef, index=X.columns).abs().sort_values(ascending=False)\n",
    "\n",
    "# 获取前 20 个基因\n",
    "top_genes_lasso = lasso_importance.head(30)\n",
    "#Rename the index instead of using columns\n",
    "top_genes_lasso = top_genes_lasso.rename_axis(\"Gene\").reset_index(name=\"Importance\")\n",
    "#top_genes_lasso.index.name = \"Gene\" # another way to rename index\n",
    "\n",
    "print(\"Lasso 逻辑回归前 30 重要基因:\")\n",
    "print(top_genes_lasso)\n",
    "\n",
    "# 保存到 CSV\n",
    "top_genes_lasso.to_csv(\"36059_top_genes_lasso.csv\", header=True, index=False) #avoid having an extra index column"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\ajxxs\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\xgboost\\core.py:158: UserWarning: [21:11:15] WARNING: C:\\buildkite-agent\\builds\\buildkite-windows-cpu-autoscaling-group-i-0ed59c031377d09b8-1\\xgboost\\xgboost-ci-windows\\src\\learner.cc:740: \n",
      "Parameters: { \"use_label_encoder\" } are not used.\n",
      "\n",
      "  warnings.warn(smsg, UserWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "XGBoost 筛选出的前 20 个重要基因:\n",
      "                            Gene  Importance\n",
      "0                      216571_at    0.056622\n",
      "1                           CCL4    0.051237\n",
      "2                           RBP7    0.036330\n",
      "3                   LOC100130950    0.034179\n",
      "4                        C9orf47    0.032541\n",
      "5                   LOC101928304    0.028879\n",
      "6                      232559_at    0.026174\n",
      "7                      243551_at    0.025429\n",
      "8                       ANGPT2.1    0.025180\n",
      "9                       TM6SF2.1    0.022975\n",
      "10                     LOC200830    0.019990\n",
      "11                         UTP23    0.018380\n",
      "12               BAGE2 /// BAGE4    0.017754\n",
      "13                       C4orf19    0.017443\n",
      "14                         CHST1    0.017018\n",
      "15                          IDO1    0.016905\n",
      "16                        NMNAT3    0.015586\n",
      "17                          PHF7    0.015575\n",
      "18                    1561906_at    0.014968\n",
      "19                 RP11-560A15.4    0.014857\n",
      "20  TRAV20 /// TRAV20 /// TRAV20    0.014791\n",
      "21                         PLA1A    0.014751\n",
      "22                     238181_at    0.014637\n",
      "23                      GAS2L1.2    0.013413\n",
      "24                        CXCL11    0.013165\n",
      "25                         PEX13    0.012308\n",
      "26                          TMC1    0.012217\n",
      "27                        NOTCH1    0.011783\n",
      "28                     216447_at    0.010643\n",
      "29                       RUFY2.3    0.010598\n"
     ]
    }
   ],
   "source": [
    "## XGboost\n",
    "import xgboost as xgb\n",
    "import pandas as pd\n",
    "from sklearn.model_selection import train_test_split\n",
    "X_train.columns = X_train.columns.str.replace(r\"[\\[\\]<>]\", \"\", regex=True)\n",
    "X_test.columns = X_test.columns.str.replace(r\"[\\[\\]<>]\", \"\", regex=True)\n",
    "# 训练 XGBoost 模型\n",
    "xgb_model = xgb.XGBClassifier(n_estimators=500, random_state=42, use_label_encoder=False, eval_metric='logloss')\n",
    "xgb_model.fit(X_train, y_train)\n",
    "\n",
    "# 提取基因重要性\n",
    "feature_importance_xgb = pd.Series(xgb_model.feature_importances_, index=X.columns).sort_values(ascending=False)\n",
    "\n",
    "# 获取前 20 个基因\n",
    "top_genes_xgb = feature_importance_xgb.head(30)\n",
    "\n",
    "# 转换为 DataFrame 并保存\n",
    "top_genes_xgb_df = pd.DataFrame({\"Gene\": top_genes_xgb.index, \"Importance\": top_genes_xgb.values})\n",
    "top_genes_xgb_df.to_csv(\"36059_top_genes_xgb.csv\", index=False)\n",
    "\n",
    "# 显示 XGBoost 筛选的前 20 个基因\n",
    "print(\"XGBoost 筛选出的前 20 个重要基因:\")\n",
    "print(top_genes_xgb_df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Gse50058's hug genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Accuracy of logistic regression model: 0.7229\n",
      "Logistic regression model AUC: 0.7731\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "from sklearn.linear_model import LogisticRegression\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.metrics import accuracy_score, roc_auc_score, roc_curve\n",
    "\n",
    "# 读取基因表达数据集\n",
    "df = pd.read_csv(\"gse36059.csv\")  # 请修改为你的数据文件路径\n",
    "\n",
    "# 确保目标变量 (1=排斥, 0=非排斥)\n",
    "df[\"Label\"] = df[\"Diagnosis\"].apply(lambda x: 1 if x in [\"TCMR\", \"ABMR\", \"Mixed\"] else 0)\n",
    "\n",
    "# 选择特征列（仅使用指定的 3 个基因）\n",
    "selected_genes = [\"FAM26F.1\", \"B2M.2\"]\n",
    "X_selected = df[selected_genes]  # 仅保留选定基因\n",
    "y = df[\"Label\"]\n",
    "\n",
    "# 划分训练集和测试集\n",
    "X_train, X_test, y_train, y_test = train_test_split(X_selected, y, test_size=0.2, random_state=42, stratify=y)\n",
    "\n",
    "# 训练逻辑回归模型\n",
    "log_reg = LogisticRegression(solver='liblinear', random_state=42)\n",
    "log_reg.fit(X_train, y_train)\n",
    "\n",
    "# 进行预测\n",
    "y_pred = log_reg.predict(X_test)\n",
    "y_pred_prob = log_reg.predict_proba(X_test)[:, 1]\n",
    "\n",
    "# 计算模型评估指标\n",
    "accuracy = accuracy_score(y_test, y_pred)\n",
    "auc = roc_auc_score(y_test, y_pred_prob)\n",
    "\n",
    "# 输出结果\n",
    "print(f\"Accuracy of logistic regression model: {accuracy:.4f}\")\n",
    "print(f\"Logistic regression model AUC: {auc:.4f}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAArMAAAIjCAYAAAAQgZNYAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjkuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8hTgPZAAAACXBIWXMAAA9hAAAPYQGoP6dpAACHWElEQVR4nOzdd1hTZ/8G8DsJJOygIkNBUdwTRUUFRQXFhaKgtna42traYWuXdmjt0C5b27datbVSq9YBaqkDiwMnrxvrQKzbKkNEGWEEkuf3hy/5iQwJAofA/bmuXJonZ3xPDuPmyXOeIxNCCBARERERmSC51AUQEREREVUUwywRERERmSyGWSIiIiIyWQyzRERERGSyGGaJiIiIyGQxzBIRERGRyWKYJSIiIiKTxTBLRERERCaLYZaIiIiITBbDLBGZjK+++grNmzeHQqGAp6en1OVUCZlMho8++qhStnX16lXIZDKEhYVVyvYIiImJgUwmQ0xMjNSlVJuwsDDIZDJcvXrV6HU/+ugjyGSyyi+K6AEMs0TlVPgDvfBhZmaGxo0bY+LEibh582aJ6wgh8Ntvv6Fv376wt7eHlZUVOnbsiI8//hgajabUfW3atAlDhgyBg4MDlEolGjVqhLFjx2L37t3lqjU3NxfffvstvL29oVarYWFhgVatWuGVV17BhQsXKnT8Uvvrr7/wzjvvwMfHBytWrMC8efOqdH8TJ06EjY1Nle6jsqxZswYLFy6s0n0UBuPCh1wuR/369TFkyBDExsZW6b7pvn79+kEmk6Fly5Ylvh4dHW04P+Hh4dVcHZF0zKQugMjUfPzxx2jWrBlyc3Px3//+F2FhYThw4ADOnDkDCwsLw3I6nQ7jx4/H+vXr0adPH3z00UewsrLC/v37MXfuXGzYsAE7d+6Ek5OTYR0hBCZPnoywsDB06dIFM2bMgLOzMxITE7Fp0yb4+/vj4MGD6N27d6n1paamYvDgwTh+/DiGDx+O8ePHw8bGBgkJCVi7di2WLVsGrVZbpe9RVdi9ezfkcjmWL18OpVIpdTlVJicnB2Zmxv1oXrNmDc6cOYPXX3+9SHvTpk2Rk5MDc3PzSqvvySefxNChQ6HT6XDhwgUsXrwY/fv3x9GjR9GxY8dK209N1bdvX+Tk5Ej2NWhhYYGLFy/iyJEj6NGjR5HXVq9eDQsLC+Tm5kpSG5FUGGaJjDRkyBB069YNAPDcc8/BwcEBX3zxBSIjIzF27FjDcl9++SXWr1+Pt956C1999ZWh/YUXXsDYsWMRHByMiRMnYvv27YbXFixYgLCwMLz++uv45ptvinw89/777+O33357ZNCZOHEiTp48ifDwcISEhBR57ZNPPsH777//WMdfqKCgAHq9vtp+qaekpMDS0rLS9ieEQG5uLiwtLStle5XlwT+IHpdMJqvU7QFA165d8fTTTxue9+nTB0OGDMGPP/6IxYsXV+q+HkWj0cDa2rpa9ymXyyv9PTWGh4cHCgoK8PvvvxcJs7m5udi0aROGDRuGiIgIyeojkgKHGRA9pj59+gAALl26ZGjLycnBV199hVatWmH+/PnF1gkKCsKECRMQFRWF//73v4Z15s+fjzZt2uDrr78ucZzZM888U6w35kGHDx/G1q1bMWXKlGJBFgBUKhW+/vprw/N+/fqhX79+xZabOHEi3N3dDc8LP2L++uuvsXDhQnh4eEClUuHkyZMwMzPD3Llzi20jISEBMpkMP/zwg6Ht3r17eP311+Hm5gaVSoUWLVrgiy++gF6vL/WYgPuhbMWKFdBoNIaPUQvHgRYUFOCTTz4x1OTu7o733nsPeXl5Rbbh7u6O4cOHY8eOHejWrRssLS2xdOnSMvdbHhs2bICXlxcsLS3h4OCAp59+usRhJxs2bEC7du1gYWGBDh06YNOmTcXe58JjfXDMbGZmJl5//XW4u7tDpVLB0dERAwcOxIkTJwDcP4dbt27FtWvXDO9N4TZLGzN7/vx5jB07Fg0bNoSlpSVat25d4T9ySvr6B8p/ru/cuYNnnnkGdnZ2sLe3x4QJE3Dq1KlidRcO+7h06RKGDh0KW1tbPPXUUwAAvV6PhQsXon379rCwsICTkxOmTp2Ku3fvFtnXsWPHEBgYCAcHB1haWqJZs2aYPHlykWXWrl0LLy8v2Nraws7ODh07dsR3331neL20MbPl+TooPIabN28iODgYNjY2aNiwId566y3odLpyv+dPPvkk1q1bV+S9/PPPP5GdnV3kD+oHnTx5EkOGDIGdnR1sbGzg7+9v+NnzoLNnz2LAgAGwtLSEq6srPv3001K/P7dv344+ffrA2toatra2GDZsGM6ePVvu4yCqLOyZJXpMhRdF1KtXz9B24MAB3L17F9OnTy+1J/XZZ5/FihUrsGXLFvTs2RMHDhxAWloaXn/9dSgUigrVEhkZCeB+6K0KK1asQG5uLl544QWoVCq4uLjAz88P69evx5w5c4osu27dOigUCowZMwYAkJ2dDT8/P9y8eRNTp05FkyZNcOjQIcyaNQuJiYlljvn87bffsGzZMhw5cgQ///wzABiGWjz33HP49ddfERoaijfffBOHDx/G/PnzER8fj02bNhXZTkJCAp588klMnToVzz//PFq3bv1Y70dYWBgmTZqE7t27Y/78+UhOTsZ3332HgwcP4uTJk7C3twcAbN26FePGjUPHjh0xf/583L17F1OmTEHjxo0fuY8XX3wR4eHheOWVV9CuXTvcuXMHBw4cQHx8PLp27Yr3338f6enp+Pfff/Htt98CQJljff/++2/06dMH5ubmeOGFF+Du7o5Lly7hzz//xGeffWb0e1DS1395z7Ver0dQUBCOHDmCl156CW3atMEff/yBCRMmlLivgoICBAYGwtfXF19//TWsrKwAAFOnTjWci9deew1XrlzBDz/8gJMnT+LgwYMwNzdHSkoKBg0ahIYNG2LmzJmwt7fH1atXsXHjRsP2o6Oj8eSTT8Lf3x9ffPEFACA+Ph4HDx7E9OnTS30Pyvt1ANwffhQYGAhvb298/fXX2LlzJxYsWAAPDw+89NJL5XrPx48fj48++ggxMTEYMGAAgPtDTfz9/eHo6Fhs+bNnz6JPnz6ws7PDO++8A3NzcyxduhT9+vXD3r174e3tDQBISkpC//79UVBQgJkzZ8La2hrLli0r8dOL3377DRMmTEBgYCC++OILZGdn48cff4Svry9OnjxZ7I80oioliKhcVqxYIQCInTt3itu3b4sbN26I8PBw0bBhQ6FSqcSNGzcMyy5cuFAAEJs2bSp1e2lpaQKAGD16tBBCiO++++6R6zzKqFGjBABx9+7dci3v5+cn/Pz8irVPmDBBNG3a1PD8ypUrAoCws7MTKSkpRZZdunSpACBOnz5dpL1du3ZiwIABhueffPKJsLa2FhcuXCiy3MyZM4VCoRDXr18vs9YJEyYIa2vrIm1xcXECgHjuueeKtL/11lsCgNi9e7ehrWnTpgKAiIqKKnM/Ze3vQVqtVjg6OooOHTqInJwcQ/uWLVsEADF79mxDW8eOHYWrq6vIzMw0tMXExAgARd5nIYQAIObMmWN4rlarxcsvv1xmrcOGDSu2HSH+/7ytWLHC0Na3b19ha2srrl27VmRZvV5f5j4KtzV37lxx+/ZtkZSUJPbv3y+6d+8uAIgNGzYYli3vuY6IiBAAxMKFCw3L6HQ6MWDAgGJ1T5gwQQAQM2fOLLLN/fv3CwBi9erVRdqjoqKKtG/atEkAEEePHi31GKdPny7s7OxEQUFBqcvs2bNHABB79uwRQhj3dVB4DB9//HGRbXbp0kV4eXmVus9Cfn5+on379kIIIbp16yamTJkihBDi7t27QqlUil9//dVQ34PnIzg4WCiVSnHp0iVD261bt4Stra3o27evoe31118XAMThw4cNbSkpKUKtVgsA4sqVK0IIITIzM4W9vb14/vnni9SXlJQk1Gp1kfY5c+YIRg2qahxmQGSkgIAANGzYEG5ubggNDYW1tTUiIyPh6upqWCYzMxMAYGtrW+p2Cl/LyMgo8m9Z6zxKZWyjLCEhIWjYsGGRttGjR8PMzAzr1q0ztJ05cwbnzp3DuHHjDG0bNmxAnz59UK9ePaSmphoeAQEB0Ol02Ldvn9H1bNu2DQAwY8aMIu1vvvkmgPs9og9q1qwZAgMDjd5PSY4dO4aUlBRMmzatyBjKYcOGoU2bNoZ937p1C6dPn8azzz5bpMfUz8+vXBdM2dvb4/Dhw7h169Zj13z79m3s27cPkydPRpMmTYq8Vt7pk+bMmYOGDRvC2dkZffr0QXx8PBYsWIDQ0FDDMuU911FRUTA3N8fzzz9vWFcul+Pll18udf8P915u2LABarUaAwcOLLIvLy8v2NjYYM+ePQBg6B3dsmUL8vPzS9y2vb09NBoNoqOjy/VeAOX/OnjQiy++WOR5nz59cPny5XLvE7jfO7tx40ZotVqEh4dDoVBg1KhRxZbT6XT466+/EBwcjObNmxvaXVxcMH78eBw4cMDwc2Pbtm3o2bNnkaFMDRs2NAznKBQdHY179+7hySefLPKeKxQKeHt7G95zourCMEtkpEWLFiE6Ohrh4eEYOnQoUlNToVKpiixTGCYLQ21JHg68dnZ2j1znUSpjG2Vp1qxZsTYHBwf4+/tj/fr1hrZ169bBzMwMo0ePNrT9888/iIqKQsOGDYs8AgICANy/wMtY165dg1wuR4sWLYq0Ozs7w97eHteuXXtk/RVVuO2Shiq0adPG8Hrhvw/XWFrbw7788kucOXMGbm5u6NGjBz766COjg0+hwvU6dOhQofWB+xcwRkdH488//8Qbb7yBnJycYuM9y3uur127BhcXF8NwgUKlvS9mZmZF/mgs3Fd6ejocHR2L7S8rK8uwLz8/P4SEhGDu3LlwcHDAyJEjsWLFiiJjq6dNm4ZWrVphyJAhcHV1xeTJkxEVFVXm+1Her4NCFhYWxf4grFevXrHxvY/yxBNPID09Hdu3b8fq1asxfPjwEv+IvX37NrKzs0usr23bttDr9bhx44bhWEqa9uvhdf/55x8AwIABA4q953/99VeFvpeJHgfHzBIZqUePHobZDIKDg+Hr64vx48cjISHB0PPWtm1bAPfHJwYHB5e4nb///hsA0K5dOwD3f/EBwOnTp0td51Ee3EbhhTllkclkEEIUay/tYpTSrvx/4oknMGnSJMTFxcHT0xPr16+Hv78/HBwcDMvo9XoMHDgQ77zzTonbaNWq1SPrLU15exVr2swF5TF27Fj06dMHmzZtwl9//YWvvvoKX3zxBTZu3IghQ4ZUez0tW7Y0hNLhw4dDoVBg5syZ6N+/v+H7oqrOtUqlglxetA9Gr9fD0dERq1evLnGdwuBYOPfqf//7X/z555/YsWMHJk+ejAULFuC///0vbGxs4OjoiLi4OOzYsQPbt2/H9u3bsWLFCjz77LP49ddfK1Tzwyo6Hv5hLi4u6NevHxYsWICDBw9W6wwGhReE/fbbb3B2di72urFTyxE9LvbMEj0GhUKB+fPn49atW0Wu2vf19YW9vT3WrFlTajBcuXIlgPuBoHCdevXq4ffffzfqyuYHBQUFAQBWrVpVruXr1auHe/fuFWt/uDfpUYKDg6FUKrFu3TrExcXhwoULeOKJJ4os4+HhgaysLAQEBJT4ePhj7/Jo2rQp9Hq9oaeoUHJyMu7du4emTZsavU1j9g3cv6jsYQkJCYbXC/+9ePFiseVKaiuJi4sLpk2bhs2bN+PKlSto0KBBkYu1yhvmCz9mPnPmTLmWL4/3338ftra2+OCDDwxt5T3XTZs2RWJiIrKzs4tss7zvS+G+7ty5Ax8fnxL31blz5yLL9+zZE5999hmOHTuG1atX4+zZs1i7dq3hdaVSiaCgICxevBiXLl3C1KlTsXLlylJrKu/XQVUYP3489u/fDzs7OwwdOrTEZRo2bAgrK6sS6zt//jzkcjnc3NwA3D+Wh7+XgOLH5uHhAQBwdHQs8T0vaYYUoqrEMEv0mPr164cePXpg4cKFhsnKrays8NZbbyEhIaHEKY+2bt2KsLAwBAYGomfPnoZ13n33XcTHx+Pdd98tscd01apVOHLkSKm19OrVC4MHD8bPP/+MzZs3F3tdq9XirbfeMjz38PDA+fPncfv2bUPbqVOncPDgwXIfP3B/rGFgYCDWr1+PtWvXQqlUFutdHjt2LGJjY7Fjx45i69+7dw8FBQVG7ROA4Rf4wzMhfPPNNwDuj1usKt26dYOjoyOWLFlS5KPq7du3Iz4+3rDvRo0aoUOHDli5ciWysrIMy+3duxenT58ucx86nQ7p6elF2hwdHdGoUaMi+7S2ti62XEkaNmyIvn374pdffsH169eLvFbS11t52NvbY+rUqdixYwfi4uIAlP9cBwYGIj8/Hz/99JPhdb1ej0WLFpV7/2PHjoVOp8Mnn3xS7LWCggLDH2t3794tdoyFt0QufC/v3LlT5HW5XI5OnToVWeZh5f06qAqhoaGYM2cOFi9eXOr8ywqFAoMGDcIff/xR5Ha0ycnJWLNmDXx9fQ3Dk4YOHYr//ve/RX7G3L59u1ivd2BgIOzs7DBv3rwSxx8/+POEqDrwswCiSvD2229jzJgxCAsLM1zcMXPmTJw8eRJffPEFYmNjERISAktLSxw4cACrVq1C27Zti310+fbbb+Ps2bNYsGAB9uzZg9DQUDg7OyMpKQmbN2/GkSNHcOjQoTJrWblyJQYNGoTRo0cjKCgI/v7+sLa2xj///IO1a9ciMTHRMNfs5MmT8c033yAwMBBTpkxBSkoKlixZgvbt2xsuCimvcePG4emnn8bixYsRGBhYZDqiwmOLjIzE8OHDMXHiRHh5eUGj0eD06dMIDw/H1atXiwxLKI/OnTtjwoQJWLZsGe7duwc/Pz8cOXIEv/76K4KDg9G/f3+jtvew/Px8fPrpp8Xa69evj2nTpuGLL77ApEmT4OfnhyeffNIwJZO7uzveeOMNw/Lz5s3DyJEj4ePjg0mTJuHu3bv44Ycf0KFDhyIB92GZmZlwdXVFaGgoOnfuDBsbG+zcuRNHjx7FggULDMt5eXlh3bp1mDFjBrp37w4bGxtDL/3Dvv/+e/j6+qJr16544YUX0KxZM1y9ehVbt241hFFjTZ8+HQsXLsTnn3+OtWvXlvtcBwcHo0ePHnjzzTdx8eJFtGnTBpGRkUhLSwNQvh5nPz8/TJ06FfPnz0dcXBwGDRoEc3Nz/PPPP9iwYQO+++47hIaG4tdff8XixYsxatQoeHh4IDMzEz/99FORXs3nnnsOaWlpGDBgAFxdXXHt2jX85z//gaenp2Ho0MPMzc3L/XVQ2dRqdZE5iUvz6aefIjo6Gr6+vpg2bRrMzMywdOlS5OXl4csvvzQs98477+C3337D4MGDMX36dMPUXE2bNjUMiwLuj83/8ccf8cwzz6Br16544okn0LBhQ1y/fh1bt26Fj49PkU+qiKqctJMpEJmOwqm5SpraR6fTCQ8PD+Hh4VFkWh+dTidWrFghfHx8hJ2dnbCwsBDt27cXc+fOFVlZWaXuKzw8XAwaNEjUr19fmJmZCRcXFzFu3DgRExNTrlqzs7PF119/Lbp37y5sbGyEUqkULVu2FK+++qq4ePFikWVXrVolmjdvLpRKpfD09BQ7duwodWqur776qtR9ZmRkCEtLSwFArFq1qsRlMjMzxaxZs0SLFi2EUqkUDg4Oonfv3uLrr78WWq22zGMqbaqs/Px8MXfuXNGsWTNhbm4u3NzcxKxZs0Rubm6R5Zo2bSqGDRtW5j4e3h+AEh8eHh6G5datWye6dOkiVCqVqF+/vnjqqafEv//+W2x7a9euFW3atBEqlUp06NBBREZGipCQENGmTZsiy+GBqbny8vLE22+/LTp37ixsbW2FtbW16Ny5s1i8eHGRdbKyssT48eOFvb19kem+SpqaSwghzpw5I0aNGiXs7e2FhYWFaN26tfjwww/LfD8e9TUwceJEoVAoDF9f5T3Xt2/fFuPHjxe2trZCrVaLiRMnioMHDwoAYu3atUXOR1lTpS1btkx4eXkJS0tLYWtrKzp27CjeeecdcevWLSGEECdOnBBPPvmkaNKkiVCpVMLR0VEMHz5cHDt2zLCNwu87R0dHoVQqRZMmTcTUqVNFYmKiYZmHp+YqVJ6vg9KOobzTVz04NVdpSpqaq/D4AwMDhY2NjbCyshL9+/cXhw4dKrb+33//Lfz8/ISFhYVo3Lix+OSTT8Ty5cuLTM314L4CAwOFWq0WFhYWwsPDQ0ycOLHIe8qpuag6yISo4GdLRET0WDw9PdGwYUOjpoKqCzZv3oxRo0bhwIED8PHxkbocIqrhOGaWiKiK5efnFxsTHBMTg1OnTtX5i2VycnKKPNfpdPjPf/4DOzs7dO3aVaKqiMiUcMwsEVEVu3nzJgICAvD000+jUaNGOH/+PJYsWQJnZ+diE+jXNa+++ipycnLQq1cv5OXlYePGjTh06BDmzZtnklOpEVH14zADIqIqlp6ejhdeeAEHDx7E7du3YW1tDX9/f3z++eeGaY7qqjVr1mDBggW4ePEicnNz0aJFC7z00kt45ZVXpC6NiEwEwywRERERmSyOmSUiIiIik8UwS0REREQmq85dAKbX63Hr1i3Y2tqW+xaQRERERFR9hBDIzMxEo0aNIJeX3fda58LsrVu3DPehJiIiIqKa68aNG3B1dS1zmToXZm1tbQHcf3MK70dNRERERDVHRkYG3NzcDLmtLHUuzBYOLbCzs2OYJSIiIqrByjMklBeAEREREZHJYpglIiIiIpPFMEtEREREJqvOjZktDyEECgoKoNPppC6FiEycQqGAmZkZpwIkIqoiDLMP0Wq1SExMRHZ2ttSlEFEtYWVlBRcXFyiVSqlLISKqdRhmH6DX63HlyhUoFAo0atQISqWSvSlEVGFCCGi1Wty+fRtXrlxBy5YtHzn5NxERGYdh9gFarRZ6vR5ubm6wsrKSuhwiqgUsLS1hbm6Oa9euQavVwsLCQuqSiIhqFXYRlIA9J0RUmfgzhYio6vAnLBERERGZLIZZIiIiIjJZDLNUbu7u7li4cGGF1w8LC4O9vX2l1VObPO57a4xnnnkG8+bNq5Z91QVLlixBUFCQ1GUQEdVZDLO1xMSJExEcHFyl+zh69CheeOGFci1bUjgbN24cLly4UOH9h4WFQSaTQSaTQS6Xw8XFBePGjcP169crvM2awpj39nGcOnUK27Ztw2uvvVbstd9//x0KhQIvv/xysdfK+kNEJpNh8+bNRdoiIiLQr18/qNVq2NjYoFOnTvj444+RlpZWGYdRorS0NDz11FOws7ODvb09pkyZgqysrFKXv3r1quHr6eHHhg0bABT9mnv4kZKSAgCYPHkyTpw4gf3791fZsRERUekYZqncGjZs+FizPFhaWsLR0fGxarCzs0NiYiJu3ryJiIgIJCQkYMyYMY+1zfLIz8+v0u0/7ntbXv/5z38wZswY2NjYFHtt+fLleOedd/D7778jNze3wvt4//33MW7cOHTv3h3bt2/HmTNnsGDBApw6dQq//fbb45Rfpqeeegpnz55FdHQ0tmzZgn379pX5B4KbmxsSExOLPObOnQsbGxsMGTIEwP0/wB5eJjAwEH5+foavZaVSifHjx+P777+vsmMjIqIyiDomPT1dABDp6enFXsvJyRHnzp0TOTk5hja9XoisLGkeen35j2vChAli5MiRpb4eExMjunfvLpRKpXB2dhbvvvuuyM/PN7yekZEhxo8fL6ysrISzs7P45ptvhJ+fn5g+fbphmaZNm4pvv/32f++LXsyZM0e4ubkJpVIpXFxcxKuvviqEEMLPz08AKPIQQogVK1YItVpdpK7IyEjRrVs3oVKpRIMGDURwcHCpx1DS+t9//32x87l582bRpUsXoVKpRLNmzcRHH31U5Fjj4+OFj4+PUKlUom3btiI6OloAEJs2bRJCCHHlyhUBQKxdu1b07dtXqFQqsWLFCiGEED/99JNo06aNUKlUonXr1mLRokWG7ebl5YmXX35ZODs7C5VKJZo0aSLmzZv3yPfr4fdWCCGuXbsmRowYIaytrYWtra0YM2aMSEpKMrw+Z84c0blzZ7Fy5UrRtGlTYWdnJ8aNGycyMjJKff8KCgqEWq0WW7ZsKfba5cuXhaWlpbh3757w9vYWq1evfuR7X+jB9+7w4cMCgFi4cGGJy969e7fU+h7HuXPnBABx9OhRQ9v27duFTCYTN2/eLPd2PD09xeTJk0t9PSUlRZibm4uVK1cWad+7d69QKpUiOzu7xPVK+tlCRESlKyuvPUzSntl9+/YhKCgIjRo1KvGjypLExMSga9euUKlUaNGiBcLCwqq0xuxswMZGmkdl3YTs5s2bGDp0KLp3745Tp07hxx9/xPLly/Hpp58alpkxYwYOHjyIyMhIREdHY//+/Thx4kSp24yIiMC3336LpUuX4p9//sHmzZvRsWNHAMDGjRvh6uqKjz/+2NCbVZKtW7di1KhRGDp0KE6ePIldu3ahR48e5T6ulJQUbNq0CQqFAgqFAgCwf/9+PPvss5g+fTrOnTuHpUuXIiwsDJ999hkAQKfTITg4GFZWVjh8+DCWLVuG999/v8Ttz5w5E9OnT0d8fDwCAwOxevVqzJ49G5999hni4+Mxb948fPjhh/j1118BAN9//z0iIyOxfv16JCQkYPXq1XB3d3/k+/UwvV6PkSNHIi0tDXv37kV0dDQuX76McePGFVnu0qVL2Lx5M7Zs2YItW7Zg7969+Pzzz0t9v/7++2+kp6ejW7duxV5bsWIFhg0bBrVajaeffhrLly8v+80vxerVq2FjY4Np06aV+HpZY6bbt28PGxubUh+FvaUliY2Nhb29fZFjCwgIgFwux+HDh8tV+/HjxxEXF4cpU6aUuszKlSthZWWF0NDQIu3dunVDQUFBufdFRESVR9KbJmg0GnTu3BmTJ0/G6NGjH7n8lStXMGzYMLz44otYvXo1du3aheeeew4uLi4IDAyshopN0+LFi+Hm5oYffvgBMpkMbdq0wa1bt/Duu+9i9uzZ0Gg0+PXXX7FmzRr4+/sDuB9uGjVqVOo2r1+/DmdnZwQEBMDc3BxNmjQxBNH69etDoVDA1tYWzs7OpW7js88+wxNPPIG5c+ca2jp37lzmsaSnp8PGxgZCCMMth1977TVYW1sDAObOnYuZM2diwoQJAIDmzZvjk08+wTvvvIM5c+YgOjoaly5dQkxMjKG2zz77DAMHDiy2r9dff73I1+WcOXOwYMECQ1uzZs0MgXnChAm4fv06WrZsCV9fX8hkMjRt2rRc79fDdu3ahdOnT+PKlStwc3MDcD9EtW/fHkePHkX37t0B3A+9YWFhsLW1BXD/wq5du3YZgvvDrl27BoVCUWyoR+F2/vOf/wAAnnjiCbz55pu4cuUKmjVrVuq5KMk///yD5s2bw9zc3Kj1AGDbtm1lDuewtLQs9bWkpKRix2VmZob69esjKSmpXPtfvnw52rZti969e5e5zPjx44vVYmVlBbVajWvXrpVrX0REVHkkDbNDhgwps7flYUuWLEGzZs2wYMECAEDbtm1x4MABfPvtt1UWZq2sgDKuIalSlTWEMj4+Hr169Spya14fHx9kZWXh33//xd27d5Gfn18kXKnVarRu3brUbY4ZMwYLFy5E8+bNMXjwYAwdOhRBQUEwMyv/l1RcXByef/55o47F1tYWJ06cQH5+PrZv347Vq1cXCW+nTp3CwYMHi7TpdDrk5uYiOzsbCQkJcHNzKxKySwuVD/byaTQaXLp0CVOmTClSc0FBAdRqNYD7F+ENHDgQrVu3xuDBgzF8+HAMGjQIgHHvV3x8PNzc3AxBFgDatWsHe3t7xMfHG8Ksu7u7IcgCgIuLi+GipJLk5ORApVIVu0VzdHQ0NBoNhg4dCgBwcHDAwIED8csvv+CTTz4pdXslEUIYtfyDHgz/1S0nJwdr1qzBhx9+WOoysbGxiI+PL3Xcr6WlpeEPLCKimkCIyvuUV6/XQy6Xw8oKeOjXiORM6na2sbGxCAgIKNIWGBiI119/vdR18vLykJeXZ3iekZFh1D5lMuB/nX70ADc3NyQkJGDnzp2Ijo7GtGnT8NVXX2Hv3r3l7pUrq6etNHK5HC1atABw/4+ZS5cu4aWXXjIEjKysLMydO7fEnn5jbyNq/cCJL7wq/qeffoK3t3eR5QqHOHTt2hVXrlzB9u3bsXPnTowdOxYBAQEIDw+vlPfrYQ+vJ5PJoNfrS13ewcEB2dnZ0Gq1UCqVhvbly5cjLS2tyPnQ6/X4+++/MXfuXMjlctjZ2UGj0Rh+mBW6d+8eABgCfatWrXDgwAHk5+cbfVzt27cvs2ezT58+2L59e4mvOTs7FwvyBQUFSEtLK/PTgULh4eHIzs7Gs88+W+oyP//8Mzw9PeHl5VXi62lpaWjYsOEj90VEVB2EAHx9gUOHHntL6Nr1JHr2/C9++WUyUlMtalwuMqnZDJKSkuDk5FSkzcnJCRkZGcjJySlxnfnz50OtVhseD/Z21RVt27ZFbGxskV6zgwcPwtbWFq6uroaPhY8ePWp4PT09/ZHTaFlaWiIoKAjff/89YmJiEBsbi9OnTwO4f4W3Tqcrc/1OnTph165dj3Fk98e1rlu3zjC+t2vXrkhISECLFi2KPeRyOVq3bo0bN24gOTnZsI0Hj7s0Tk5OaNSoES5fvlxsuw9+FG9nZ4dx48bhp59+wrp16xAREWGYjqqs9+tBbdu2xY0bN3Djxg1D27lz53Dv3j20a9euwu+Vp6enYVuF7ty5gz/++ANr165FXFyc4XHy5EncvXsXf/31FwCgdevWKCgoQFxcXJFtFr7vrVq1AgCMHz8eWVlZWLx4cYk1FIbfkmzbtq1IDQ8/fv7551LX7dWrF+7du4fjx48b2nbv3g29Xl/sj4+SLF++HCNGjCg1jGZlZWH9+vWljqe9dOkScnNz0aVLl0fui4ioOmRnP36QVSrzEBKyESNG/AlHx9vo1u3Rvy+lYFI9sxUxa9YszJgxw/A8IyOj1gba9PT0YmGjQYMGmDZtGhYuXIhXX30Vr7zyChISEjBnzhzMmDEDcrkctra2mDBhAt5++23Ur18fjo6OmDNnDuRyebGPpAuFhYVBp9PB29sbVlZWWLVqFSwtLQ0fFbu7u2Pfvn144oknoFKp4ODgUGwbc+bMgb+/Pzw8PPDEE0+goKAA27Ztw7vvvlvuY3Zzc8OoUaMwe/ZsbNmyBbNnz8bw4cPRpEkThIaGQi6X49SpUzhz5gw+/fRTDBw4EB4eHpgwYQK+/PJLZGZm4oMPPgCAUo+10Ny5c/Haa69BrVZj8ODByMvLw7Fjx3D37l3MmDED33zzDVxcXNClSxfI5XJs2LABzs7OsLe3f+T79aCAgAB07NgRTz31FBYuXIiCggJMmzYNfn5+JV68VV4NGzZE165dceDAAUOw/e2339CgQQOMHTu22PEPHToUy5cvx+DBg9G+fXsMGjQIkydPxoIFC9C8eXMkJCTg9ddfx7hx49C4cWMAgLe3N9555x28+eabuHnzJkaNGoVGjRrh4sWLWLJkCXx9fTF9+vQS63ucYQZt27bF4MGD8fzzz2PJkiXIz8/HK6+8gieeeMIw9vvmzZvw9/fHypUriwwtuXjxIvbt24dt27aVuv1169ahoKAATz/9dImv79+/H82bN4eHh0eFj4GIqKokJxv/KXNKShL+/HMD7t5Ng0wmg6/vALz5pk+lDYGsVFU9tUJ54YHpfUrTp0+fIlNFCSHEL7/8Iuzs7Mq9H2On5jIVEyZMKDYdFgAxZcoUIUTFpubq0aOHmDlzpmGZB6eP2rRpk/D29hZ2dnbC2tpa9OzZU+zcudOwbGxsrOjUqZNQqVRlTs0VEREhPD09hVKpFA4ODmL06NGlHmNp00PFxsYKAOLw4cNCCCGioqJE7969haWlpbCzsxM9evQQy5YtMyxfODWXUqkUbdq0EX/++acAIKKiooQQ/z8118mTJ4vta/Xq1YZ669WrJ/r27Ss2btwohBBi2bJlwtPTU1hbWws7Ozvh7+8vTpw4Ua73q6JTcz3o22+/FU2bNi31/RNCiMWLF4uePXsannfs2FFMmzatxGXXrVsnlEqluH37thDi/rRar732mvDw8BCWlpaiZcuW4p133hGZmZklrtu3b19ha2srrK2tRadOncTHH39cZVNzCSHEnTt3xJNPPilsbGyEnZ2dmDRpUpHaCs/rnj17iqw3a9Ys4ebmJnQ6Xanb7tWrlxg/fnyprw8aNEjMnz+/1NdN+WcLEZmmrCwh7g82uP//8tLr9eLIkSPik08+ER999JH45ptvxPXr16uu0FIYMzWXTIjHuGKjEslkMmzatKnMu1i9++672LZtW5GPZsePH4+0tDRERUWVaz8ZGRlQq9VIT0+HnZ1dkddyc3MNV3AbO76yttFoNGjcuDEWLFhQ5lRFtcHBgwfh6+uLixcv1vqetZycHLRu3Rrr1q1Dr169pC6nVjh79iwGDBiACxcuGMYOP4w/W4ioumk096f5BO5fyF7entk7d+5g8eLF0Ov1aNWqFUaOHFktN/V5WFl57WGSDjPIysrCxYsXDc+vXLmCuLg41K9fH02aNMGsWbNw8+ZNrFy5EgDw4osv4ocffsA777yDyZMnY/fu3Vi/fj22bt0q1SHUGidPnsT58+fRo0cPpKen4+OPPwYAjBw5UuLKKt+mTZtgY2ODli1b4uLFi5g+fTp8fHxqfZAF7o/bXblyJVJTU6UupdZITEzEypUrSw2yRESmpEGDBggMDIROp0PPnj0fOQSvJpA0zB47dgz9+/c3PC8c2zphwgSEhYUhMTER169fN7zerFkzbN26FW+88Qa+++47uLq64ueff+Ycs5Xk66+/RkJCApRKJby8vLB///4Sx7qauszMTLz77ru4fv06HBwcEBAQYJjurS7o16+f1CXUKg/PsEJEZEqEEDhy5AiaNm1qmP3FmBsY1QQ1ZphBdeEwAyKqbvzZQkTVrTzDDHJychAZGYnz58+jfv36mDp1apGpG6VkMsMMiIiIiKj6/fvvvwgPD0d6ejoUCgW8vb0rPO+51BhmS1DHOquJqIrxZwoR1RRCCMTGxmLXrl3Q6/WoV68eQkNDy7yFfU3HMPuAwr9IsrOzK3R3KiKikhTe5tZUez2IqHbQarWIiIgw3BSpffv2CAoKgkqlkriyx8Mw+wCFQgF7e3vDbTGtrKxM4io+IqqZhBDIzs5GSkoK7O3tDbc+JiKSgrm5OQoKCqBQKDB48GB4eXnVipzDMPuQwiv5Hr7POxFRRdnb2xt+thARVSeZTKCgQAfADDKZDKNGjUJWVlat+pnEMPsQmUwGFxcXODo6Ij8/X+pyiMjEmZubs0eWiCRhba3BqFGbsHu3GqNGBQEAbGxsYFM4zUEtwTBbCoVCwV9AREREZJJu3LiKF1+MgK1tFs6dM0O/fr6oV6+e1GVVCYZZIiIiolpCr9dj//792Lt3L2xtBW7fdsBbb42ptUEWYJglIiIyWUIA/5ssgwgaTRa2bt2I69evAABOnvTEtm1D8MUXNeNGCFWFYZaIiMgECQH4+gKHDkldCdUEMpnASy+thKPjbWi15ti6dRhOneosdVnVgmGWiIjIBGVnM8jS/xNChujoAPj770Z4eChSUx0AAD4+gJWVxMVVMYZZIiIiE5ecDFhbS10FVbesrEzcvZsGN7em/2tpBb2+Bb7+Wm5YxsoKqAVTyZaJYZaIiMjEWVszzNY1Fy9exKZNm6DX6zF16lTY29v/7xV5WavVSgyzRERERCZCr9dj9+7dOHjwIID7N3vS6/USVyUthlkiIiIiE5Ceno6IiAjcuHEDANCtWzcEBgbCzKxux7m6ffREREREJuDChQvYvHkzcnJyoFKpEBQUhPbt20tdVo3AMEtERERUw/3zzz/IyclBo0aNEBoaWqtvgmAshlkiIiKiGi4wMBD29vbw9vau88MKHlb3LnkjIiIiquHOnz+P9evXGy7uMjMzg4+PD4NsCfiOEBEREdUQBQUFiI6OxpEjRwAAJ0+ehJeXl8RV1WwMs0REREQ1QFpaGsLDw5GYmAgA6NWrFzw9PaUtygQwzBIRERFJ7OzZs/jzzz+Rl5cHS0tLBAcHo1WrVlKXZRIYZomIiIgktH//fuzevRsA4ObmhpCQEKjVaomrMh28AIyIiIhIQq1atYK5uTl8fX0xceJEBlkjsWeWiIioDEIA2dlSV1GcRiN1BfQ47ty5gwYNGgAAnJyc8Oqrr8LW1lbiqkwTwywREVEphAB8fYFDh6SuhGqL/Px8REVFIS4uDpMmTYKrqysAMMg+BoZZIiKiUmRn1/wg6+MDWFlJXQWVx+3btxEeHo6UlBQAwM2bNw1hliqOYZaIiKgckpMBa2upqyjOygqQyaSugh4lLi4O27ZtQ35+PqytrTF69Gg0b95c6rJqBYZZIiKicrC2rplhlmo2rVaLbdu24dSpUwCAZs2aYfTo0bCxsZG4stqDYZaIiIioipw5cwanTp2CTCZDv3794OvrC7mck0lVJoZZIiIioirSpUsX3Lx5Ex07doS7u7vU5dRK/NOAiIiIqJLk5eUhOjoaeXl5AACZTIagoCAG2SrEnlkiIiKiSpCUlITw8HDcuXMHGo0GwcHBUpdUJzDMEhERET0GIQSOHz+OqKgo6HQ62NnZoWvXrlKXVWcwzBIRERFVUG5uLrZs2YKzZ88CuH9r2pEjR8KKk/9WG4ZZIiIiogpISUnB2rVrcffuXcjlcgQEBKBnz56QceLfasUwS0RERFQBVlZW0Gq1UKvVCA0N5d28JMIwS0RERFRO+fn5MDc3BwDY2Njgqaeegr29PSwtLSWurO7i1FxERERE5fDvv/9i0aJFOHPmjKHNxcWFQVZiDLNEREREZRBCIDY2FitWrEB6ejoOHjwIIYTUZdH/cJgBERERUSmys7Pxxx9/4MKFCwCAdu3aISgoiBd51SAMs0REVCcJAWRnl72MRlM9tVDNdOPGDYSHhyMjIwMKhQKDBw+Gl5cXg2wNwzBLRER1jhCAry9w6JDUlVBNdffuXYSFhUGv16N+/foYM2YMnJ2dpS6LSsAwS0REdU52tnFB1scH4Bz4dUu9evXg7e2NrKwsDBs2DCqVSuqSqBQMs0REVKclJwPW1mUvY2UF8JPl2u/q1auoV68e1Go1ACAgIAAymYzDCmo4hlkiIqrTrK0fHWapdtPr9di/fz/27t2Lxo0bY+LEiVAoFJDLOemTKWCYJSIiojorKysLGzduxJUrVwAADRo0gF6vh0KhkLgyKi+GWSIiIqqTrly5goiICGg0Gpibm2Po0KHw9PSUuiwyEsMsERER1Sl6vR579+7Fvn37AACOjo4IDQ1Fw4YNJa6MKoJhloiIiOoUvV6PhIQEAECXLl0wZMgQmJubS1wVVRTDLBEREdUpZmZmCA0NRWJiIjp27Ch1OfSYGGaJiIioVtPr9di9ezeUSiX69u0LAHBwcICDg4PElVFlYJglIiKiWis9PR0RERG4ceMGZDIZ2rdvjwYNGkhdFlUihlkiIiKqlS5cuIDNmzcjJycHKpUKQUFBDLK1EMMsERER1So6nQ67du1CbGwsAMDFxQWhoaGoX7++xJVRVWCYJSIiolpDCIFVq1bh6tWrAIAePXpg4MCBMDNj5KmteGaJiIio1igcF5uUlIQRI0agbdu2UpdEVYxhloiIiExaQUEBMjIyDMMIvLy80KZNG9jY2EhcGVUHhlkiIjIZQgDZ2Y+/HY3m8bdBNcPdu3exYcMGZGdnY+rUqbC0tIRMJmOQrUMYZomIyCQIAfj6AocOSV0J1RTnzp1DZGQk8vLyYGlpiTt37sDV1VXqsqiaMcwSEZFJyM6u/CDr4wNYWVXuNqnqFRQUYMeOHTh27BgAwM3NDSEhIVCr1RJXRlJgmCUiIpOTnAxYWz/+dqysAJns8bdD1efOnTsIDw9HUlISAMDHxwf9+/eHQqGQuDKSCsMsERGZHGvrygmzZHpiYmKQlJQEKysrjBo1Ci1atJC6JJIYwywRERGZjCFDhgAABg4cCDs7O4mroZpALnUBRERERKW5ffs29uzZAyEEAMDKygohISEMsmTAnlkiIiKqkU6dOoWtW7ciPz8f9evXR+fOnaUuiWoghlkiIiKqUbRaLbZv3464uDgAQLNmzeDh4SFtUVRjMcwSERFRjZGSkoINGzYgNTUVMpkMfn5+6NOnD+RyjoykkjHMEhERUY1w+vRpREZGoqCgADY2NggJCYG7u7vUZVENxzBLRERENYK1tTUKCgrg4eGBUaNGwZrzr1E5MMwSERGRZLRaLZRKJQCgefPmmDhxIpo0aQIZ72ZB5cQBKERERFTthBA4duwYvvvuO6SlpRnamzZtyiBLRmGYJSIiomqVl5eHiIgIbN26FdnZ2Th27JjUJZEJkzzMLlq0CO7u7rCwsIC3tzeOHDlS5vILFy5E69atYWlpCTc3N7zxxhvIzc2tpmqJiIjocdy6dQtLly7F2bNnIZfLMXDgQAwcOFDqssiESTpmdt26dZgxYwaWLFkCb29vLFy4EIGBgUhISICjo2Ox5desWYOZM2fil19+Qe/evXHhwgVMnDgRMpkM33zzjQRHQEREROUhhMCRI0cQHR0NnU4HtVqN0NBQuLq6Sl0amTiZKLw/nAS8vb3RvXt3/PDDDwAAvV4PNzc3vPrqq5g5c2ax5V955RXEx8dj165dhrY333wThw8fxoEDB8q1z4yMDKjVaqSnp/NWeEQ1kBBAdrbUVVBNpNEATk73/5+VBfBCd9Ny8uRJREZGAgDatGmDESNGwNLSUuKqqKYyJq9J1jOr1Wpx/PhxzJo1y9Aml8sREBCA2NjYEtfp3bs3Vq1ahSNHjqBHjx64fPkytm3bhmeeeabU/eTl5SEvL8/wPCMjo/IOgogqlRCAry9w6JDUlRBRZevUqRPi4uLQrl079OjRgxd5UaWRLMympqZCp9PBqfDP7P9xcnLC+fPnS1xn/PjxSE1Nha+vL4QQKCgowIsvvoj33nuv1P3Mnz8fc+fOrdTaiahqZGczyNKj+fgAVlZSV0GPIoTA6dOn0b59eygUCigUCsPQQKLKZFLzzMbExGDevHlYvHgxvL29cfHiRUyfPh2ffPIJPvzwwxLXmTVrFmbMmGF4npGRATc3t+oqmYgqKDmZHyNTyaysAOahmi0nJwebN2/GhQsXkJKSgoCAAABgkKUqIVmYdXBwgEKhQHJycpH25ORkODs7l7jOhx9+iGeeeQbPPfccAKBjx47QaDR44YUX8P7775d432aVSgWVSlX5B0BEVcrammGWyBTduHED4eHhyMjIgEKhgFqtlrokquUkm5pLqVTCy8uryMVcer0eu3btQq9evUpcJzs7u1hgVSgUAO5/nEFERETSEELgwIEDWLFiBTIyMlC/fn0899xz6N69u9SlUS0n6TCDGTNmYMKECejWrRt69OiBhQsXQqPRYNKkSQCAZ599Fo0bN8b8+fMBAEFBQfjmm2/QpUsXwzCDDz/8EEFBQYZQS0RERNVLo9Fg8+bNuHjxIgCgQ4cOGD58OD8ZpWohaZgdN24cbt++jdmzZyMpKQmenp6IiooyXBR2/fr1Ij2xH3zwAWQyGT744APcvHkTDRs2RFBQED777DOpDoGIiKjOy8nJwbVr12BmZoYhQ4agS5cuHB9L1UbSeWalwHlmiWoujQawsbn/f84jSmRazp8/j3r16hWbpYioIozJa5LfzpaIiIhMS1ZWFlatWoVr164Z2tq0acMgS5JgmCUiIqJyu3z5MpYsWYJLly4hMjISer1e6pKojjOpeWaJiIhIGnq9Hnv37sW+ffsAAA0bNsSYMWNKnBaTqDoxzBIREVGZMjMzsXHjRly9ehUA0KVLFwwZMgTm5ubSFkYEhlkiIiIqQ3p6OpYtW4bs7GyYm5tj+PDh6NSpk9RlERkwzBIREVGp7Ozs0KxZM6SmpmLMmDFo0KCB1CURFcEwS0REREVkZGRAqVTCwsICMpkMQUFBkMvlHFZANRJHbRMREZHBhQsXsGTJEkRGRhpuFa9SqRhkqcZizyxRHSQEkJ0tdRXFaTRSV0BUd+l0OuzatQuxsbEAgHv37iEvLw8WFhYSV0ZUNoZZojpGCMDXFzh0SOpKiKimuHfvHiIiIvDvv/8CAHr06IGBAwfCzIwxgWo+fpUS1THZ2TU/yPr4AFZWUldBVDecP38ef/zxB3Jzc6FSqTBy5Ei0bdtW6rKIyo1hlqgOS04GrK2lrqI4KytAJpO6CqLaLz8/H9u3b0dubi4aN26MkJAQ1KtXT+qyiIzCMEtUh1lb18wwS0TVw9zcHCEhITh//jz8/f2hUCikLonIaAyzREREdci5c+dQUFBguPFBkyZN0KRJE4mrIqo4hlkiIqI6oKCgADt27MCxY8dgZmaGxo0b8wYIVCswzBIREdVyd+7cQXh4OJKSkgAA3t7esLe3l7YookrCMEtERFSLnTlzBn/++Se0Wi2srKwQHByMli1bSl0WUaVhmCUiIqqFhBDYunUrjh8/DuD+2NiQkBDY2dlJXBlR5WKYJSIiqoVkMhms/jdhc58+fdCvXz/I5byLPdU+DLNERES1iFarhVKpBAD069cPLVu2hJubm8RVEVUd/olGRERUC2i1Wvzxxx8ICwtDQUEBAEAulzPIUq3HnlkiIiITl5KSgvDwcNy+fRsymQxXr15FixYtpC6LqFowzBIREZkoIQTi4uKwbds2FBQUwMbGBiEhIXB3d5e6NKJqwzBLRERkgvLy8rB161acPn0aAODh4YFRo0bBmveopjqGYZZMmhBAdrbUVZgWjUbqCoioMmzZsgVnzpyBTCZD//794evrC5lMJnVZRNWOYZZMlhCAry9w6JDUlRARVb8BAwYgOTkZw4cPR5MmTaQuh0gynM2ATFZ2NoPs4/DxAf43BSURmYC8vDycPXvW8LxevXp46aWXGGSpzmPPLNUKyckAh4kZx8oK4CeSRKYhMTERGzZswN27d6FSqQwzFXBYARHDLNUS1tYMs0RU+wghcPToUfz111/Q6XRQq9WwsLCQuiyiGoVhloiIqAbKzc1FZGQk4uPjAQCtW7fGyJEjYWlpKXFlRDULwywREVENc/PmTYSHh+PevXuQy+UYOHAgvL29OayAqAQMs0RERDVMamoq7t27B3t7e4SGhqJx48ZSl0RUYzHMEhER1QBCCEPPa+fOnaHVatGxY0eOkSV6BE7NRUREJLEbN27gl19+QfYDd4Hp3r07gyxROTDMEhERSUQIgYMHD2LFihX4999/sXv3bqlLIjI5HGZAREQkAY1Gg82bN+PixYsAgA4dOmDgwIESV0VkehhmiYiIqtm1a9cQERGBzMxMmJmZYfDgwejatStnKyCqAIZZIiKianT+/HmsX78eQgg0aNAAY8aMgZOTk9RlEZkshlkiIqJq5O7uDnt7e7i5uWHYsGFQKpVSl0Rk0hhmiYiIqlhycjIcHR0hk8lgYWGB5557DpaWlhxWQFQJOJsBERFRFdHr9YiJicGSJUtw7NgxQ7uVlRWDLFElYc8sERFRFcjMzMTGjRtx9epVAEBKSoq0BRHVUgyzRERElezSpUvYtGkTNBoNzM3NMXz4cHTq1EnqsohqJYZZIiKiSlI4rGD//v0AACcnJ4SGhsLBwUHiyohqL4ZZIiKiSpKcnIwDBw4AALy8vBAYGAhzc3OJqyKq3RhmiYiIKomLiwsGDhwIW1tbdOjQQepyiOoEhlkiIqIK0ul0iImJQadOndCwYUMAQK9evSSuiqhu4dRcREREFZCeno6wsDAcOHAA4eHh0Ol0UpdEVCexZ5aIiMhICQkJ2Lx5M3Jzc6FSqeDn5weFQiF1WUR1EsMsERFROel0OkRHR+Pw4cMAgEaNGiE0NBT16tWTuDKiuothloiIqBw0Gg3WrFmDW7duAQB69uyJgIAA9sgSSYxhloiIqBwsLS1hZmYGCwsLBAcHo3Xr1lKXRERgmCUiIipVQUEBZDIZFAoF5HI5QkJCoNfrYW9vL3VpRPQ/nM2AiIioBGlpaVi+fDmio6MNbXZ2dgyyRDUMe2aJiIgecubMGfz555/QarXIyMhA3759YWVlJXVZRFQChlkiIqL/yc/PR1RUFE6cOAEAaNKkCUJCQhhkiWowhlkiIiIAqamp2LBhA1JSUgAAffr0Qb9+/SCXc0QeUU3GMEtERHVeQUEBVq5ciczMTFhbW2PUqFHw8PCQuiwiKofHCrO5ubmwsLCorFqoDhACyM6unG1pNJWzHSIiMzMzBAYG4tixYxg9ejRsbW2lLomIysnoMKvX6/HZZ59hyZIlSE5OxoULF9C8eXN8+OGHcHd3x5QpU6qiTqoFhAB8fYFDh6SuhIgISElJQU5ODpo2bQoAaN++Pdq1aweZTCZxZURkDKMHAn366acICwvDl19+CaVSaWjv0KEDfv7550otjmqX7OyqCbI+PgCvzSCi8hJC4OTJk/jpp5+wfv16ZGZmGl5jkCUyPUb3zK5cuRLLli2Dv78/XnzxRUN7586dcf78+Uotjmqv5GTA2rpytmVlBfD3DxGVh1arxdatW/H3338DuD9bAS/wIjJtRofZmzdvokWLFsXa9Xo98vPzK6Uoqv2srSsvzBIRlUdycjI2bNiAO3fuQCaToX///vD19WVvLJGJMzrMtmvXDvv37zeMMSoUHh6OLl26VFphRERElUEIgRMnTiAqKgoFBQWwtbVFSEhIsd9jRGSajA6zs2fPxoQJE3Dz5k3o9Xps3LgRCQkJWLlyJbZs2VIVNRIREVWYTCbDjRs3UFBQgBYtWmDUqFG8CQJRLSITQghjV9q/fz8+/vhjnDp1CllZWejatStmz56NQYMGVUWNlSojIwNqtRrp6emws7OTupw6RaMBbGzu/z8ri8MMiKhqCSEMQwi0Wi3+/vtveHl5cVgBkQkwJq9VKMyaMoZZ6TDMElF1EELg6NGjuHr1KsaMGcPwSmSCjMlrRl/C2bx5c9y5c6dY+71799C8eXNjN0dERFRpcnNzER4eju3btyM+Ph7x8fFSl0REVczoMbNXr16FTqcr1p6Xl4ebN29WSlFERETGunnzJsLDw3Hv3j3I5XIMHDgQbdu2lbosIqpi5Q6zkZGRhv/v2LEDarXa8Fyn02HXrl1wd3ev1OKIiIgeRQiBw4cPIzo6Gnq9Hvb29ggNDUXjxo2lLo2IqkG5w2xwcDCA+1eFTpgwochr5ubmcHd3x4IFCyq1OCIiokfZvn07jh49CgBo27YtRowYAQsLC4mrIqLqUu4wq9frAQDNmjXD0aNH4eDgUGVFERERlVfnzp1x6tQp+Pv7o3v37rzgi6iO4WwGVG04mwERVQYhBJKTk+Hs7Gxoy8nJgaWlpYRVEVFlqtLZDABAo9Fg27ZtWLJkCb7//vsiD2MtWrQI7u7usLCwgLe3N44cOVLm8vfu3cPLL78MFxcXqFQqtGrVCtu2bavIYRARkYnJzs7G77//jp9//hlJSUmGdgZZorrL6NkMTp48iaFDhyI7OxsajQb169dHamoqrKys4OjoiNdee63c21q3bh1mzJiBJUuWwNvbGwsXLkRgYCASEhLg6OhYbHmtVouBAwfC0dER4eHhaNy4Ma5duwZ7e3tjD4OIiEzMtWvXEBERgczMTCgUCqSmphbpnSWiusnoYQb9+vVDq1atsGTJEqjVapw6dQrm5uZ4+umnMX36dIwePbrc2/L29kb37t3xww8/ALg/LtfNzQ2vvvoqZs6cWWz5JUuW4KuvvsL58+dhbm5uTNkGHGYgHQ4zIKKKEELgwIED2LNnD4QQaNCgAcaMGQMnJyepSyOiKlKlwwzi4uLw5ptvQi6XQ6FQIC8vD25ubvjyyy/x3nvvlXs7Wq0Wx48fR0BAwP8XI5cjICAAsbGxJa4TGRmJXr164eWXX4aTkxM6dOiAefPmlTjvbaG8vDxkZGQUeRARkWnQaDRYvXo1du/eDSEEOnXqhBdeeIFBlogMjA6z5ubmkMvvr+bo6Ijr168DANRqNW7cuFHu7aSmpkKn0xX7geTk5FRkHNSDLl++jPDwcOh0Omzbtg0ffvghFixYgE8//bTU/cyfPx9qtdrwcHNzK3eNREQkrb///huXLl2CmZkZRowYgeDgYCiVSqnLIqIaxOgxs126dMHRo0fRsmVL+Pn5Yfbs2UhNTcVvv/2GDh06VEWNBnq9Ho6Ojli2bBkUCgW8vLxw8+ZNfPXVV5gzZ06J68yaNQszZswwPM/IyGCgJSIyET179kRaWhq6d+9e4rUURERG98zOmzcPLi4uAIDPPvsM9erVw0svvYTbt29j6dKl5d6Og4MDFAoFkpOTi7Q/PN3Kg1xcXNCqVSsoFApDW9u2bZGUlAStVlviOiqVCnZ2dkUeRERUM2VmZmLLli3Iz88HcP9GPcOGDWOQJaJSGd0z261bN8P/HR0dERUVVaEdK5VKeHl5YdeuXYa7i+n1euzatQuvvPJKiev4+PhgzZo10Ov1hqEOFy5cgIuLCz92IiIycZcuXcKmTZug0Wggl8sxdOhQqUsiIhNQoXlmS3LixAkMHz7cqHVmzJiBn376Cb/++ivi4+Px0ksvQaPRYNKkSQCAZ599FrNmzTIs/9JLLyEtLQ3Tp0/HhQsXsHXrVsybNw8vv/xyZR0GERFVM71ej927d2PVqlXQaDRwdHREjx49pC6LiEyEUT2zO3bsQHR0NJRKJZ577jk0b94c58+fx8yZM/Hnn38iMDDQqJ2PGzcOt2/fxuzZs5GUlARPT09ERUUZLgq7fv26oQcWANzc3LBjxw688cYb6NSpExo3bozp06fj3XffNWq/RERUM2RkZCAiIsJwMXHXrl0xePDgCk+/SER1T7nnmV2+fDmef/551K9fH3fv3kWDBg3wzTff4NVXX8W4ceMwffp0tG3btqrrfWycZ1Y6nGeWiB50/fp1rFu3DtnZ2VAqlQgKCqryC4mJyDQYk9fK3TP73Xff4YsvvsDbb7+NiIgIjBkzBosXL8bp06fh6ur62EUTEVHdolarIYSAs7MzQkND0aBBA6lLIiITVO6eWWtra5w9exbu7u4QQkClUmHPnj3w8fGp6horFXtmpcOeWSLKzc2FhYWF4XlSUhIcHBxgZmb09chEVItVyR3AcnJyYGVlBeD+VCkqlcowRRcREdGjJCQk4Pvvv0dCQoKhzdnZmUGWiB6LUT9Bfv75Z9j8r2utoKAAYWFhcHBwKLLMa6+9VnnVERGRydPpdNi5cyf++9//AgCOHj2K1q1bS1wVEdUW5R5m4O7uDplMVvbGZDJcvny5UgqrKhxmIB0OMyCqe+7evYuIiAjcvHkTAODt7Y2BAwcWufkNEdHDquQCsKtXrz5uXUREVIfEx8fjjz/+QF5eHiwsLDBy5Ei0adNG6rKIqJbhQCUiIqp0iYmJWL9+PQDA1dUVISEhsLe3l7YoIqqVGGaJiKjSubi4oFu3blAqlRgwYACHFRBRlWGYJSKiSnHu3Dk0adLEcKHw0KFDH3mtBRHR4yr31FxEREQlyc/Px5YtW7BhwwZs3LgRer0eABhkiahasGeWiIgqLDU1FeHh4UhOTgYANG7cWOKKiKiuqVCYvXTpElasWIFLly7hu+++g6OjI7Zv344mTZqgffv2lV0jERHVQH///Te2bNmC/Px8WFlZYfTo0fDw8JC6LCKqY4weZrB371507NgRhw8fxsaNG5GVlQUAOHXqFObMmVPpBRIRUc2Sn5+PyMhIbNq0Cfn5+XB3d8eLL77IIEtEkjA6zM6cOROffvopoqOjoVQqDe0DBgww3N2FiIhqLyEEbty4AQDw8/PDM888A1tbW4mrIqK6yuhhBqdPn8aaNWuKtTs6OiI1NbVSiiIioppHCAGZTAalUonQ0FBoNBo0b95c6rKIqI4zumfW3t4eiYmJxdpPnjzJgf9ERLWQVqvF5s2bi3z65uTkxCBLRDWC0WH2iSeewLvvvoukpCTIZDLo9XocPHgQb731Fp599tmqqJGIiCSSnJyMn376CadOncLu3bsN10kQEdUURg8zmDdvHl5++WW4ublBp9OhXbt20Ol0GD9+PD744IOqqJGIiKqZEAInTpxAVFQUCgoKYGtri5CQEMMNEYiIagqZEEJUZMXr16/jzJkzyMrKQpcuXdCyZcvKrq1KZGRkQK1WIz09HXZ2dlKXU6doNEDh78GsLMDaWtp6iKhkeXl52LJlC86cOQMAaNGiBYKDg2HNb1oiqibG5DWje2YPHDgAX19fNGnSBE2aNKlwkUREVPPodDosX74ct2/fhkwmg7+/P3r37s27eRFRjWX0mNkBAwagWbNmeO+993Du3LmqqImIiCSiUCjQpUsX2NnZYdKkSfDx8WGQJaIazegwe+vWLbz55pvYu3cvOnToAE9PT3z11Vf4999/q6I+IiKqYrm5ubhz547hec+ePfHSSy/Bzc1NwqqIiMrH6DDr4OCAV155BQcPHsSlS5cwZswY/Prrr3B3d8eAAQOqokYiIqoit27dwtKlS/H7778jLy8PACCTyWBhYSFxZURE5WP0mNkHNWvWDDNnzkTnzp3x4YcfYu/evZVVFxERVSEhBA4fPozo6Gjo9XrY29sjMzMTKpVK6tKIiIxS4TB78OBBrF69GuHh4cjNzcXIkSMxf/78yqyNiIiqQE5ODiIjI3H+/HkAQJs2bTBy5Ej2xhKRSTI6zM6aNQtr167FrVu3MHDgQHz33XcYOXIkrKysqqI+IiKqRP/++y/Cw8ORnp4OhUKBQYMGoXv37rzIi4hMltFhdt++fXj77bcxduxYODg4VEVNRERURfbu3Yv09HTUq1cPoaGhaNSokdQlERE9FqPD7MGDB6uiDiIiqgYjR45ETEwMBg4cyPGxRFQrlCvMRkZGYsiQITA3N0dkZGSZy44YMaJSCiMiosd3/fp1XLp0Cf379wcA2NjYYPjw4RJXRURUecoVZoODg5GUlARHR0cEBweXupxMJoNOp6us2oiIqIKEEDhw4AD27NkDIQRcXFzQpk0bqcsiIqp05Qqzer2+xP8TEVHNo9FosGnTJly6dAkA0KlTJzRv3lziqoiIqobRN01YuXKlYWLtB2m1WqxcubJSiiIiooq5evUqlixZgkuXLsHMzAwjRoxAcHAwlEql1KUREVUJmRBCGLOCQqFAYmIiHB0di7TfuXMHjo6ONX6YQUZGBtRqNdLT02FnZyd1OXWKRgPY2Nz/f1YWYG0tbT1EtU1sbCyio6MhhICDgwPGjBlT7Gc1EZEpMCavGT2bgRCixPkI//33X6jVamM3R0RElaR+/foQQsDT0xNDhgxhbywR1QnlDrNdunSBTCaDTCaDv78/zMz+f1WdTocrV65g8ODBVVIkERGVLDc313DnrtatW+P555/n3LFEVKeUO8wWzmIQFxeHwMBA2BR+XgxAqVTC3d0dISEhlV4gEREVp9frERMTg+PHj+OFF14wfDLGIEtEdU25w+ycOXMAAO7u7hg3bhzv4U1EJJGMjAxs3LgR165dAwCcO3cOvXr1krgqIiJpGD1mdsKECVVRBxERlcPFixexadMmZGdnQ6lUIigoCB06dJC6LCIiyZQrzNavXx8XLlyAg4MD6tWrV+IFYIXS0tIqrTgiIrpPp9Nhz549hluKOzs7IzQ0FA0aNJC4MiIiaZUrzH777bewtbU1/L+sMEtERJXv8OHDhiDbvXt3DBo0qMiFuEREdZXR88yaOs4zKx3OM0tUcfn5+Vi1ahW8vb3Rrl07qcshIqpSxuQ1o+8AduLECZw+fdrw/I8//kBwcDDee+89aLVa46slIqJidDodjh07ZriFuLm5OSZOnMggS0T0EKPD7NSpU3HhwgUAwOXLlzFu3DhYWVlhw4YNeOeddyq9QCKiuubevXtYsWIFtm7div379xvaOcSLiKg4o8PshQsX4OnpCQDYsGED/Pz8sGbNGoSFhSEiIqKy6yMiqlPi4+OxdOlS3Lx5ExYWFnBycpK6JCKiGq1Ct7Mt/Nhr586dGD58OADAzc0NqamplVsdEVEdUVBQgOjoaBw5cgQA4OrqipCQENjb20tbGBFRDWd0mO3WrRs+/fRTBAQEYO/evfjxxx8BAFeuXGEPAhFRBaSlpSE8PByJiYkAgF69esHf3x8KhULiyoiIaj6jw+zChQvx1FNPYfPmzXj//ffRokULAEB4eDh69+5d6QUSEdV2Wq0WKSkpsLS0RHBwMFq1aiV1SUREJqPSpubKzc2FQqGAubl5ZWyuynBqLulwai6i/yeEKHJB1/nz5+Hi4gK1Wi1hVURENYMxea3CM24fP34c8fHxAIB27dqha9euFd0UEVGdcufOHWzcuBFDhw5F48aNAQBt2rSRuCoiItNkdJhNSUnBuHHjsHfvXsOFCffu3UP//v2xdu1aNGzYsLJrJCKqNU6fPo0tW7ZAq9Vi+/btmDJlCqfcIiJ6DEZPzfXqq68iKysLZ8+eRVpaGtLS0nDmzBlkZGTgtddeq4oaiYhMXn5+PiIjI7Fx40ZotVq4u7tj3LhxDLJERI/J6J7ZqKgo7Ny5E23btjW0tWvXDosWLcKgQYMqtTgiotrg9u3bCA8PR0pKCgDAz88Pffv2hVxudH8CERE9xOgwq9frS7zIy9zc3DD/LBER3ZeSkoKff/4Z+fn5sLa2RkhICJo1ayZ1WUREtYbR3QIDBgzA9OnTcevWLUPbzZs38cYbb8Df379SiyMiMnUNGzZEs2bN0KxZM7z44osMskRElczontkffvgBI0aMgLu7O9zc3AAAN27cQIcOHbBq1apKL5CIyNSkpKTA3t4eSqUSMpkMISEhMDMz47ACIqIqYHSYdXNzw4kTJ7Br1y7D1Fxt27ZFQEBApRdHRGRKhBA4efIktm/fjnbt2iE4OBgymQxKpVLq0oiIai2jwuy6desQGRkJrVYLf39/vPrqq1VVFxGRScnLy8PWrVtx+vRpAEB2djZ0Oh3MzCo8nTcREZVDuX/K/vjjj3j55ZfRsmVLWFpaYuPGjbh06RK++uqrqqyPiKjGS0pKwoYNG5CWlgaZTAZ/f3/07t2b024REVWDct/Otn379hg7dizmzJkDAFi1ahWmTp0KjUZTpQVWNt7O1nhCANnZj78djQZwcrr/f97OlmoDIQSOHTuGHTt2QKfTwc7ODqGhoYbrCYiIqGKMyWvlDrOWlpaIj4+Hu7s7gPtTdFlaWuLq1atwcXF57KKrC8OscYQAfH2BQ4cqd7sMs1Qb5OTkYNGiRdBoNGjVqhVGjhwJKysrqcsiIjJ5xuS1cg8zyMvLg/UD6UMul0OpVCInJ6filVKNl51d+UHWxwfg73uqDSwtLTF69GgkJyejZ8+eHFZARCQBo65M+PDDD4v0Omi1Wnz22WdQq9WGtm+++abyqqMaJTm5cnpTrawA/s4nUySEwJEjR2Bra4t27doBAJo3b47mzZtLXBkRUd1V7jDbt29fJCQkFGnr3bs3Ll++bHjOXonazdqaQwOo7srJyUFkZCTOnz8PpVIJV1dXDlUiIqoByh1mY2JiqrAMIqKa699//0V4eDjS09OhUCjg7+8PW1tbqcsiIiJU4KYJRER1hRACsbGx2LVrF/R6PerVq4fQ0FA0atRI6tKIiOh/GGaJiEqg1+uxbt06XLhwAcD96QmDgoKgUqkkroyIiB7EMEtEVAK5XI769etDoVBg8ODB8PLy4nUBREQ1EMMsEdH/CCGQl5cHCwsLAEBAQAC6du2Khg0bSlwZERGVRi51AURENYFGo8GaNWuwZs0a6HQ6AIBCoWCQJSKq4SoUZvfv34+nn34avXr1ws2bNwEAv/32Gw4cOFCpxRERVYerV69i6dKluHjxIhITE5GUlCR1SUREVE5Gh9mIiAgEBgbC0tISJ0+eRF5eHgAgPT0d8+bNq/QCiYiqil6vx969e7Fy5UpkZmbCwcEBzz//PBo3bix1aUREVE5Gh9lPP/0US5YswU8//QRzc3NDu4+PD06cOFGpxRERVZWsrCysWrUKMTExEELA09MTzz//PBwdHaUujYiIjGD0BWAJCQno27dvsXa1Wo179+5VRk1ERFVu06ZNuHLlCszNzTFs2DB07txZ6pKIiKgCjO6ZdXZ2xsWLF4u1HzhwoML3J1+0aBHc3d1hYWEBb29vHDlypFzrrV27FjKZDMHBwRXaLxHVXUOGDIGrqyteeOEFBlkiIhNmdJh9/vnnMX36dBw+fBgymQy3bt3C6tWr8dZbb+Gll14yuoB169ZhxowZmDNnDk6cOIHOnTsjMDAQKSkpZa539epVvPXWW+jTp4/R+ySiuiczMxOnT582PHdwcMDkyZPh4OAgYVVERPS4jB5mMHPmTOj1evj7+yM7Oxt9+/aFSqXCW2+9hVdffdXoAr755hs8//zzmDRpEgBgyZIl2Lp1K3755RfMnDmzxHV0Oh2eeuopzJ07F/v37+fwBiIq08WLF7Fp0ybk5OTAzs4OTZs2BQDeBIGIqBYwOszKZDK8//77ePvtt3Hx4kVkZWWhXbt2sLGxMXrnWq0Wx48fx6xZswxtcrkcAQEBiI2NLXW9jz/+GI6OjpgyZQr2799f5j7y8vIMMy4AQEZGhtF1EpFp0uv12L17Nw4ePAjg/jCpivysIiKimqvCdwBTKpVo167dY+08NTUVOp0OTk5ORdqdnJxw/vz5Etc5cOAAli9fjri4uHLtY/78+Zg7d+5j1UlEpic9PR0RERG4ceMGAKBbt24IDAyEmRlvfEhEVJsY/VO9f//+ZX40t3v37scqqCyZmZl45pln8NNPP5V7nNusWbMwY8YMw/OMjAy4ublVVYlEVANcuHABmzdvRk5ODlQqFYKCgtC+fXupyyIioipgdJj19PQs8jw/Px9xcXE4c+YMJkyYYNS2HBwcoFAokJycXKQ9OTkZzs7OxZa/dOkSrl69iqCgIEObXq8HAJiZmSEhIQEeHh5F1lGpVFCpVEbVRUSmLT09HTk5OXBxcUFoaCjq168vdUlERFRFjA6z3377bYntH330EbKysozallKphJeXF3bt2mWYXkuv12PXrl145ZVXii3fpk2bIlcjA8AHH3yAzMxMfPfdd+xxJarDhBCGT426desGc3NzdOjQgcMKiIhquUr7Kf/000+jR48e+Prrr41ab8aMGZgwYQK6deuGHj16YOHChdBoNIbZDZ599lk0btwY8+fPh4WFBTp06FBkfXt7ewAo1k5Edcf58+exb98+PPvss7CwsIBMJiv2KRIREdVOlRZmY2NjYWFhYfR648aNw+3btzF79mwkJSXB09MTUVFRhovCrl+/Drnc6OlwiagOKCgowM6dO3H48GEAwKFDhzBgwACJqyIiouokE0IIY1YYPXp0kedCCCQmJuLYsWP48MMPMWfOnEotsLJlZGRArVYjPT0ddnZ2UpdT42k0QOFMRllZgLW1tPUQFUpLS0N4eDgSExMBAL169YK/vz8UCoXElRER0eMyJq8Z3TOrVquLPJfL5WjdujU+/vhjDBo0yNjNEREZ7ezZs/jzzz+Rl5cHS0tLBAcHo1WrVlKXRUREEjAqzOp0OkyaNAkdO3ZEvXr1qqomIqJSHT9+HFu2bAEAuLm5ITQ0lJ+yEBHVYUYNRlUoFBg0aBBvH0tEkmnbti3s7Ozg6+uLiRMnMsgSEdVxRl9Z1aFDB1y+fLkqaiEiKlHhXbwAwMrKCtOmTYO/vz8vDiUiIuPD7Keffoq33noLW7ZsQWJiIjIyMoo8iIgqS35+PiIjI/HLL78UuYU1b4RCRESFyj1m9uOPP8abb76JoUOHAgBGjBhR5La2hROW63S6yq+SiOqc27dvIzw8HCkpKQDu386aiIjoYeWemkuhUCAxMRHx8fFlLufn51cphVUVTs1lHE7NRVI4deoUtm7divz8fFhbW2P06NFo3ry51GUREVE1qZKpuQozb00Pq0RkurRaLbZv324YUtC8eXOMGjUKNoV/URERET3EqKm5HhxWQERU2W7duoW4uDjIZDL069cPvr6+vMiLiIjKZFSYbdWq1SMDbVpa2mMVRER1l7u7OwYNGgQXFxe4u7tLXQ4REZkAo8Ls3Llzi90BjIioovLy8vDXX3/Bx8cH9evXB3D/trRERETlZVSYfeKJJ+Do6FhVtRBRHZKUlITw8HDcuXMHKSkpmDx5MocyERGR0codZvlLhogqgxACx48fR1RUFHQ6Hezs7DBw4ED+jCEiogoxejYDIqKKys3NxZYtW3D27FkA98fhjxw5ElZWVhJXRkREpqrcYVav11dlHURUy929exe//fYb7t69C7lcjoCAAPTs2ZM9skRE9FiMGjNLRFRRdnZ2sLS0hF6vR2hoKFxdXaUuiYiIagGGWSKqMrm5uVAqlZDL5VAoFBg7diyUSiUsLS2lLo2IiGoJzkZORFXi5s2bWLp0Kfbs2WNoU6vVDLJERFSpGGaJqFIJIRAbG4tffvkF9+7dw7lz56DVaqUui4iIaikOMyCiSpOTk4PNmzfjwoULAIB27dohKCgISqVS4sqIiKi2Ypglokpx48YNhIeHIyMjAwqFAoMHD4aXlxdnKyAioirFMEtEjy03NxerV69GXl4e6tevjzFjxsDZ2VnqsoiIqA5gmCWix2ZhYYHBgwfj8uXLGDZsGFQqldQlERFRHcEwS0QVcu3aNcjlcri5uQEAPD090blzZw4rICKiasUwS0RG0ev1OHDgAGJiYmBjY4MXX3zRcDtaBlkiIqpuDLNEVG5ZWVnYtGkTLl++DABo3rw5zMz4Y4SIiKTD30JEVC5XrlxBREQENBoNzM3NMXToUHh6ekpdFhER1XEMs0RUJiEEYmJisG/fPgCAo6MjQkND0bBhQ4krIyIiYpglonJITU0FAHTp0gVDhgyBubm5xBURERHdxzBLRCUSQkAmk0EmkyEoKAjt27dHu3btpC6LiIioCLnUBRBRzaLX67Fz506Eh4dDCAHg/jyyDLJERFQTsWeWiAzS09MRERGBGzduALg/l6y7u7u0RREREZWBYZaIAAAXLlzA5s2bkZOTA5VKhaCgIAZZIiKq8Rhmieo4nU6HXbt2ITY2FgDg4uKC0NBQ1K9fX+LKiIiIHo1hlqiOi4iIQHx8PACgR48eGDhwIG+EQEREJoO/sYjqOG9vb1y7dg1BQUFo06aN1OUQEREZhWGWqI4pKChAUlISXF1dAQBNmzbF9OnToVQqJa6MiIjIeJyai6gOuXv3Ln755ResXLkSt2/fNrQzyBIRkalizyxRHXHu3DlERkYiLy8PlpaWyMrK4i1piYjI5DHMEtVyBQUF2LFjB44dOwYAcHNzQ0hICNRqtcSVERERPT6GWaJa7M6dOwgPD0dSUhIAwMfHB/3794dCoZC4MiIiosrBMEtUi/39999ISkqClZUVRo0ahRYtWkhdEhERUaVimCWqxfz8/KDVatGrVy/Y2dlJXQ4REVGl42wGRLVIamoqNm/ejIKCAgCAXC5HYGAggywREdVa7JklqiVOnTqFrVu3Ij8/H3Z2dhgwYIDUJREREVU5hlkiE6fVarF9+3bExcUBAJo1a4YePXpIWxQREVE1YZglMmEpKSkIDw/H7du3IZPJ4Ofnhz59+kAu5wgiIiKqGxhmiUzU+fPnERERgYKCAtjY2CAkJATu7u5Sl0VERFStGGaJTJSjoyMUCgWaNm2KUaNGwdraWuqSiIiIqh3DLJEJ0Wg0htBav359TJkyBQ4ODpDJZBJXRkREJA0OrCMyAUIIHDt2DAsXLsSlS5cM7Q0bNmSQJSKiOo09s0Q1XG5uLrZs2YKzZ88CAM6cOQMPDw+JqyIiIqoZGGaJarBbt24hPDwcd+/ehVwuh7+/P3r16iV1WURERDUGwyxRDSSEwJEjRxAdHQ2dTge1Wo3Q0FC4urpKXRoREVGNwjBLVANduXIFUVFRAIA2bdpgxIgRsLS0lLgqIiKimodhlqgGat68Obp27QpHR0f06NGDF3kRERGVgmGWqAYonK2gffv2sLKyAgAEBQVJXBUREVHNx6m5iCSWnZ2NtWvXYtu2bdi8eTOEEFKXREREZDLYM1tLCQFkZz/+djSax98Gle7GjRsIDw9HRkYGFAoFWrZsKXVJREREJoVhthYSAvD1BQ4dkroSKo0QAgcPHsTu3bshhED9+vUxZswYODs7S10aERGRSWGYrYWysys/yPr4AP8bykmPKTs7G5s2bcLFixcBAB06dMDw4cOhUqkkroyIiMj0MMzWcsnJgLX142/HygrgBfWVQy6XIzU1FWZmZhgyZAi6dOnC2QqIiIgqiGG2lrO2rpwwS4+n8KIumUwGCwsLjB07FnK5HE5OThJXRkREZNo4mwFRFcvKysKqVatw7NgxQ5uLiwuDLBERUSVgzyxRFbpy5QoiIiKg0WiQmJiITp06cWwsERFRJWKYJaoCer0ee/fuxb59+wAADRs2xJgxYxhkiYiIKhnDLFEly8zMxMaNG3H16lUAQJcuXTBkyBCYm5tLWxgREVEtxDBLVIm0Wi2WLVuGrKwsmJubY/jw4ejUqZPUZREREdVaDLNElUipVKJ79+44d+4cxowZgwYNGkhdEhERUa3GMEv0mDIyMpCfn28Irr6+vujduzfMzPjtRUREVNU4NRfRY7hw4QKWLFmC9evXIz8/H8D9myIwyBIREVUP/sYlqgCdToddu3YhNjYWAGBvb4+cnBxe5EVERFTNGGaJjHTv3j1ERETg33//BQD06NEDAwcOZG8sERGRBGrEMINFixbB3d0dFhYW8Pb2xpEjR0pd9qeffkKfPn1Qr1491KtXDwEBAWUuT1SZzp8/j6VLl+Lff/+FSqXC2LFjMWTIEAZZIiIiiUgeZtetW4cZM2Zgzpw5OHHiBDp37ozAwECkpKSUuHxMTAyefPJJ7NmzB7GxsXBzc8OgQYNw8+bNaq6c6hohBGJjY5Gbm4tGjRph6tSpaNu2rdRlERER1WkyIYSQsgBvb290794dP/zwA4D7d05yc3PDq6++ipkzZz5yfZ1Oh3r16uGHH37As88++8jlMzIyoFarkZ6eDjs7u8euv7IIAWRnV862NBrAyen+/7OyAGvrytkuAenp6Th27Bj69esHhUIhdTlERES1kjF5TdLPRrVaLY4fP45Zs2YZ2uRyOQICAgwX1jxKdnY28vPzUb9+/RJfz8vLQ15enuF5RkbG4xVdBYQAfH2BQ4ekroQedu7cOSQnJ6N///4AALVaDX9/f4mrIiIiokKSDjNITU2FTqeDU2E34v84OTkhKSmpXNt499130ahRIwQEBJT4+vz586FWqw0PNze3x667smVnV02Q9fEBrKwqf7t1QUFBAbZu3YoNGzZg3759uHLlitQlERERUQlM+qqVzz//HGvXrkVMTAwsLCxKXGbWrFmYMWOG4XlGRkaNDLSFkpMrb1iAlRUgk1XOtuqSO3fuIDw83PAHlY+PD5o0aSJxVURERFQSScOsg4MDFAoFkpOTi7QnJyfD2dm5zHW//vprfP7559i5cyc6depU6nIqlQoqlapS6q0O1tYc4yql06dPY8uWLdBqtbCyssKoUaPQokULqcsiIiKiUkg6zECpVMLLywu7du0ytOn1euzatQu9evUqdb0vv/wSn3zyCaKiotCtW7fqKJXqgB07dmDjxo3QarVo2rQppk6dyiBLRERUw0k+zGDGjBmYMGECunXrhh49emDhwoXQaDSYNGkSAODZZ59F48aNMX/+fADAF198gdmzZ2PNmjVwd3c3fBRsY2MDGxsbyY6DTJ+rqysAoE+fPujXrx/kcslnriMiIqJHkDzMjhs3Drdv38bs2bORlJQET09PREVFGS4Ku379epFQ8eOPP0Kr1SI0NLTIdubMmYOPPvqoOkunWiArK8vwR1D79u3h5OQEBwcHiasiIiKi8pJ8ntnqVhPnmdVogMJOZc4LWz20Wi22b9+Of/75By+++CJ79YmIiGoQk5lnlkgKKSkpCA8Px+3btyGTyXD58uUyLyIkIiKimothluoMIQTi4uKwbds2FBQUwMbGBiEhIXB3d5e6NCIiIqoghlmqE7RaLbZs2YLTp08DADw8PDBq1ChYc0wHERGRSWOYpTph3759OH36NGQyGfr37w9fX1/IeEcJIiIik8cwS3VC3759kZiYCD8/P97Ni4iIqBbhRJpUK+Xl5eHQoUMonKxDqVTimWeeYZAlIiKqZdgzS7VOYmIiwsPDkZaWBgDo3bu3xBURERFRVWGYpVpDCIGjR4/ir7/+gk6ng1qtZk8sERFRLccwS7VCbm4uIiMjER8fDwBo3bo1Ro4cCUtLS4krIyIioqrEMEsm79atW9iwYQPu3bsHuVyOgQMHwtvbm7MVEBER1QEMs2TyhBDIyMiAvb09QkND0bhxY6lLIiIiomrCMEsmSa/XQy6/PxlH48aNMW7cODRp0gQWFhYSV0ZERETViVNzkcm5ceMGFi9ejKSkJENbq1atGGSJiIjqIIZZMhlCCBw8eBArVqzAnTt3sHv3bqlLIiIiIolxmAGZBI1Gg82bN+PixYsAgA4dOmD48OESV0VERERSY5ilGu/atWuIiIhAZmYmzMzMMHjwYHTt2pWzFRARERHDLNVs169fx6+//gohBBo0aIAxY8bAyclJ6rKIiIiohmCYpRrN1dUV7u7usLW1xbBhw6BUKqUuiYiIiGoQhlmqca5fvw4XFxeYm5tDLpfjySefhLm5udRlERERUQ3E2QyoxtDr9YiJicGKFSuwY8cOQzuDLBEREZWGPbNUI2RmZmLjxo24evUqAECn0xW5MQIRERFRSRhmSXKXLl3Cxo0bkZ2dDXNzcwwfPhydOnWSuiwiIiIyAQyzJBm9Xo89e/bgwIEDAAAnJyeEhobCwcFB4sqIiIjIVDDMkmQ0Gg2OHz8OAPDy8kJgYCDHxxIREZFRGGarmBBAdnbZy2g01VNLTWNra4vg4GBotVp06NBB6nKIiIjIBDHMViEhAF9f4NAhqSupGXQ6HXbv3o0mTZqgdevWAIBWrVpJXBURERGZMl4qXoWys40Lsj4+gJVV1dUjpfT0dISFheHQoUP4448/kJubK3VJREREVAuwZ7aaJCcD1tZlL2NlBchk1VNPdUpISMDmzZuRm5sLlUqFoKAgWFhYSF0WERER1QIMs9XE2vrRYba20el0iI6OxuHDhwEAjRo1QmhoKOrVqydxZURERFRbMMxSlcjPz0dYWBhu3boFAOjZsycCAgKgUCgkroyIiIhqE4ZZqhLm5uZwdnZGWloagoODDRd8EREREVUmhlmqNAUFBcjPz4elpSUAYPDgwejbty/UarXElREREVFtxdkMqFKkpaVh+fLl2LBhA/R6PYD7vbMMskRERFSV2DNLj+3MmTP4888/odVqYWlpibt376JBgwZSl0VERER1AMMsVVh+fj6ioqJw4sQJAECTJk0QEhICOzs7iSsjIiKiuoJhliokNTUV4eHhSE5OBgD06dMH/fr1g1zOkStERERUfRhmyWhCCGzcuBHJycmwsrLC6NGj4eHhIXVZREREVAcxzJLRZDIZRowYgV27dmHEiBGwtbWVuiQiIiKqo/iZMJVLSkoK/v77b8NzZ2dnPPXUUwyyREREJCn2zFKZhBCIi4vDtm3boNfr0aBBAzRu3FjqsoiIiIgAMMxSGbRaLbZu3WrokW3evDns7e2lLYqIiIjoAQyzVKLk5GRs2LABd+7cgUwmQ//+/eHr6wuZTCZ1aUREREQGDLNUzIkTJ7Bt2zbodDrY2toiJCQETZs2lbosIiIiomIYZqmY3Nxc6HQ6tGjRAqNGjYKVlZXUJRERERGViGGWAAB6vd5ww4NevXpBrVajXbt2HFZARERENRqn5qrjhBA4cuQIli1bBq1WC+D+PLLt27dnkCUiIqIajz2zdVhubi4iIyMRHx8P4P5Y2Z49e0pcFREREVH5MczWUTdv3kR4eDju3bsHuVyOgQMHwtvbW+qyiIiIiIzCMFvHCCFw+PBhREdHQ6/Xw97eHqGhobwRAhEREZkkhtk6Zt++fYiJiQEAtG3bFiNGjICFhYW0RRERERFVEMNsHePl5YWTJ0+id+/e6N69Oy/yIiIiIpPGMFvLCSFw+fJleHh4AABsbGzwyiuvwMyMp56IiIhMH6fmqsWys7Px+++/Y9WqVTh79qyhnUGWiIiIagummlrq2rVriIiIQGZmJhQKBfLz86UuiYiIiKjSMczWMkIIHDhwAHv27IEQAg0aNMCYMWPg5OQkdWlERERElY5hthbRaDTYuHEjLl++DADo1KkThg0bBqVSKXFlRERERFWDYbYWuXnzJi5fvgwzMzMMHToUnp6enK2AiIiIajWG2VqkVatWGDRoEDw8PODo6Ch1OURERERVjrMZmLDMzEysX78e6enphrZevXoxyBIREVGdwZ5ZE3Xp0iVs2rQJGo0GWq0WTz/9tNQlEREREVU7hlkTo9frERMTg/379wMAHB0dMXjwYImrIiIiIpIGw6wJycjIQEREBK5fvw4A6Nq1KwYPHgxzc3OJKyMiIiKSBsOsiUhKSsLKlSuRk5MDpVKJoKAgdOjQQeqyiIiIiCTFMGsiGjRoAFtbW6jVaoSGhqJBgwZSl0REREQkOYbZGiwzMxM2NjaQyWQwNzfH+PHjYW1tDTMznjYiIiIigFNz1VgJCQlYvHix4UIvAFCr1QyyRERERA9gmK1hdDodduzYgbVr1yI3Nxf//PMP9Hq91GURERER1Ujs5qtB7t69i4iICNy8eRMA4O3tjYEDB0Iu598cRERERCVhmK0h4uPj8ccffyAvLw8WFhYYOXIk2rRpI3VZRERERDUaw2wNkJmZiYiICOh0Ori6uiIkJAT29vZSl0VERERU4zHM1gC2trYYPHgw0tLS4O/vD4VCIXVJRERERCaBYVYiZ8+ehb29PRo3bgwA6Natm8QVEREREZkehtlqlp+fjx07duD48eOwt7fH1KlTYWFhIXVZRERERCapRlwmv2jRIri7u8PCwgLe3t44cuRImctv2LABbdq0gYWFBTp27Iht27ZVU6WPJzU1FcuXL8fx48cBAB06dIBSqZS4KiIiIiLTJXmYXbduHWbMmIE5c+bgxIkT6Ny5MwIDA5GSklLi8ocOHcKTTz6JKVOm4OTJkwgODkZwcDDOnDlTzZUb59y5v7Fs2TIkJyfDysoKTz/9NPz9/TntFhEREdFjkAkhhJQFeHt7o3v37vjhhx8AAHq9Hm5ubnj11Vcxc+bMYsuPGzcOGo0GW7ZsMbT17NkTnp6eWLJkySP3l5GRAbVajfT0dNjZ2VXegZRAowHU6gIMG7YVXbvGAQDc3d0xevRo2NraVum+iYiIiEyVMXlN0m5BrVaL48ePIyAgwNAml8sREBCA2NjYEteJjY0tsjwABAYGlrp8Xl4eMjIyijyqk16vgI2NBgDg5+eHZ555hkGWiIiIqJJIegFYamoqdDodnJycirQ7OTnh/PnzJa6TlJRU4vJJSUklLj9//nzMnTu3cgquACFk2Lw5GMeOpaBtW3fJ6iAiIiKqjWr9gM1Zs2YhPT3d8Lhx40a17dvKCsjKAlJSrNCmjXu17ZeIiIiorpC0Z9bBwQEKhQLJyclF2pOTk+Hs7FziOs7OzkYtr1KpoFKpKqdgI8lkgLW1JLsmIiIiqhMk7ZlVKpXw8vLCrl27DG16vR67du1Cr169SlynV69eRZYHgOjo6FKXJyIiIqLaS/KbJsyYMQMTJkxAt27d0KNHDyxcuBAajQaTJk0CADz77LNo3Lgx5s+fDwCYPn06/Pz8sGDBAgwbNgxr167FsWPHsGzZMikPg4iIiIgkIHmYHTduHG7fvo3Zs2cjKSkJnp6eiIqKMlzkdf369SJzsfbu3Rtr1qzBBx98gPfeew8tW7bE5s2b0aFDB6kOgYiIiIgkIvk8s9WtOueZJSIiIiLjmcw8s0REREREj4NhloiIiIhMFsMsEREREZkshlkiIiIiMlkMs0RERERkshhmiYiIiMhkMcwSERERkclimCUiIiIik8UwS0REREQmi2GWiIiIiEwWwywRERERmSyGWSIiIiIyWQyzRERERGSyzKQuoLoJIQAAGRkZEldCRERERCUpzGmFua0sdS7MZmZmAgDc3NwkroSIiIiIypKZmQm1Wl3mMjJRnshbi+j1ety6dQu2traQyWRVvr+MjAy4ubnhxo0bsLOzq/L9UeXjOTR9PIemj+fQtPH8mb7qPodCCGRmZqJRo0aQy8seFVvnemblcjlcXV2rfb92dnb8BjZxPIemj+fQ9PEcmjaeP9NXnefwUT2yhXgBGBERERGZLIZZIiIiIjJZDLNVTKVSYc6cOVCpVFKXQhXEc2j6eA5NH8+haeP5M301+RzWuQvAiIiIiKj2YM8sEREREZkshlkiIiIiMlkMs0RERERkshhmiYiIiMhkMcxWgkWLFsHd3R0WFhbw9vbGkSNHylx+w4YNaNOmDSwsLNCxY0ds27atmiql0hhzDn/66Sf06dMH9erVQ7169RAQEPDIc05Vz9jvw0Jr166FTCZDcHBw1RZIj2TsObx37x5efvlluLi4QKVSoVWrVvx5KiFjz9/ChQvRunVrWFpaws3NDW+88QZyc3OrqVp62L59+xAUFIRGjRpBJpNh8+bNj1wnJiYGXbt2hUqlQosWLRAWFlbldZZI0GNZu3atUCqV4pdffhFnz54Vzz//vLC3txfJycklLn/w4EGhUCjEl19+Kc6dOyc++OADYW5uLk6fPl3NlVMhY8/h+PHjxaJFi8TJkydFfHy8mDhxolCr1eLff/+t5sqpkLHnsNCVK1dE48aNRZ8+fcTIkSOrp1gqkbHnMC8vT3Tr1k0MHTpUHDhwQFy5ckXExMSIuLi4aq6chDD+/K1evVqoVCqxevVqceXKFbFjxw7h4uIi3njjjWqunApt27ZNvP/++2Ljxo0CgNi0aVOZy1++fFlYWVmJGTNmiHPnzon//Oc/QqFQiKioqOop+AEMs4+pR48e4uWXXzY81+l0olGjRmL+/PklLj927FgxbNiwIm3e3t5i6tSpVVonlc7Yc/iwgoICYWtrK3799deqKpEeoSLnsKCgQPTu3Vv8/PPPYsKECQyzEjP2HP7444+iefPmQqvVVleJVAZjz9/LL78sBgwYUKRtxowZwsfHp0rrpPIpT5h95513RPv27Yu0jRs3TgQGBlZhZSXjMIPHoNVqcfz4cQQEBBja5HI5AgICEBsbW+I6sbGxRZYHgMDAwFKXp6pVkXP4sOzsbOTn56N+/fpVVSaVoaLn8OOPP4ajoyOmTJlSHWVSGSpyDiMjI9GrVy+8/PLLcHJyQocOHTBv3jzodLrqKpv+pyLnr3fv3jh+/LhhKMLly5exbds2DB06tFpqpsdXk/KMWbXvsRZJTU2FTqeDk5NTkXYnJyecP3++xHWSkpJKXD4pKanK6qTSVeQcPuzdd99Fo0aNin1TU/WoyDk8cOAAli9fjri4uGqokB6lIufw8uXL2L17N5566ils27YNFy9exLRp05Cfn485c+ZUR9n0PxU5f+PHj0dqaip8fX0hhEBBQQFefPFFvPfee9VRMlWC0vJMRkYGcnJyYGlpWW21sGeW6DF8/vnnWLt2LTZt2gQLCwupy6FyyMzMxDPPPIOffvoJDg4OUpdDFaTX6+Ho6Ihly5bBy8sL48aNw/vvv48lS5ZIXRqVQ0xMDObNm4fFixfjxIkT2LhxI7Zu3YpPPvlE6tLIBLFn9jE4ODhAoVAgOTm5SHtycjKcnZ1LXMfZ2dmo5alqVeQcFvr666/x+eefY+fOnejUqVNVlkllMPYcXrp0CVevXkVQUJChTa/XAwDMzMyQkJAADw+Pqi2aiqjI96GLiwvMzc2hUCgMbW3btkVSUhK0Wi2USmWV1kz/ryLn78MPP8QzzzyD5557DgDQsWNHaDQavPDCC3j//fchl7OvraYrLc/Y2dlVa68swJ7Zx6JUKuHl5YVdu3YZ2vR6PXbt2oVevXqVuE6vXr2KLA8A0dHRpS5PVasi5xAAvvzyS3zyySeIiopCt27dqqNUKoWx57BNmzY4ffo04uLiDI8RI0agf//+iIuLg5ubW3WWT6jY96GPjw8uXrxo+EMEAC5cuAAXFxcG2WpWkfOXnZ1dLLAW/mEihKi6YqnS1Kg8U+2XnNUya9euFSqVSoSFhYlz586JF154Qdjb24ukpCQhhBDPPPOMmDlzpmH5gwcPCjMzM/H111+L+Ph4MWfOHE7NJTFjz+Hnn38ulEqlCA8PF4mJiYZHZmamVIdQ5xl7Dh/G2QykZ+w5vH79urC1tRWvvPKKSEhIEFu2bBGOjo7i008/leoQ6jRjz9+cOXOEra2t+P3338Xly5fFX3/9JTw8PMTYsWOlOoQ6LzMzU5w8eVKcPHlSABDffPONOHnypLh27ZoQQoiZM2eKZ555xrB84dRcb7/9toiPjxeLFi3i1Fym7D//+Y9o0qSJUCqVokePHuK///2v4TU/Pz8xYcKEIsuvX79etGrVSiiVStG+fXuxdevWaq6YHmbMOWzatKkAUOwxZ86c6i+cDIz9PnwQw2zNYOw5PHTokPD29hYqlUo0b95cfPbZZ6KgoKCaq6ZCxpy//Px88dFHHwkPDw9hYWEh3NzcxLRp08Tdu3erv3ASQgixZ8+eEn+3FZ63CRMmCD8/v2LreHp6CqVSKZo3by5WrFhR7XULIYRMCPbnExEREZFp4phZIiIiIjJZDLNEREREZLIYZomIiIjIZDHMEhEREZHJYpglIiIiIpPFMEtEREREJothloiIiIhMFsMsEREREZkshlkiIgBhYWGwt7eXuowKk8lk2Lx5c5nLTJw4EcHBwdVSDxFRdWGYJaJaY+LEiZDJZMUeFy9elLo0hIWFGeqRy+VwdXXFpEmTkJKSUinbT0xMxJAhQwAAV69ehUwmQ1xcXJFlvvvuO4SFhVXK/krz0UcfGY5ToVDAzc0NL7zwAtLS0ozaDoM3EZWXmdQFEBFVpsGDB2PFihVF2ho2bChRNUXZ2dkhISEBer0ep06dwqRJk3Dr1i3s2LHjsbft7Oz8yGXUavVj76c82rdvj507d0Kn0yE+Ph6TJ09Geno61q1bVy37J6K6hT2zRFSrqFQqODs7F3koFAp888036NixI6ytreHm5oZp06YhKyur1O2cOnUK/fv3h62tLezs7ODl5YVjx44ZXj9w4AD69OkDS0tLuLm54bXXXoNGoymzNplMBmdnZzRq1AhDhgzBa6+9hp07dyInJwd6vR4ff/wxXF1doVKp4OnpiaioKMO6Wq0Wr7zyClxcXGBhYYGmTZti/vz5RbZdOMygWbNmAIAuXbpAJpOhX79+AIr2di5btgyNGjWCXq8vUuPIkSMxefJkw/M//vgDXbt2hYWFBZo3b465c+eioKCgzOM0MzODs7MzGjdujICAAIwZMwbR0dGG13U6HaZMmYJmzZrB0tISrVu3xnfffWd4/aOPPsKvv/6KP/74w9DLGxMTAwC4ceMGxo4dC3t7e9SvXx8jR47E1atXy6yHiGo3hlkiqhPkcjm+//57nD17Fr/++it2796Nd955p9Tln3rqKbi6uuLo0aM4fvw4Zs6cCXNzcwDApUuXMHjwYISEhODvv//GunXrcODAAbzyyitG1WRpaQm9Xo+CggJ89913WLBgAb7++mv8/fffCAwMxIgRI/DPP/8AAL7//ntERkZi/fr1SEhIwOrVq+Hu7l7ido8cOQIA2LlzJxITE7Fx48Ziy4wZMwZ37tzBnj17DG1paWmIiorCU089BQDYv38/nn32WUyfPh3nzp3D0qVLERYWhs8++6zcx3j16lXs2LEDSqXS0KbX6+Hq6ooNGzbg3LlzmD17Nt577z2sX78eAPDWW29h7NixGDx4MBITE5GYmIjevXsjPz8fgYGBsLW1xf79+3Hw4EHY2Nhg8ODB0Gq15a6JiGoZQURUS0yYMEEoFAphbW1teISGhpa47IYNG0SDBg0Mz1esWCHUarXhua2trQgLCytx3SlTpogXXnihSNv+/fuFXC4XOTk5Ja7z8PYvXLggWrVqJbp16yaEEKJRo0bis88+K7JO9+7dxbRp04QQQrz66qtiwIABQq/Xl7h9AGLTpk1CCCGuXLkiAIiTJ08WWWbChAli5MiRhucjR44UkydPNjxfunSpaNSokdDpdEIIIfz9/cW8efOKbOO3334TLi4uJdYghBBz5swRcrlcWFtbCwsLCwFA/F979xYS9dbGcfz7WpiH9EJKai4sSB2Espq0MotIOhgZ4hROKXRjIpYZWlEXZg3RwUKFohOIQTak1E2SqdGFZROEFSpUjlnagSDIQJGcNJ33Ihz2ZBq2Ye93fH+fu/n/11r/Z625eeaZtWYAV0lJyZh9XC6Xa/fu3a4tW7aMGevIs41Go8cafPv2zeXv7++qr68fd3wRmby0Z1ZEJpU1a9Zw8eJF9+vAwEDgR5Xy5MmTtLW10dvby/fv33E6nXz9+pWAgIBR4+Tn57Nz504qKircX5XPmzcP+LEFobW1FZvN5m7vcrkYHh6ms7OTqKioX8bW09PD9OnTGR4exul0snLlSsrKyujt7eXjx4/Ex8d7tI+Pj6elpQX4sUVg3bp1GI1GEhMTSUpKYv369X9rrdLT08nMzOTChQtMmzYNm83Gtm3b8PHxcc/Tbrd7VGKHhobGXTcAo9FIdXU1TqeTa9eu0dzczJ49ezzanD9/nvLyct69e0d/fz8DAwMsWrRo3HhbWlro6OggKCjI47rT6eT169d/sAIiMhkomRWRSSUwMJDw8HCPa11dXSQlJZGdnc3x48cJCQnh4cOHZGRkMDAw8Muk7OjRo6SlpVFTU0NtbS1HjhyhsrKSlJQU+vr6yMrKIjc3d1S/sLCwMWMLCgri2bNn+Pj4MHv2bPz9/QHo7e397bxMJhOdnZ3U1tZy7949UlNTWbt2LTdv3vxt37Fs3rwZl8tFTU0NsbGxNDY2Ulpa6r7f19eH1WrFbDaP6uvn5zfmuL6+vu734NSpU2zatAmr1cqxY8cAqKysZP/+/RQXFxMXF0dQUBBnzpzh8ePH48bb19fHkiVLPD5EjPhfOeQnIv88JbMiMuk9ffqU4eFhiouL3VXHkf2Z44mMjCQyMpK8vDy2b9/OlStXSElJwWQy8eLFi1FJ8+/4+Pj8sk9wcDAGgwG73c7q1avd1+12O0uXLvVoZ7FYsFgsbN26lcTERL58+UJISIjHeCP7U4eGhsaNx8/PD7PZjM1mo6OjA6PRiMlkct83mUw4HI4Jz/NnBQUFJCQkkJ2d7Z7nihUr2LVrl7vNz5VVX1/fUfGbTCaqqqoIDQ0lODj4b8UkIpOHDoCJyKQXHh7O4OAg586d482bN1RUVHDp0qUx2/f395OTk0NDQwNv377FbrfT1NTk3j5w8OBBHj16RE5ODs3Nzbx69Ypbt25N+ADYXx04cICioiKqqqpwOBwcOnSI5uZm9u7dC0BJSQnXr1+nra2N9vZ2bty4waxZs375Rw+hoaH4+/tTV1fHp0+f6OnpGfO56enp1NTUUF5e7j74NaKwsJCrV69itVp5/vw5L1++pLKykoKCggnNLS4ujujoaE6cOAFAREQET548ob6+nvb2dg4fPkxTU5NHn7lz59La2orD4eDz588MDg6Snp7OjBkzSE5OprGxkc7OThoaGsjNzeXDhw8TiklEJg8lsyIy6S1cuJCSkhKKioqYP38+NpvN42etfjZlyhS6u7vZsWMHkZGRpKamsnHjRqxWKwDR0dHcv3+f9vZ2Vq1axeLFiyksLMRgMPxxjLm5ueTn57Nv3z4WLFhAXV0d1dXVREREAD+2KJw+fZqYmBhiY2Pp6urizp077krzX02dOpWzZ89y+fJlDAYDycnJYz43ISGBkJAQHA4HaWlpHvc2bNjA7du3uXv3LrGxsSxfvpzS0lLmzJkz4fnl5eVRVlbG+/fvycrKwmw2Y7FYWLZsGd3d3R5VWoDMzEyMRiMxMTHMnDkTu91OQEAADx48ICwsDLPZTFRUFBkZGTidTlVqRf6P/cflcrn+7SBERERERP6EKrMiIiIi4rWUzIqIiIiI11IyKyIiIiJeS8msiIiIiHgtJbMiIiIi4rWUzIqIiIiI11IyKyIiIiJeS8msiIiIiHgtJbMiIiIi4rWUzIqIiIiI11IyKyIiIiJe678NIk8id95UYwAAAABJRU5ErkJggg==",
      "text/plain": [
       "<Figure size 800x600 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import matplotlib.pyplot as plt # Import pyplot\n",
    "from sklearn.metrics import roc_curve # Import roc_curve\n",
    "\n",
    "fpr, tpr, _ = roc_curve(y_test, y_pred_prob)\n",
    "plt.figure(figsize=(8, 6))\n",
    "plt.plot(fpr, tpr, label=f\"Logistic Regression (AUC = {auc:.2f})\", color=\"blue\")\n",
    "plt.plot([0, 1], [0, 1], linestyle=\"--\", color=\"gray\")  # 参考线\n",
    "plt.xlabel(\"False Positive Rate\")\n",
    "plt.ylabel(\"True Positive Rate\")\n",
    "plt.title(\"ROC Curve for Logistic Regression Model\")\n",
    "plt.legend()\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Accuracy of logistic regression model: 0.6506\n",
      "Logistic regression model AUC: 0.6938\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "from sklearn.linear_model import LogisticRegression\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.metrics import accuracy_score, roc_auc_score, roc_curve\n",
    "\n",
    "# 读取基因表达数据集\n",
    "df = pd.read_csv(\"gse36059.csv\")  # 请修改为你的数据文件路径\n",
    "\n",
    "# 确保目标变量 (1=排斥, 0=非排斥)\n",
    "df[\"Label\"] = df[\"Diagnosis\"].apply(lambda x: 1 if x in [\"TCMR\", \"ABMR\", \"Mixed\"] else 0)\n",
    "\n",
    "# 选择特征列（仅使用指定的 3 个基因）\n",
    "selected_genes = [\"GPR171\", \"FAXDC2\", \"229668_at\"]\n",
    "X_selected = df[selected_genes]  # 仅保留选定基因\n",
    "y = df[\"Label\"]\n",
    "\n",
    "# 划分训练集和测试集\n",
    "X_train, X_test, y_train, y_test = train_test_split(X_selected, y, test_size=0.2, random_state=42, stratify=y)\n",
    "\n",
    "# 训练逻辑回归模型\n",
    "log_reg = LogisticRegression(solver='liblinear', random_state=42)\n",
    "log_reg.fit(X_train, y_train)\n",
    "\n",
    "# 进行预测\n",
    "y_pred = log_reg.predict(X_test)\n",
    "y_pred_prob = log_reg.predict_proba(X_test)[:, 1]\n",
    "\n",
    "# 计算模型评估指标\n",
    "accuracy = accuracy_score(y_test, y_pred)\n",
    "auc = roc_auc_score(y_test, y_pred_prob)\n",
    "\n",
    "# 输出结果\n",
    "print(f\"Accuracy of logistic regression model: {accuracy:.4f}\")\n",
    "print(f\"Logistic regression model AUC: {auc:.4f}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Accuracy of logistic regression model: 0.6988\n",
      "Logistic regression model AUC: 0.6131\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "from sklearn.linear_model import LogisticRegression\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.metrics import accuracy_score, roc_auc_score, roc_curve\n",
    "\n",
    "# 读取基因表达数据集\n",
    "df = pd.read_csv(\"gse36059.csv\")  # 请修改为你的数据文件路径\n",
    "\n",
    "# 确保目标变量 (1=排斥, 0=非排斥)\n",
    "df[\"Label\"] = df[\"Diagnosis\"].apply(lambda x: 1 if x in [\"TCMR\", \"ABMR\", \"Mixed\"] else 0)\n",
    "\n",
    "# 选择特征列（仅使用指定的 3 个基因）\n",
    "selected_genes = [\"WDR3\", \"LOC100507073\", \"SCMH1\"]\n",
    "X_selected = df[selected_genes]  # 仅保留选定基因\n",
    "y = df[\"Label\"]\n",
    "\n",
    "# 划分训练集和测试集\n",
    "X_train, X_test, y_train, y_test = train_test_split(X_selected, y, test_size=0.2, random_state=42, stratify=y)\n",
    "\n",
    "# 训练逻辑回归模型\n",
    "log_reg = LogisticRegression(solver='liblinear', random_state=42)\n",
    "log_reg.fit(X_train, y_train)\n",
    "\n",
    "# 进行预测\n",
    "y_pred = log_reg.predict(X_test)\n",
    "y_pred_prob = log_reg.predict_proba(X_test)[:, 1]\n",
    "\n",
    "# 计算模型评估指标\n",
    "accuracy = accuracy_score(y_test, y_pred)\n",
    "auc = roc_auc_score(y_test, y_pred_prob)\n",
    "\n",
    "# 输出结果\n",
    "print(f\"Accuracy of logistic regression model: {accuracy:.4f}\")\n",
    "print(f\"Logistic regression model AUC: {auc:.4f}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAArMAAAIjCAYAAAAQgZNYAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjkuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8hTgPZAAAACXBIWXMAAA9hAAAPYQGoP6dpAACIMElEQVR4nOzdd1hTZ/8G8DsJJOygIkNBUdwTJyooKiguFAW1tUOtbe22tcsurR3apbV9W2ttrdSqdYBa6sDiwMnrxjoQ67bIUFFGGIHk+f3hS34iQwKBQ+D+XFcuzZMzvieHcfPkOc+RCSEEiIiIiIjMkFzqAoiIiIiIKothloiIiIjMFsMsEREREZkthlkiIiIiMlsMs0RERERkthhmiYiIiMhsMcwSERERkdlimCUiIiIis8UwS0RERERmi2GWiMzGl19+iZYtW0KhUMDb21vqcqqFTCbDhx9+aJJtXblyBTKZDOHh4SbZHgGxsbGQyWSIjY2VupQaEx4eDplMhitXrhi97ocffgiZTGb6oojuwzBLVEFFP9CLHhYWFmjatCmmTJmCpKSkUtcRQuC3337DgAED4OjoCBsbG3Tu3BkfffQRNBpNmfvauHEjhg8fDicnJyiVSjRp0gQTJkzArl27KlRrXl4evv76a/j4+ECtVsPKygpt2rTBSy+9hPPnz1fq+KX2119/4a233oKvry+WL1+OefPmVev+pkyZAjs7u2rdh6msXr0aixYtqtZ9FAXjoodcLkfDhg0xfPhwxMXFVeu+6Z6BAwdCJpOhdevWpb4eExNjOD8RERE1XB2RdCykLoDI3Hz00Udo0aIF8vLy8N///hfh4eHYv38/Tp8+DSsrK8NyOp0OkyZNwrp169C/f398+OGHsLGxwb59+zB37lysX78eO3bsgIuLi2EdIQSeeuophIeHo1u3bpg5cyZcXV2RnJyMjRs3IiAgAAcOHEC/fv3KrO/WrVsYNmwYjh07hlGjRmHSpEmws7NDYmIi1qxZg6VLl0Kr1Vbre1Qddu3aBblcjmXLlkGpVEpdTrXJzc2FhYVxP5pXr16N06dP49VXXy3W3rx5c+Tm5sLS0tJk9T366KMYMWIEdDodzp8/j8WLF2PQoEE4cuQIOnfubLL91FYDBgxAbm6uZF+DVlZWuHDhAg4fPozevXsXe23VqlWwsrJCXl6eJLURSYVhlshIw4cPR8+ePQEATz/9NJycnPD5558jKioKEyZMMCz3xRdfYN26dXjjjTfw5ZdfGtqfffZZTJgwASEhIZgyZQq2bdtmeG3BggUIDw/Hq6++ioULFxb7eO69997Db7/99tCgM2XKFJw4cQIREREIDQ0t9trHH3+M9957r0rHX6SwsBB6vb7GfqmnpaXB2traZPsTQiAvLw/W1tYm2Z6p3P8HUVXJZDKTbg8Aunfvjscff9zwvH///hg+fDh++OEHLF682KT7ehiNRgNbW9sa3adcLjf5e2oMLy8vFBYW4vfffy8WZvPy8rBx40aMHDkSkZGRktVHJAUOMyCqov79+wMALl68aGjLzc3Fl19+iTZt2mD+/Pkl1gkODsbkyZMRHR2N//73v4Z15s+fj3bt2uGrr74qdZzZE088UaI35n6HDh3Cli1bMG3atBJBFgBUKhW++uorw/OBAwdi4MCBJZabMmUKPD09Dc+LPmL+6quvsGjRInh5eUGlUuHEiROwsLDA3LlzS2wjMTERMpkM3333naHt7t27ePXVV+Hh4QGVSoVWrVrh888/h16vL/OYgHuhbPny5dBoNIaPUYvGgRYWFuLjjz821OTp6Yl3330X+fn5xbbh6emJUaNGYfv27ejZsyesra3x448/lrvfili/fj169OgBa2trODk54fHHHy912Mn69evRoUMHWFlZoVOnTti4cWOJ97noWO8fM5uVlYVXX30Vnp6eUKlUcHZ2xpAhQ3D8+HEA987hli1bcPXqVcN7U7TNssbMnjt3DhMmTEDjxo1hbW2Ntm3bVvqPnNK+/oGKn+vbt2/jiSeegIODAxwdHTF58mScPHmyRN1Fwz4uXryIESNGwN7eHo899hgAQK/XY9GiRejYsSOsrKzg4uKC6dOn486dO8X2dfToUQQFBcHJyQnW1tZo0aIFnnrqqWLLrFmzBj169IC9vT0cHBzQuXNnfPPNN4bXyxozW5Gvg6JjSEpKQkhICOzs7NC4cWO88cYb0Ol0FX7PH330Uaxdu7bYe/nnn38iJyen2B/U9ztx4gSGDx8OBwcH2NnZISAgwPCz535nzpzB4MGDYW1tDXd3d3zyySdlfn9u27YN/fv3h62tLezt7TFy5EicOXOmwsdBZCrsmSWqoqKLIho0aGBo279/P+7cuYMZM2aU2ZP65JNPYvny5di8eTP69OmD/fv3Iz09Ha+++ioUCkWlaomKigJwL/RWh+XLlyMvLw/PPvssVCoV3Nzc4O/vj3Xr1mHOnDnFll27di0UCgXGjx8PAMjJyYG/vz+SkpIwffp0NGvWDAcPHsQ777yD5OTkcsd8/vbbb1i6dCkOHz6Mn3/+GQAMQy2efvpp/PrrrwgLC8Prr7+OQ4cOYf78+UhISMDGjRuLbScxMRGPPvoopk+fjmeeeQZt27at0vsRHh6OqVOnolevXpg/fz5SU1PxzTff4MCBAzhx4gQcHR0BAFu2bMHEiRPRuXNnzJ8/H3fu3MG0adPQtGnTh+7jueeeQ0REBF566SV06NABt2/fxv79+5GQkIDu3bvjvffeQ0ZGBv799198/fXXAFDuWN+///4b/fv3h6WlJZ599ll4enri4sWL+PPPP/Hpp58a/R6U9vVf0XOt1+sRHByMw4cP4/nnn0e7du3wxx9/YPLkyaXuq7CwEEFBQfDz88NXX30FGxsbAMD06dMN5+KVV17B5cuX8d133+HEiRM4cOAALC0tkZaWhqFDh6Jx48aYNWsWHB0dceXKFWzYsMGw/ZiYGDz66KMICAjA559/DgBISEjAgQMHMGPGjDLfg4p+HQD3hh8FBQXBx8cHX331FXbs2IEFCxbAy8sLzz//fIXe80mTJuHDDz9EbGwsBg8eDODeUJOAgAA4OzuXWP7MmTPo378/HBwc8NZbb8HS0hI//vgjBg4ciD179sDHxwcAkJKSgkGDBqGwsBCzZs2Cra0tli5dWuqnF7/99hsmT56MoKAgfP7558jJycEPP/wAPz8/nDhxosQfaUTVShBRhSxfvlwAEDt27BA3b94U169fFxEREaJx48ZCpVKJ69evG5ZdtGiRACA2btxY5vbS09MFADFu3DghhBDffPPNQ9d5mLFjxwoA4s6dOxVa3t/fX/j7+5donzx5smjevLnh+eXLlwUA4eDgINLS0oot++OPPwoA4tSpU8XaO3ToIAYPHmx4/vHHHwtbW1tx/vz5YsvNmjVLKBQKce3atXJrnTx5srC1tS3WFh8fLwCIp59+ulj7G2+8IQCIXbt2GdqaN28uAIjo6Ohy91Pe/u6n1WqFs7Oz6NSpk8jNzTW0b968WQAQs2fPNrR17txZuLu7i6ysLENbbGysAFDsfRZCCABizpw5hudqtVq8+OKL5dY6cuTIEtsR4v/P2/Llyw1tAwYMEPb29uLq1avFltXr9eXuo2hbc+fOFTdv3hQpKSli3759olevXgKAWL9+vWHZip7ryMhIAUAsWrTIsIxOpxODBw8uUffkyZMFADFr1qxi29y3b58AIFatWlWsPTo6ulj7xo0bBQBx5MiRMo9xxowZwsHBQRQWFpa5zO7duwUAsXv3biGEcV8HRcfw0UcfFdtmt27dRI8ePcrcZxF/f3/RsWNHIYQQPXv2FNOmTRNCCHHnzh2hVCrFr7/+aqjv/vMREhIilEqluHjxoqHtxo0bwt7eXgwYMMDQ9uqrrwoA4tChQ4a2tLQ0oVarBQBx+fJlIYQQWVlZwtHRUTzzzDPF6ktJSRFqtbpY+5w5cwSjBlU3DjMgMlJgYCAaN24MDw8PhIWFwdbWFlFRUXB3dzcsk5WVBQCwt7cvcztFr2VmZhb7t7x1HsYU2yhPaGgoGjduXKxt3LhxsLCwwNq1aw1tp0+fxtmzZzFx4kRD2/r169G/f380aNAAt27dMjwCAwOh0+mwd+9eo+vZunUrAGDmzJnF2l9//XUA93pE79eiRQsEBQUZvZ/SHD16FGlpaXjhhReKjaEcOXIk2rVrZ9j3jRs3cOrUKTz55JPFekz9/f0rdMGUo6MjDh06hBs3blS55ps3b2Lv3r146qmn0KxZs2KvVXT6pDlz5qBx48ZwdXVF//79kZCQgAULFiAsLMywTEXPdXR0NCwtLfHMM88Y1pXL5XjxxRfL3P+DvZfr16+HWq3GkCFDiu2rR48esLOzw+7duwHA0Du6efNmFBQUlLptR0dHaDQaxMTEVOi9ACr+dXC/5557rtjz/v3749KlSxXeJ3Cvd3bDhg3QarWIiIiAQqHA2LFjSyyn0+nw119/ISQkBC1btjS0u7m5YdKkSdi/f7/h58bWrVvRp0+fYkOZGjdubBjOUSQmJgZ3797Fo48+Wuw9VygU8PHxMbznRDWFYZbISN9//z1iYmIQERGBESNG4NatW1CpVMWWKQqTRaG2NA8GXgcHh4eu8zCm2EZ5WrRoUaLNyckJAQEBWLdunaFt7dq1sLCwwLhx4wxt//zzD6Kjo9G4ceNij8DAQAD3LvAy1tWrVyGXy9GqVati7a6urnB0dMTVq1cfWn9lFW27tKEK7dq1M7xe9O+DNZbV9qAvvvgCp0+fhoeHB3r37o0PP/zQ6OBTpGi9Tp06VWp94N4FjDExMfjzzz/x2muvITc3t8R4z4qe66tXr8LNzc0wXKBIWe+LhYVFsT8ai/aVkZEBZ2fnEvvLzs427Mvf3x+hoaGYO3cunJycMGbMGCxfvrzY2OoXXngBbdq0wfDhw+Hu7o6nnnoK0dHR5b4fFf06KGJlZVXiD8IGDRqUGN/7MI888ggyMjKwbds2rFq1CqNGjSr1j9ibN28iJyen1Prat28PvV6P69evG46ltGm/Hlz3n3/+AQAMHjy4xHv+119/Vep7magqOGaWyEi9e/c2zGYQEhICPz8/TJo0CYmJiYaet/bt2wO4Nz4xJCSk1O38/fffAIAOHToAuPeLDwBOnTpV5joPc/82ii7MKY9MJoMQokR7WRejlHXl/yOPPIKpU6ciPj4e3t7eWLduHQICAuDk5GRYRq/XY8iQIXjrrbdK3UabNm0eWm9ZKtqrWNtmLqiICRMmoH///ti4cSP++usvfPnll/j888+xYcMGDB8+vMbrad26tSGUjho1CgqFArNmzcKgQYMM3xfVda5VKhXk8uJ9MHq9Hs7Ozli1alWp6xQFx6K5V//73//izz//xPbt2/HUU09hwYIF+O9//ws7Ozs4OzsjPj4e27dvx7Zt27Bt2zYsX74cTz75JH799ddK1fygyo6Hf5CbmxsGDhyIBQsW4MCBAzU6g0HRBWG//fYbXF1dS7xu7NRyRFXFnlmiKlAoFJg/fz5u3LhR7Kp9Pz8/ODo6YvXq1WUGwxUrVgC4FwiK1mnQoAF+//13o65svl9wcDAAYOXKlRVavkGDBrh7926J9gd7kx4mJCQESqUSa9euRXx8PM6fP49HHnmk2DJeXl7Izs5GYGBgqY8HP/auiObNm0Ov1xt6ioqkpqbi7t27aN68udHbNGbfwL2Lyh6UmJhoeL3o3wsXLpRYrrS20ri5ueGFF17Apk2bcPnyZTRq1KjYxVoVDfNFHzOfPn26QstXxHvvvQd7e3u8//77hraKnuvmzZsjOTkZOTk5xbZZ0felaF+3b9+Gr69vqfvq2rVrseX79OmDTz/9FEePHsWqVatw5swZrFmzxvC6UqlEcHAwFi9ejIsXL2L69OlYsWJFmTVV9OugOkyaNAn79u2Dg4MDRowYUeoyjRs3ho2NTan1nTt3DnK5HB4eHgDuHcuD30tAyWPz8vICADg7O5f6npc2QwpRdWKYJaqigQMHonfv3li0aJFhsnIbGxu88cYbSExMLHXKoy1btiA8PBxBQUHo06ePYZ23334bCQkJePvtt0vtMV25ciUOHz5cZi19+/bFsGHD8PPPP2PTpk0lXtdqtXjjjTcMz728vHDu3DncvHnT0Hby5EkcOHCgwscP3BtrGBQUhHXr1mHNmjVQKpUlepcnTJiAuLg4bN++vcT6d+/eRWFhoVH7BGD4Bf7gTAgLFy4EcG/cYnXp2bMnnJ2dsWTJkmIfVW/btg0JCQmGfTdp0gSdOnXCihUrkJ2dbVhuz549OHXqVLn70Ol0yMjIKNbm7OyMJk2aFNunra1tieVK07hxYwwYMAC//PILrl27Vuy10r7eKsLR0RHTp0/H9u3bER8fD6Di5zooKAgFBQX46aefDK/r9Xp8//33Fd7/hAkToNPp8PHHH5d4rbCw0PDH2p07d0ocY9EtkYvey9u3bxd7XS6Xo0uXLsWWeVBFvw6qQ1hYGObMmYPFixeXOf+yQqHA0KFD8ccffxS7HW1qaipWr14NPz8/w/CkESNG4L///W+xnzE3b94s0esdFBQEBwcHzJs3r9Txx/f/PCGqCfwsgMgE3nzzTYwfPx7h4eGGiztmzZqFEydO4PPPP0dcXBxCQ0NhbW2N/fv3Y+XKlWjfvn2Jjy7ffPNNnDlzBgsWLMDu3bsRFhYGV1dXpKSkYNOmTTh8+DAOHjxYbi0rVqzA0KFDMW7cOAQHByMgIAC2trb4559/sGbNGiQnJxvmmn3qqaewcOFCBAUFYdq0aUhLS8OSJUvQsWNHw0UhFTVx4kQ8/vjjWLx4MYKCgopNR1R0bFFRURg1ahSmTJmCHj16QKPR4NSpU4iIiMCVK1eKDUuoiK5du2Ly5MlYunQp7t69C39/fxw+fBi//vorQkJCMGjQIKO296CCggJ88sknJdobNmyIF154AZ9//jmmTp0Kf39/PProo4YpmTw9PfHaa68Zlp83bx7GjBkDX19fTJ06FXfu3MF3332HTp06FQu4D8rKyoK7uzvCwsLQtWtX2NnZYceOHThy5AgWLFhgWK5Hjx5Yu3YtZs6ciV69esHOzs7QS/+gb7/9Fn5+fujevTueffZZtGjRAleuXMGWLVsMYdRYM2bMwKJFi/DZZ59hzZo1FT7XISEh6N27N15//XVcuHAB7dq1Q1RUFNLT0wFUrMfZ398f06dPx/z58xEfH4+hQ4fC0tIS//zzD9avX49vvvkGYWFh+PXXX7F48WKMHTsWXl5eyMrKwk8//VSsV/Ppp59Geno6Bg8eDHd3d1y9ehX/+c9/4O3tbRg69CBLS8sKfx2YmlqtLjYncVk++eQTxMTEwM/PDy+88AIsLCzw448/Ij8/H1988YVhubfeegu//fYbhg0bhhkzZhim5mrevLlhWBRwb2z+Dz/8gCeeeALdu3fHI488gsaNG+PatWvYsmULfH19i31SRVTtpJ1Mgch8FE3NVdrUPjqdTnh5eQkvL69i0/rodDqxfPly4evrKxwcHISVlZXo2LGjmDt3rsjOzi5zXxEREWLo0KGiYcOGwsLCQri5uYmJEyeK2NjYCtWak5MjvvrqK9GrVy9hZ2cnlEqlaN26tXj55ZfFhQsXii27cuVK0bJlS6FUKoW3t7fYvn17mVNzffnll2XuMzMzU1hbWwsAYuXKlaUuk5WVJd555x3RqlUroVQqhZOTk+jXr5/46quvhFarLfeYypoqq6CgQMydO1e0aNFCWFpaCg8PD/HOO++IvLy8Yss1b95cjBw5stx9PLg/AKU+vLy8DMutXbtWdOvWTahUKtGwYUPx2GOPiX///bfE9tasWSPatWsnVCqV6NSpk4iKihKhoaGiXbt2xZbDfVNz5efnizfffFN07dpV2NvbC1tbW9G1a1exePHiYutkZ2eLSZMmCUdHx2LTfZU2NZcQQpw+fVqMHTtWODo6CisrK9G2bVvxwQcflPt+POxrYMqUKUKhUBi+vip6rm/evCkmTZok7O3thVqtFlOmTBEHDhwQAMSaNWuKnY/ypkpbunSp6NGjh7C2thb29vaic+fO4q233hI3btwQQghx/Phx8eijj4pmzZoJlUolnJ2dxahRo8TRo0cN2yj6vnN2dhZKpVI0a9ZMTJ8+XSQnJxuWeXBqriIV+Too6xgqOn3V/VNzlaW0qbmKjj8oKEjY2dkJGxsbMWjQIHHw4MES6//999/C399fWFlZiaZNm4qPP/5YLFu2rNjUXPfvKygoSKjVamFlZSW8vLzElClTir2nnJqLaoJMiEp+tkRERFXi7e2Nxo0bGzUVVH2wadMmjB07Fvv374evr6/U5RBRLccxs0RE1aygoKDEmODY2FicPHmy3l8sk5ubW+y5TqfDf/7zHzg4OKB79+4SVUVE5oRjZomIqllSUhICAwPx+OOPo0mTJjh37hyWLFkCV1fXEhPo1zcvv/wycnNz0bdvX+Tn52PDhg04ePAg5s2bZ5ZTqRFRzeMwAyKiapaRkYFnn30WBw4cwM2bN2Fra4uAgAB89tlnhmmO6qvVq1djwYIFuHDhAvLy8tCqVSs8//zzeOmll6QujYjMBMMsEREREZktjpklIiIiIrPFMEtEREREZqveXQCm1+tx48YN2NvbV/gWkERERERUc4QQyMrKQpMmTSCXl9/3Wu/C7I0bNwz3oSYiIiKi2uv69etwd3cvd5l6F2bt7e0B3Htziu5HTURERES1R2ZmJjw8PAy5rTz1LswWDS1wcHBgmCUiIiKqxSoyJJQXgBERERGR2WKYJSIiIiKzxTBLRERERGar3o2ZrQghBAoLC6HT6aQuhYjMnEKhgIWFBacCJCKqJgyzD9BqtUhOTkZOTo7UpRBRHWFjYwM3NzcolUqpSyEiqnMYZu+j1+tx+fJlKBQKNGnSBEqlkr0pRFRpQghotVrcvHkTly9fRuvWrR86+TcRERmHYfY+Wq0Wer0eHh4esLGxkbocIqoDrK2tYWlpiatXr0Kr1cLKykrqkoiI6hR2EZSCPSdEZEr8mUJEVH34E5aIiIiIzBbDLBERERGZLYZZqjBPT08sWrSo0uuHh4fD0dHRZPXUJVV9b43xxBNPYN68eTWyr/rg7NmzcHd3h0ajkboUIqJ6iWG2jpgyZQpCQkKqdR9HjhzBs88+W6FlSwtnEydOxPnz5yu9//DwcMhkMshkMsjlcri5uWHixIm4du1apbdZWxjz3lbFyZMnsXXrVrzyyislXvv999+hUCjw4osvlnitvD9EZDIZNm3aVKwtMjISAwcOhFqthp2dHbp06YKPPvoI6enppjiMUqWnp+Oxxx6Dg4MDHB0dMW3aNGRnZz90vbi4OAwePBi2trZwcHDAgAEDkJuba3j9008/Rb9+/WBjY1Pqe9ChQwf06dMHCxcuNOXhEBFRBTHMUoU1bty4SrM8WFtbw9nZuUo1ODg4IDk5GUlJSYiMjERiYiLGjx9fpW1WREFBQbVuv6rvbUX95z//wfjx42FnZ1fitWXLluGtt97C77//jry8vErv47333sPEiRPRq1cvbNu2DadPn8aCBQtw8uRJ/Pbbb1Upv1yPPfYYzpw5g5iYGGzevBl79+596B8IcXFxGDZsGIYOHYrDhw/jyJEjeOmll4pdsKXVajF+/Hg8//zzZW5n6tSp+OGHH1BYWGiy4yEiogoS9UxGRoYAIDIyMkq8lpubK86ePStyc3MNbXq9ENnZ0jz0+oof1+TJk8WYMWPKfD02Nlb06tVLKJVK4erqKt5++21RUFBgeD0zM1NMmjRJ2NjYCFdXV7Fw4ULh7+8vZsyYYVimefPm4uuvv/7f+6IXc+bMER4eHkKpVAo3Nzfx8ssvCyGE8Pf3FwCKPYQQYvny5UKtVherKyoqSvTs2VOoVCrRqFEjERISUuYxlLb+t99+W+J8btq0SXTr1k2oVCrRokUL8eGHHxY71oSEBOHr6ytUKpVo3769iImJEQDExo0bhRBCXL58WQAQa9asEQMGDBAqlUosX75cCCHETz/9JNq1aydUKpVo27at+P777w3bzc/PFy+++KJwdXUVKpVKNGvWTMybN++h79eD760QQly9elWMHj1a2NraCnt7ezF+/HiRkpJieH3OnDmia9euYsWKFaJ58+bCwcFBTJw4UWRmZpb5/hUWFgq1Wi02b95c4rVLly4Ja2trcffuXeHj4yNWrVr10Pe+yP3v3aFDhwQAsWjRolKXvXPnTpn1VcXZs2cFAHHkyBFD27Zt24RMJhNJSUllrufj4yPef//9Cu2jvPcgPz9fqFQqsWPHjlJfL+1nCxERla28vPYgSXtm9+7di+DgYDRp0qTUjypLExsbi+7du0OlUqFVq1YIDw+v1hpzcgA7O2keproJWVJSEkaMGIFevXrh5MmT+OGHH7Bs2TJ88sknhmVmzpyJAwcOICoqCjExMdi3bx+OHz9e5jYjIyPx9ddf48cff8Q///yDTZs2oXPnzgCADRs2wN3dHR999BGSk5ORnJxc6ja2bNmCsWPHYsSIEThx4gR27tyJ3r17V/i40tLSsHHjRigUCigUCgDAvn378OSTT2LGjBk4e/YsfvzxR4SHh+PTTz8FAOh0OoSEhMDGxgaHDh3C0qVL8d5775W6/VmzZmHGjBlISEhAUFAQVq1ahdmzZ+PTTz9FQkIC5s2bhw8++AC//vorAODbb79FVFQU1q1bh8TERKxatQqenp4Pfb8epNfrMWbMGKSnp2PPnj2IiYnBpUuXMHHixGLLXbx4EZs2bcLmzZuxefNm7NmzB5999lmZ79fff/+NjIwM9OzZs8Rry5cvx8iRI6FWq/H4449j2bJl5b/5ZVi1ahXs7OzwwgsvlPp6eWOmO3bsCDs7uzIfw4cPL3PduLg4ODo6Fju2wMBAyOVyHDp0qNR10tLScOjQITg7O6Nfv35wcXGBv78/9u/fX7GDvY9SqYS3tzf27dtn9LpERFQ1kt40QaPRoGvXrnjqqacwbty4hy5/+fJljBw5Es899xxWrVqFnTt34umnn4abmxuCgoJqoGLztHjxYnh4eOC7776DTCZDu3btcOPGDbz99tuYPXs2NBoNfv31V6xevRoBAQEA7oWbJk2alLnNa9euwdXVFYGBgbC0tESzZs0MQbRhw4ZQKBSwt7eHq6trmdv49NNP8cgjj2Du3LmGtq5du5Z7LBkZGbCzs4MQwnDL4VdeeQW2trYAgLlz52LWrFmYPHkyAKBly5b4+OOP8dZbb2HOnDmIiYnBxYsXERsba6jt008/xZAhQ0rs69VXXy32dTlnzhwsWLDA0NaiRQtDYJ48eTKuXbuG1q1bw8/PDzKZDM2bN6/Q+/WgnTt34tSpU7h8+TI8PDwAACtWrEDHjh1x5MgR9OrVC8C90BseHg57e3sA9y7s2rlzpyG4P+jq1atQKBQlhnoUbec///kPAOCRRx7B66+/jsuXL6NFixZlnovS/PPPP2jZsiUsLS2NWg8Atm7dWu5wDmtr6zJfS0lJKXFcFhYWaNiwIVJSUkpd59KlSwCADz/8EF999RW8vb2xYsUKBAQE4PTp02jdurVR9Tdp0gRXr141ah0iIqo6ScPs8OHDy+1tedCSJUvQokULLFiwAADQvn177N+/H19//XW1hVkbG6AC15BU275NISEhAX379i12a15fX19kZ2fj33//xZ07d1BQUFAsXKnVarRt27bMbY4fPx6LFi1Cy5YtMWzYMIwYMQLBwcGwsKj4l1R8fDyeeeYZo47F3t4ex48fR0FBAbZt24ZVq1YVC28nT57EgQMHirXpdDrk5eUhJycHiYmJ8PDwKBayywqV9/fyaTQaXLx4EdOmTStWc2FhIdRqNYB7F+ENGTIEbdu2xbBhwzBq1CgMHToUgHHvV0JCAjw8PAxBFrh3kZGjoyMSEhIMYdbT09MQZAHAzc0NaWlpZb53ubm5UKlUJW7RHBMTA41GgxEjRgAAnJycMGTIEPzyyy/4+OOPy9xeaYQQRi1/v/vDf03Q6/UAgOnTp2Pq1KkAgG7dumHnzp345ZdfMH/+fKO2Z21tbfgDi4joQUKY7hNXKej1esjlctjYAA/8GpGcWd3ONi4uDoGBgcXagoKC8Oqrr5a5Tn5+PvLz8w3PMzMzjdqnTAb8r9OP7uPh4YHExETs2LEDMTExeOGFF/Dll19iz549Fe6VK6+nrSxyuRytWrUCcO+PmYsXL+L55583XFiUnZ2NuXPnltrTb+xtRG3vO/FFV8X/9NNP8PHxKbZc0RCH7t274/Lly9i2bRt27NiBCRMmIDAwEBERESZ5vx704HoymcwQ0Erj5OSEnJwcaLVaKJVKQ/uyZcuQnp5e7Hzo9Xr8/fffmDt3LuRyORwcHKDRaAw/zIrcvXsXAAyBvk2bNti/fz8KCgqMPq6OHTuW27PZv39/bNu2rdTXXF1dSwT5wsJCpKenl/npgJubG4B7fyjcr3379pWaISM9PR1eXl5Gr0dEdZ8QgJ8fcPCg1JVUhkD37ifQp89/8csvT+HWLatal4vMajaDlJQUuLi4FGtzcXFBZmZmsal07jd//nyo1WrD4/7ervqiffv2iIuLK9ZrduDAAdjb28Pd3d3wsfCRI0cMr2dkZDx0Gi1ra2sEBwfj22+/RWxsLOLi4nDq1CkA98YQ6nS6ctfv0qULdu7cWYUjuzeude3atYbxvd27d0diYiJatWpV4iGXy9G2bVtcv34dqamphm3cf9xlcXFxQZMmTXDp0qUS273/o3gHBwdMnDgRP/30E9auXYvIyEjDdFTlvV/3a9++Pa5fv47r168b2s6ePYu7d++WCF7G8Pb2NmyryO3bt/HHH39gzZo1iI+PNzxOnDiBO3fu4K+//gIAtG3bFoWFhYiPjy+2zaL3vU2bNgCASZMmITs7G4sXLy61hqLwW5qtW7cWq+HBx88//1zmun379sXdu3dx7NgxQ9uuXbug1+tL/PFRxNPTE02aNEFiYmKx9vPnz1eql/j06dPo1q2b0esRUd2Xk2OeQVapzEdo6AaMHv0nnJ1vomfPh/++lIJZ9cxWxjvvvIOZM2canmdmZtbZQJuRkVEibDRq1AgvvPACFi1ahJdffhkvvfQSEhMTMWfOHMycORNyuRz29vaYPHky3nzzTTRs2BDOzs6YM2cO5HJ5iY+ki4SHh0On08HHxwc2NjZYuXIlrK2tDSHA09MTe/fuxSOPPAKVSgUnJ6cS25gzZw4CAgLg5eWFRx55BIWFhdi6dSvefvvtCh+zh4cHxo4di9mzZ2Pz5s2YPXs2Ro0ahWbNmiEsLAxyuRwnT57E6dOn8cknn2DIkCHw8vLC5MmT8cUXXyArKwvvv/8+AJR5rEXmzp2LV155BWq1GsOGDUN+fj6OHj2KO3fuYObMmVi4cCHc3NzQrVs3yOVyrF+/Hq6urnB0dHzo+3W/wMBAdO7cGY899hgWLVqEwsJCvPDCC/D39y/14q2Katy4Mbp37479+/cbgu1vv/2GRo0aYcKECSWOf8SIEVi2bBmGDRuGjh07YujQoXjqqaewYMECtGzZEomJiXj11VcxceJENG3aFADg4+ODt956C6+//jqSkpIwduxYNGnSBBcuXMCSJUvg5+eHGTNmlFpfVYYZtG/fHsOGDcMzzzyDJUuWoKCgAC+99BIeeeQRw9jvpKQkBAQEYMWKFejduzdkMhnefPNNzJkzB127doW3tzd+/fVXnDt3DhEREYZtX7t2Denp6bh27Rp0Op3he6xVq1aGKc6uXLmCpKSkEp8cERE9KDXVPD7xTUtLwZ9/rsedO+mQyWTw8xuM11/3NdkQSJOq7qkVKgr3Te9Tlv79+xebKkoIIX755Rfh4OBQ4f0YOzWXuZg8eXKJ6bAAiGnTpgkhKjc1V+/evcWsWbMMy9w/fdTGjRuFj4+PcHBwELa2tqJPnz7FpiWKi4sTXbp0ESqVqtypuSIjI4W3t7dQKpXCyclJjBs3rsxjLGtqpLi4OAFAHDp0SAghRHR0tOjXr5+wtrYWDg4Oonfv3mLp0qWG5Yum5lIqlaJdu3bizz//FABEdHS0EOL/p+Y6ceJEiX2tWrXKUG+DBg3EgAEDxIYNG4QQQixdulR4e3sLW1tb4eDgIAICAsTx48cr9H5Vdmqu+3399deiefPmZb5/QgixePFi0adPH8Pzzp07ixdeeKHUZdeuXSuUSqW4efOmEOLetFqvvPKK8PLyEtbW1qJ169birbfeEllZWaWuO2DAAGFvby9sbW1Fly5dxEcffVRtU3MJIcTt27fFo48+Kuzs7ISDg4OYOnVqsdqKzuvu3buLrTd//nzh7u4ubGxsRN++fcW+ffuKvV7W99b925k3b54ICgoqszZz/tlCRFWXnS3EvcEG9/5fm+n1enH48GHx8ccfiw8//FAsXLhQXLt2rcbrMGZqLpkQVbhiw4RkMhk2btxY7l2s3n77bWzdurXYR7OTJk1Ceno6oqOjK7SfzMxMqNVqZGRkwMHBodhreXl5hiu4jR1fWddoNBo0bdoUCxYswLRp06Qup1odOHAAfn5+uHDhQp0f85ibm4u2bdti7dq16Nu3r9Tl1AlarRatW7fG6tWr4evrW+oy/NlCVL9pNPem3ATuXVRem3tmb9++jcWLF0Ov16NNmzYYM2ZMjdzU50Hl5bUHSTrMIDs7GxcuXDA8v3z5MuLj49GwYUM0a9YM77zzDpKSkrBixQoAwHPPPYfvvvsOb731Fp566ins2rUL69atw5YtW6Q6hDrjxIkTOHfuHHr37o2MjAx89NFHAIAxY8ZIXJnpbdy4EXZ2dmjdujUuXLiAGTNmwNfXt84HWeDeuN0VK1bg1q1bUpdSZ1y7dg3vvvtumUGWiMicNGrUCEFBQdDpdOjTp89Dh+DVBpKG2aNHj2LQoEGG50VjWydPnozw8HAkJycXu6q4RYsW2LJlC1577TV88803cHd3x88//8w5Zk3kq6++QmJiIpRKJXr06IF9+/aVOtbV3GVlZeHtt9/GtWvX4OTkhMDAQMN0b/XBwIEDpS6hTim6EJCIyBwJIXD48GE0b97cMPuLMTcwqg1qzTCDmsJhBkRU0/izhah+q63DDHJzcxEVFYVz586hYcOGmD59erGpG6VkNsMMiIiIiKjm/fvvv4iIiEBGRgYUCgV8fHwqPe+51BhmS1HPOquJqJrxZwoR1RZCCMTFxWHnzp3Q6/Vo0KABwsLCyr2FfW3HMHufor9IcnJyKnV3KiKi0hTd5tZcez2IqG7QarWIjIw03BSpY8eOCA4OhkqlkriyqmGYvY9CoYCjo6Phtpg2NjZmcRUfEdVOQgjk5OQgLS0Njo6OhlsfExFJwdLSEoWFhVAoFBg2bBh69OhRJ3IOw+wDiq7ke/A+70REleXo6Gj42UJEVJOEENDpdLCwsIBMJsPYsWORnZ1dp34mMcw+QCaTwc3NDc7OzigoKJC6HCIyc5aWluyRJSJJaDQabNy4EWq1GsHBwQAAOzs7w6246wqG2TIoFAr+AiIiIiKzdOXKFURGRiI7OxsWFhbw8/NDgwYNpC6rWjDMEhEREdURer0e+/btw549eyCEgJOTE8aPH19ngyzAMEtERERUaUIA/5uwpEwaTc3Ukp2djQ0bNuDy5csAAG9vbwwfPrzW3AihujDMEhEREVWCEICfH3DwoNSV3LvQa8WKFbh58yYsLS0xcuRIdO3aVeqyagTDLBEREVEl5OQYF2R9fQEbm+qpRSaTITAwELt27UJYWBicnJyqZ0e1EMMsERERURWlpgK2tuUvY2MDmHJa16ysLKSnp6N58+YAgDZt2qBVq1aQy+Wm24kZYJglIiIiqiJb24eHWVO6cOECNm7cCL1ej+nTp8PR0REA6l2QBRhmiYiIiMyGXq/Hrl27cODAAQD3bvak1+slrkpaDLNEREREZiAjIwORkZG4fv06AKBnz54ICgqChUX9jnP1++iJiIiIzMD58+exadMm5ObmQqVSITg4GB07dpS6rFqBYZaIiIiolvvnn3+Qm5uLJk2aICwsrE7fBMFYDLNEREREtVxQUBAcHR3h4+NT74cVPKj+XfJGREREVMudO3cO69atM1zcZWFhAV9fXwbZUvAdISIiIqolCgsLERMTg8OHDwMATpw4gR49ekhcVe3GMEtERERUC6SnpyMiIgLJyckAgL59+8Lb21vaoswAwywRERGRxM6cOYM///wT+fn5sLa2RkhICNq0aSN1WWaBYZaIiIhIQvv27cOuXbsAAB4eHggNDYVarZa4KvPBC8CIiIiIJNSmTRtYWlrCz88PU6ZMYZA1EntmiYiIqE4QAsjJqbn9aTSVX/f27dto1KgRAMDFxQUvv/wy7O3tTVRZ/cIwS0RERGZPCMDPDzh4UOpKyldQUIDo6GjEx8dj6tSpcHd3BwAG2SpgmCUiIiKzl5MjXZD19QVsbB6+3M2bNxEREYG0tDQAQFJSkiHMUuUxzBIREVGdkpoK2NrW3P5sbACZrPxl4uPjsXXrVhQUFMDW1hbjxo1Dy5Yta6bAOo5hloiIiOoUW9uaDbPl0Wq12Lp1K06ePAkAaNGiBcaNGwc7OzuJK6s7GGaJiIiIqsnp06dx8uRJyGQyDBw4EH5+fpDLOZmUKTHMEhEREVWTbt26ISkpCZ07d4anp6fU5dRJ/NOAiIiIyETy8/MRExOD/Px8AIBMJkNwcDCDbDVizywRERGRCaSkpCAiIgK3b9+GRqNBSEiI1CXVCwyzRERERFUghMCxY8cQHR0NnU4HBwcHdO/eXeqy6g2GWSIiIqJKysvLw+bNm3HmzBkA925NO2bMGNhUZOJZMgmGWSIiIqJKSEtLw5o1a3Dnzh3I5XIEBgaiT58+kD1s0lkyKYZZIiIiokqwsbGBVquFWq1GWFgY7+YlEYZZIiIiogoqKCiApaUlAMDOzg6PPfYYHB0dYW1tLXFl9Ren5iIiIiKqgH///Rfff/89Tp8+bWhzc3NjkJUYwywRERFROYQQiIuLw/Lly5GRkYEDBw5ACCF1WfQ/HGZAREREVIacnBz88ccfOH/+PACgQ4cOCA4O5kVetQjDLBEREUlCCCAnxzTb0mhMs537Xb9+HREREcjMzIRCocCwYcPQo0cPBtlahmGWiIiIapwQgJ8fcPCg1JWU7s6dOwgPD4der0fDhg0xfvx4uLq6Sl0WlYJhloiIiGpcTk71BFlfX8AU9yto0KABfHx8kJ2djZEjR0KlUlV9o1QtGGaJiIhIUqmpgK2tabZlYwNUdhTAlStX0KBBA6jVagBAYGAgZDIZhxXUcgyzREREJClbW9OF2crQ6/XYt28f9uzZg6ZNm2LKlClQKBSQyznpkzlgmCUiIqJ6Kzs7Gxs2bMDly5cBAI0aNYJer4dCoZC4MqoohlkiIiKqly5fvozIyEhoNBpYWlpixIgR8Pb2lrosMhLDLBEREdUrer0ee/bswd69ewEAzs7OCAsLQ+PGjSWujCqDYZaIiIjqFb1ej8TERABAt27dMHz4cFhaWkpcFVUWwywRERHVKxYWFggLC0NycjI6d+4sdTlURQyzREREVKfp9Xrs2rULSqUSAwYMAAA4OTnByclJ4srIFBhmiYiIqM7KyMhAZGQkrl+/DplMho4dO6JRo0ZSl0UmxDBLREREddL58+exadMm5ObmQqVSITg4mEG2DmKYJSIiojpFp9Nh586diIuLAwC4ubkhLCwMDRs2lLgyqg4Ms0RERFRnCCGwcuVKXLlyBQDQu3dvDBkyBBYWjDx1Fc8sERER1RlF42JTUlIwevRotG/fXuqSqJoxzBIREZFZKywsRGZmpmEYQY8ePdCuXTvY2dlJXBnVBIZZIiIiMikhgJyc8pfRaEyzrzt37mD9+vXIycnB9OnTYW1tDZlMxiBbjzDMEhERkckIAfj5AQcPVv++zp49i6ioKOTn58Pa2hq3b9+Gu7t79e+YahWGWSIiIjKZnBzjgqyvL2BjY9w+CgsLsX37dhw9ehQA4OHhgdDQUKjVauM2RHUCwywRERFVi9RUwNa2/GVsbACZrOLbvH37NiIiIpCSkgIA8PX1xaBBg6BQKKpQKZkzhlkiIiKqFra2Dw+zxoqNjUVKSgpsbGwwduxYtGrVyrQ7ILPDMEtERERmY/jw4QCAIUOGwMHBQeJqqDaQS10AERERUVlu3ryJ3bt3QwgBALCxsUFoaCiDLBmwZ5aIiIhqpZMnT2LLli0oKChAw4YN0bVrV6lLolqIYZaIiIhqFa1Wi23btiE+Ph4A0KJFC3h5eUlbFNVaDLNERERUa6SlpWH9+vW4desWZDIZ/P390b9/f8jlHBlJpWOYJSIiolrh1KlTiIqKQmFhIezs7BAaGgpPT0+py6JajmGWiIiIagVbW1sUFhbCy8sLY8eOha2p5/WiOolhloiIiCSj1WqhVCoBAC1btsSUKVPQrFkzyIy5kwLVaxyAQkRERDVOCIGjR4/im2++QXp6uqG9efPmDLJkFIZZIiIiqlH5+fmIjIzEli1bkJOTg6NHj0pdEpkxycPs999/D09PT1hZWcHHxweHDx8ud/lFixahbdu2sLa2hoeHB1577TXk5eXVULVERERUFTdu3MCPP/6IM2fOQC6XY8iQIRgyZIjUZZEZk3TM7Nq1azFz5kwsWbIEPj4+WLRoEYKCgpCYmAhnZ+cSy69evRqzZs3CL7/8gn79+uH8+fOYMmUKZDIZFi5cKMEREBERUUUIIXD48GHExMRAp9NBrVYjLCwM7u7uUpdGZk7SntmFCxfimWeewdSpU9GhQwcsWbIENjY2+OWXX0pd/uDBg/D19cWkSZPg6emJoUOH4tFHH31oby4RERFJKz4+HtHR0dDpdGjXrh2mT5/OIEsmIVmY1Wq1OHbsGAIDA/+/GLkcgYGBiIuLK3Wdfv364dixY4bweunSJWzduhUjRowocz/5+fnIzMws9iAiIqKa1aVLFzRr1gzDhg3DhAkTYG1tLXVJVEdINszg1q1b0Ol0cHFxKdbu4uKCc+fOlbrOpEmTcOvWLfj5+UEIgcLCQjz33HN49913y9zP/PnzMXfuXJPWTkRERA8jcPbsKXTv3hEKhQIKhcIwNJDIlCS/AMwYsbGxmDdvHhYvXozjx49jw4YN2LJlCz7++OMy13nnnXeQkZFheFy/fr0GKyYiIqp/rK1z8eija7B160bs3r3b0M4gS9VBsp5ZJycnKBQKpKamFmtPTU2Fq6trqet88MEHeOKJJ/D0008DADp37gyNRoNnn30W7733Xqn3bVapVFCpVKY/ACIiIiohKek6nnsuAmp1JhQKBdRqtdQlUR0nWc+sUqlEjx49sHPnTkObXq/Hzp070bdv31LXycnJKRFYFQoFgHtXSRIREZE0hBDYv38/1qxZDrU6E7dvN8Rjjz2NXr16SV0a1XGSTs01c+ZMTJ48GT179kTv3r2xaNEiaDQaTJ06FQDw5JNPomnTppg/fz4AIDg4GAsXLkS3bt3g4+ODCxcu4IMPPkBwcLAh1BIREVHN0mg02LRpEy5cuAAAOHWqE/78cxTmz+cno1T9JA2zEydOxM2bNzF79mykpKTA29sb0dHRhovCrl27Vqwn9v3334dMJsP777+PpKQkNG7cGMHBwfj000+lOgQiIqJ6Lzc3F1evXoWFhQUGDx6ODz/sBoDjY6lmyEQ9+3w+MzMTarUaGRkZcHBwkLocIiKiOuHcuXNo0KAB7OxcYGd3ry07G7C1lbYuMk/G5DWzms2AiIiIpJednY2VK1fi6tWrhrZ27dqVmG6TqCYwzBIREVGFXbp0CUuWLMHFixcRFRUFvV4vdUlUz0k6ZpaIiIjMg16vx549e7B3714AQOPGjTF+/PhSp8UkqkkMs0RERFSurKwsbNiwAVeuXAEAdOvWDcOHD4elpaW0hRGBYZaIiIjKkZGRgaVLlyInJweWlpYYNWoUunTpInVZRAYMs0RERFQmBwcHtGjRArdu3cL48ePRqFEjqUsiKoZhloiIiIrJzMyEUqmElZUVZDIZgoODIZfLOayAaiWO2iYiIiKD8+fPY8mSJYiKijLcKl6lUjHIUq3FnlkiIqIqEgLIyZG6iqrR6XTYt28njh6NAwCkp99Feno+rKysjNqORlMd1RGVjWGWiIioCoQA/PyAgwelrqTy1Oq7CAuLhIfHvwCAQ4d646+/huD55xkTqPbjVykREVEV5OSYd5Bt1+4cxoz5A9bWecjLU+GPP8YgIaF9lbfr6wvY2JigQKKHYJglIiIykdRUwNZW6ioqrqCgAL/8sg1ZWXlwdW2KUaNC8f77DUyybRsbQCYzyaaIysUwS0REZCK2tuYVZgFLhIWF4ty5cwgICIBCoZC6ICKjMcwSERHVI2fPnkVhYaHhxgfNmjVDs2bNJK6KqPIYZomIiOqBwsJCbN++HUePHoWFhQWaNm3KGyBQncAwS0REVMfdvn0bERERSElJAQD4+PjA0dFR2qKITIRhloiIqA47ffo0/vzzT2i1WtjY2CAkJAStW7eWuiwik2GYJSIiqoOEENiyZQuOHTsG4N7Y2NDQUDg4OEhcGZFpMcwSERHVQTKZDDb/m+i1f//+GDhwIORy3sWe6h6GWSIiojpEq9VCqVQCAAYOHIjWrVvDw8ND4qqIqg//RCMiIqoDtFot/vjjD4SHh6OwsBAAIJfLGWSpzmPPLBERkZlLS0tDREQEbt68CZlMhitXrqBVq1ZSl0VUIxhmiYiIzJQQAvHx8di6dSsKCwthZ2eH0NBQeHp6Sl0aUY1hmCUiIjJD+fn52LJlC06dOgUA8PLywtixY2FrXvfTJaoyhlkiIqJyCAHk5JT9ukZTc7Xcb/PmzTh9+jRkMhkGDRoEPz8/yGQyaYohkhDDLBERURmEAPz8gIMHpa6kpMGDByM1NRWjRo1Cs2bNpC6HSDKczYCIiKgMOTkVD7K+vsD/pnWtFvn5+Thz5ozheYMGDfD8888zyFK9x55ZIiKiCkhNBcobjmpjA1TXp/zJyclYv3497ty5A5VKZZipgMMKiBhmiYiIKsTWtvwwWx2EEDhy5Aj++usv6HQ6qNVqWFlZ1WwRRLUcwywREVEtlJeXh6ioKCQkJAAA2rZtizFjxsDa2lriyohqF4ZZIiKiWiYpKQkRERG4e/cu5HI5hgwZAh8fHw4rICoFwywREVEtc+vWLdy9exeOjo4ICwtD06ZNpS6JqNZimCUiIqoFhBCGnteuXbtCq9Wic+fOHCNL9BCcmouIiEhi169fxy+//IKc++7O0KtXLwZZogpgmCUiIpKIEAIHDhzA8uXL8e+//2LXrl1Sl0RkdjjMgIiISAIajQabNm3ChQsXAACdOnXCkCFDJK6KyPwwzBIREdWwq1evIjIyEllZWbCwsMCwYcPQvXt3zlZAVAkMs0RERDXo3LlzWLduHYQQaNSoEcaPHw8XFxepyyIyWwyzRERENcjT0xOOjo7w8PDAyJEjoVQqpS6JyKwxzBIREVWz1NRUODs7QyaTwcrKCk8//TSsra05rIDIBDibARERUTXR6/WIjY3FkiVLcPToUUO7jY0NgyyRibBnlojIhIQA7psqlMycRlP5dbOysrBhwwZcuXIFAJCWlmaaooioGIZZIiITEQLw8wMOHpS6EpLaxYsXsXHjRmg0GlhaWmLUqFHo0qWL1GUR1UkMs0REJpKTwyBbV/n6AjY2D1+uaFjBvn37AAAuLi4ICwuDk5NTNVdIVH8xzBIRVYPUVMDWVuoqyFRsbICKDHFNTU3F/v37AQA9evRAUFAQLC0tq7k6ovqNYZaIqBrY2jLM1kdubm4YMmQI7O3t0alTJ6nLIaoXGGaJiIgqSafTITY2Fl26dEHjxo0BAH379pW4KqL6hVNzERERVUJGRgbCw8Oxf/9+REREQKfTSV0SUb3EnlkiIiIjJSYmYtOmTcjLy4NKpYK/vz8UCoXUZRHVSwyzREREFaTT6RATE4NDhw4BAJo0aYKwsDA0aNBA4sqI6i+GWSIiogrQaDRYvXo1bty4AQDo06cPAgMD2SNLJDGGWSIiogqwtraGhYUFrKysEBISgrZt20pdEhGBYZaIiKhMhYWFkMlkUCgUkMvlCA0NhV6vh6Ojo9SlEdH/cDYDIiKiUqSnp2PZsmWIiYkxtDk4ODDIEtUy7JklIiJ6wOnTp/Hnn39Cq9UiMzMTAwYMgE1F7mdLRDWOYZaIiOh/CgoKEB0djePHjwMAmjVrhtDQUAZZolqMYZaIiAjArVu3sH79eqSlpQEA+vfvj4EDB0Iu54g8otqMYZaIiOq9wsJCrFixAllZWbC1tcXYsWPh5eUldVlEVAFVCrN5eXmwsrIyVS1EZIaEAHJypK6idtBopK6AKsvCwgJBQUE4evQoxo0bB3t7e6lLIqIKMjrM6vV6fPrpp1iyZAlSU1Nx/vx5tGzZEh988AE8PT0xbdq06qiTiGohIQA/P+DgQakrITJeWloacnNz0bx5cwBAx44d0aFDB8hkMokrIyJjGD0Q6JNPPkF4eDi++OILKJVKQ3unTp3w888/m7Q4IqrdcnIYZEvj6wvweqHaSwiBEydO4KeffsK6deuQlZVleI1Blsj8GN0zu2LFCixduhQBAQF47rnnDO1du3bFuXPnTFocEZmP1FTA1lbqKmoHGxuAmah20mq12LJlC/7++28A92Yr4AVeRObN6DCblJSEVq1alWjX6/UoKCgwSVFEZH5sbRlmqXZLTU3F+vXrcfv2bchkMgwaNAh+fn7sjSUyc0aH2Q4dOmDfvn2GMUZFIiIi0K1bN5MVRkREZApCCBw/fhzR0dEoLCyEvb09QkNDS/weIyLzZHSYnT17NiZPnoykpCTo9Xps2LABiYmJWLFiBTZv3lwdNRIREVWaTCbD9evXUVhYiFatWmHs2LG8CQJRHSITQghjV9q3bx8++ugjnDx5EtnZ2ejevTtmz56NoUOHVkeNJpWZmQm1Wo2MjAw4ODhIXQ6RWdNoADu7e//PzuYwA6pdhBCGIQRarRZ///03evTowWEFRGbAmLxWqTBrzhhmiUyHYZZqIyEEjhw5gitXrmD8+PEMr0RmyJi8ZvQlnC1btsTt27dLtN+9exctW7Y0dnNEREQmk5eXh4iICGzbtg0JCQlISEiQuiQiqmZGj5m9cuUKdDpdifb8/HwkJSWZpCgiIiJjJSUlISIiAnfv3oVcLseQIUPQvn17qcsiompW4TAbFRVl+P/27duhVqsNz3U6HXbu3AlPT0+TFkdERPQwQggcOnQIMTEx0Ov1cHR0RFhYGJo2bSp1aURUAyocZkNCQgDcuyp08uTJxV6ztLSEp6cnFixYYNLiiIiIHmbbtm04cuQIAKB9+/YYPXo0rKysJK6KiGpKhcOsXq8HALRo0QJHjhyBk5NTtRVFRERUUV27dsXJkycREBCAXr168YIvonqGsxkQUaVxNgOSghACqampcHV1NbTl5ubC2tpawqqIyJSqdTYDANBoNNi6dSuWLFmCb7/9ttjDWN9//z08PT1hZWUFHx8fHD58uNzl7969ixdffBFubm5QqVRo06YNtm7dWpnDICIiM5OTk4Pff/8dP//8M1JSUgztDLJE9ZfRsxmcOHECI0aMQE5ODjQaDRo2bIhbt27BxsYGzs7OeOWVVyq8rbVr12LmzJlYsmQJfHx8sGjRIgQFBSExMRHOzs4lltdqtRgyZAicnZ0RERGBpk2b4urVq3B0dDT2MIiIyMxcvXoVkZGRyMrKgkKhwK1bt4r1zhJR/WT0MIOBAweiTZs2WLJkCdRqNU6ePAlLS0s8/vjjmDFjBsaNG1fhbfn4+KBXr1747rvvANwbl+vh4YGXX34Zs2bNKrH8kiVL8OWXX+LcuXOwtLQ0pmwDDjMgMh0OM6CaIITA/v37sXv3bggh0KhRI4wfPx4uLi5Sl0ZE1aRahxnEx8fj9ddfh1wuh0KhQH5+Pjw8PPDFF1/g3XffrfB2tFotjh07hsDAwP8vRi5HYGAg4uLiSl0nKioKffv2xYsvvggXFxd06tQJ8+bNK3Xe2yL5+fnIzMws9iAiIvOg0WiwatUq7Nq1C0IIdOnSBc8++yyDLBEZGB1mLS0tIZffW83Z2RnXrl0DAKjValy/fr3C27l16xZ0Ol2JH0guLi7FxkHd79KlS4iIiIBOp8PWrVvxwQcfYMGCBfjkk0/K3M/8+fOhVqsNDw8PjwrXSERE0vr7779x8eJFWFhYYPTo0QgJCYFSqZS6LCKqRYweM9utWzccOXIErVu3hr+/P2bPno1bt27ht99+Q6dOnaqjRgO9Xg9nZ2csXboUCoUCPXr0QFJSEr788kvMmTOn1HXeeecdzJw50/A8MzOTgZaIyEz06dMH6enp6NWrV6nXUhARGd0zO2/ePLi5uQEAPv30UzRo0ADPP/88bt68iR9//LHC23FycoJCoUBqamqx9genW7mfm5sb2rRpA4VCYWhr3749UlJSoNVqS11HpVLBwcGh2IOIiGqnrKwsbN68GQUFBQDu3ahn5MiRDLJEVCaje2Z79uxp+L+zszOio6MrtWOlUokePXpg586dhruL6fV67Ny5Ey+99FKp6/j6+mL16tXQ6/WGoQ7nz5+Hm5sbP3YiIjJzFy9exMaNG6HRaCCXyzFixAipSyIiM1CpeWZLc/z4cYwaNcqodWbOnImffvoJv/76KxISEvD8889Do9Fg6tSpAIAnn3wS77zzjmH5559/Hunp6ZgxYwbOnz+PLVu2YN68eXjxxRdNdRhERFTD9Ho9du3ahZUrV0Kj0cDZ2Rm9e/eWuiwiMhNG9cxu374dMTExUCqVePrpp9GyZUucO3cOs2bNwp9//omgoCCjdj5x4kTcvHkTs2fPRkpKCry9vREdHW24KOzatWuGHlgA8PDwwPbt2/Haa6+hS5cuaNq0KWbMmIG3337bqP0SEVHtkJmZicjISMPFxN27d8ewYcMqPf0iEdU/FZ5ndtmyZXjmmWfQsGFD3LlzB40aNcLChQvx8ssvY+LEiZgxYwbat29f3fVWGeeZJTIdzjNLVXHt2jWsXbsWOTk5UCqVCA4OrvYLiYnIPBiT1yrcM/vNN9/g888/x5tvvonIyEiMHz8eixcvxqlTp+Du7l7loomIqH5Rq9UQQsDV1RVhYWFo1KiR1CURkRmqcM+sra0tzpw5A09PTwghoFKpsHv3bvj6+lZ3jSbFnlki02HPLBkrLy8PVlZWhucpKSlwcnKChYXR1yMTUR1WLXcAy83NhY2NDYB7U6WoVCrDFF1EREQPk5iYiG+//RaJiYmGNldXVwZZIqoSo36C/Pzzz7D7XzdMYWEhwsPD4eTkVGyZV155xXTVERGR2dPpdNixYwf++9//AgCOHDmCtm3bSlwVEdUVFR5m4OnpCZlMVv7GZDJcunTJJIVVFw4zIDIdDjOgh7lz5w4iIyORlJQEAPDx8cGQIUOK3fyGiOhB1XIB2JUrV6paFxER1SMJCQn4448/kJ+fDysrK4wZMwbt2rWTuiwiqmM4UImIiEwuOTkZ69atAwC4u7sjNDQUjo6O0hZFRHUSwywREZmcm5sbevbsCaVSicGDB3NYARFVG4ZZonpICCAnp+rb0Wiqvg2qO86ePYtmzZoZLhQeMWLEQ6+1ICKqKoZZonpGCMDPDzh4UOpKqK4oKCjA9u3bcezYMbRo0QKPP/445HI5gywR1QiGWaJ6JifH9EHW1xf43zTUVM/cunULERERSE1NBQA0bdpU4oqIqL6pVJi9ePEili9fjosXL+Kbb76Bs7Mztm3bhmbNmqFjx46mrpGIqklqqmmm07KxAdgJV//8/fff2Lx5MwoKCmBjY4Nx48bBy8tL6rKIqJ4xOszu2bMHw4cPh6+vL/bu3YtPP/0Uzs7OOHnyJJYtW4aIiIjqqJOIqoGtLeeGJeMVFBRg27ZtOHHiBIB785CPGzcO9vb2EldGRPVRhW9nW2TWrFn45JNPEBMTA6VSaWgfPHiw4e4uRERUdwkhcP36dQCAv78/nnjiCQZZIpKM0T2zp06dwurVq0u0Ozs749atWyYpioiIah8hBGQyGZRKJcLCwqDRaNCyZUupyyKies7onllHR0ckJyeXaD9x4gQH/hMR1UFarRabNm0q9umbi4sLgywR1QpGh9lHHnkEb7/9NlJSUiCTyaDX63HgwAG88cYbePLJJ6ujRiIikkhqaip++uknnDx5Ert27UJ2drbUJRERFWP0MIN58+bhxRdfhIeHB3Q6HTp06ACdTodJkybh/fffr44aiYiohgkhcPz4cURHR6OwsBD29vYIDQ013BCBiKi2kAkhRGVWvHbtGk6fPo3s7Gx069YNrVu3NnVt1SIzMxNqtRoZGRlwcHCQuhyiGqfRAEV5JDubsxlQSfn5+di8eTNOnz4NAGjVqhVCQkJgyy8WIqohxuQ1o3tm9+/fDz8/PzRr1gzNmjWrdJFERFT76HQ6LFu2DDdv3oRMJkNAQAD69evHu3kRUa1l9JjZwYMHo0WLFnj33Xdx9uzZ6qiJiIgkolAo0K1bNzg4OGDq1Knw9fVlkCWiWs3oMHvjxg28/vrr2LNnDzp16gRvb298+eWX+Pfff6ujPiIiqmZ5eXm4ffu24XmfPn3w/PPPw8PDQ8KqiIgqxugw6+TkhJdeegkHDhzAxYsXMX78ePz666/w9PTE4MGDq6NGIiKqJjdu3MCPP/6I33//Hfn5+QAAmUwGKysriSsjIqoYo8fM3q9FixaYNWsWunbtig8++AB79uwxVV1ERFSNhBA4dOgQYmJioNfr4ejoiKysLKhUKqlLIyIySqXD7IEDB7Bq1SpEREQgLy8PY8aMwfz5801ZGxERVYPc3FxERUXh3LlzAIB27dphzJgx7I0lIrNkdJh95513sGbNGty4cQNDhgzBN998gzFjxsDGxqY66iMiIhP6999/ERERgYyMDCgUCgwdOhS9evXiRV5EZLaMDrN79+7Fm2++iQkTJsDJyak6aiIiomqyZ88eZGRkoEGDBggLC0OTJk2kLomIqEqMDrMHDhyojjqIiKgGjBkzBrGxsRgyZAjHxxJRnVChMBsVFYXhw4fD0tISUVFR5S47evRokxRGRERVd+3aNVy8eBGDBg0CANjZ2WHUqFESV0VEZDoVCrMhISFISUmBs7MzQkJCylxOJpNBp9OZqjYiIqokIQT279+P3bt3QwgBNzc3tGvXTuqyiIhMrkJhVq/Xl/p/IiKqfTQaDTZu3IiLFy8CALp06YKWLVtKXBURUfUw+qYJK1asMEysfT+tVosVK1aYpCgiIqqcK1euYMmSJbh48SIsLCwwevRohISEQKlUSl0aEVG1kAkhhDErKBQKJCcnw9nZuVj77du34ezsXOuHGWRmZkKtViMjIwMODg5Sl0NU4zQawM7u3v+zswFbW2nrIdOJi4tDTEwMhBBwcnLC+PHjS/ysJiIyB8bkNaNnMxBClDof4b///gu1Wm3s5oiIyEQaNmwIIQS8vb0xfPhw9sYSUb1Q4TDbrVs3yGQyyGQyBAQEwMLi/1fV6XS4fPkyhg0bVi1FEhFR6fLy8gx37mrbti2eeeYZzh1LRPVKhcNs0SwG8fHxCAoKgl3R55QAlEolPD09ERoaavICiYioJL1ej9jYWBw7dgzPPvus4ZMxBlkiqm8qHGbnzJkDAPD09MTEiRN5D28iIolkZmZiw4YNuHr1KgDg7Nmz6Nu3r8RVERFJw+gxs5MnT66OOoiIqAIuXLiAjRs3IicnB0qlEsHBwejUqZPUZRERSaZCYbZhw4Y4f/48nJyc0KBBg1IvACuSnp5usuKIiOgenU6H3bt3G24p7urqirCwMDRq1EjiyoiIpFWhMPv111/D3t7e8P/ywiwREZneoUOHDEG2V69eGDp0aLELcYmI6iuj55k1d5xnluo7zjNrngoKCrBy5Ur4+PigQ4cOUpdDRFStjMlrRt8B7Pjx4zh16pTh+R9//IGQkBC8++670Gq1xldLREQl6HQ6HD161HALcUtLS0yZMoVBlojoAUaH2enTp+P8+fMAgEuXLmHixImwsbHB+vXr8dZbb5m8QCKi+ubu3btYvnw5tmzZgn379hnaOcSLiKgko8Ps+fPn4e3tDQBYv349/P39sXr1aoSHhyMyMtLU9RER1SsJCQn48ccfkZSUBCsrK7i4uEhdEhFRrVap29kWfey1Y8cOjBo1CgDg4eGBW7dumbY6IqJ6orCwEDExMTh8+DAAwN3dHaGhoXB0dJS2MCKiWs7oMNuzZ0988sknCAwMxJ49e/DDDz8AAC5fvsweBCKiSkhPT0dERASSk5MBAH379kVAQAAUCoXElRER1X5Gh9lFixbhsccew6ZNm/Dee++hVatWAICIiAj069fP5AUSEdV1Wq0WaWlpsLa2RkhICNq0aSN1SUREZsNkU3Pl5eVBoVDA0tLSFJurNpyai+o7Ts1VOwghil3Qde7cObi5uUGtVktYFRFR7WBMXqv0jNvHjh1DQkICAKBDhw7o3r17ZTdFRFSv3L59Gxs2bMCIESPQtGlTAEC7du0kroqIyDwZHWbT0tIwceJE7Nmzx3Bhwt27dzFo0CCsWbMGjRs3NnWNRER1xqlTp7B582ZotVps27YN06ZN45RbRERVYPTUXC+//DKys7Nx5swZpKenIz09HadPn0ZmZiZeeeWV6qiRiMjsFRQUICoqChs2bIBWq4WnpycmTpzIIEtEVEVG98xGR0djx44daN++vaGtQ4cO+P777zF06FCTFkdEVBfcvHkTERERSEtLAwD4+/tjwIABkMuN7k8gIqIHGB1m9Xp9qRd5WVpaGuafJSKie9LS0vDzzz+joKAAtra2CA0NRYsWLaQui4iozjC6W2Dw4MGYMWMGbty4YWhLSkrCa6+9hoCAAJMWR0Rk7ho3bowWLVqgRYsWeO655xhkiYhMzOie2e+++w6jR4+Gp6cnPDw8AADXr19Hp06dsHLlSpMXSERkbtLS0uDo6AilUgmZTIbQ0FBYWFhwWAERUTUwOsx6eHjg+PHj2Llzp2Fqrvbt2yMwMNDkxRERmRMhBE6cOIFt27ahQ4cOCAkJgUwmg1KplLo0IqI6y6gwu3btWkRFRUGr1SIgIAAvv/xyddVFRGRW8vPzsWXLFpw6dQoAkJOTA51OBwuLSk/nTUREFVDhn7I//PADXnzxRbRu3RrW1tbYsGEDLl68iC+//LI66yMiqvVSUlKwfv16pKenQyaTISAgAP369eO0W0RENaDCt7Pt2LEjJkyYgDlz5gAAVq5cienTp0Oj0VRrgabG29mSORICyMkxzbY0GsDF5d7/eTvbqhFC4OjRo9i+fTt0Oh0cHBwQFhZmuJ6AiIgqx5i8VuEwa21tjYSEBHh6egK4N0WXtbU1rly5Ajc3tyoXXVMYZsncCAH4+QEHD5p+2wyzVZObm4vvv/8eGo0Gbdq0wZgxY2BjYyN1WUREZs+YvFbhYQb5+fmwve+3nlwuh1KpRG5ubuUrJaKHysmpniDr6wswd1WNtbU1xo0bh9TUVPTp04fDCoiIJGDUlQkffPBBsV4HrVaLTz/9FGq12tC2cOFC01VHRMWkppquJ9XGBmD2Mo4QAocPH4a9vT06dOgAAGjZsiVatmwpcWVERPVXhcPsgAEDkJiYWKytX79+uHTpkuE5eyWIqpetLYcFSCU3NxdRUVE4d+4clEol3N3dOVSJiKgWqHCYjY2NrcYyiIhqr3///RcRERHIyMiAQqFAQEAA7O3tpS6LiIhQiZsmEBHVF0IIxMXFYefOndDr9WjQoAHCwsLQpEkTqUsjIqL/YZglIiqFXq/H2rVrcf78eQD3picMDg6GSqWSuDIiIrofwywRUSnkcjkaNmwIhUKBYcOGoUePHrwugIioFmKYJSL6HyEE8vPzYWVlBQAIDAxE9+7d0bhxY4krIyKissilLoCIqDbQaDRYvXo1Vq9eDZ1OBwBQKBQMskREtVylwuy+ffvw+OOPo2/fvkhKSgIA/Pbbb9i/f79JiyMiqglXrlzBjz/+iAsXLiA5ORkpKSlSl0RERBVkdJiNjIxEUFAQrK2tceLECeTn5wMAMjIyMG/ePJMXSERUXfR6Pfbs2YMVK1YgKysLTk5OeOaZZ9C0aVOpSyMiogoyOsx+8sknWLJkCX766SdYWloa2n19fXH8+HGTFkdEVF2ys7OxcuVKxMbGQggBb29vPPPMM3B2dpa6NCIiMoLRF4AlJiZiwIABJdrVajXu3r1ripqIiKrdxo0bcfnyZVhaWmLkyJHo2rWr1CUREVElGN0z6+rqigsXLpRo379/f6XvT/7999/D09MTVlZW8PHxweHDhyu03po1ayCTyRASElKp/RJR/TV8+HC4u7vj2WefZZAlIjJjRofZZ555BjNmzMChQ4cgk8lw48YNrFq1Cm+88Qaef/55owtYu3YtZs6ciTlz5uD48ePo2rUrgoKCkJaWVu56V65cwRtvvIH+/fsbvU8iqn+ysrJw6tQpw3MnJyc89dRTcHJykrAqIiKqKqOHGcyaNQt6vR4BAQHIycnBgAEDoFKp8MYbb+Dll182uoCFCxfimWeewdSpUwEAS5YswZYtW/DLL79g1qxZpa6j0+nw2GOPYe7cudi3bx+HN5BZEwLIySn7dY2m5mqpqy5cuICNGzciNzcXDg4OaN68OQDwJghERHWA0WFWJpPhvffew5tvvokLFy4gOzsbHTp0gJ2dndE712q1OHbsGN555x1Dm1wuR2BgIOLi4spc76OPPoKzszOmTZuGffv2lbuP/Px8w4wLAJCZmWl0nUTVRQjAzw84eFDqSuomvV6PXbt24cCBAwDuDZOqzM8qIiKqvSp9BzClUokOHTpUaee3bt2CTqeDi4tLsXYXFxecO3eu1HX279+PZcuWIT4+vkL7mD9/PubOnVulOomqS05OxYOsry9gY1O99dQlGRkZiIyMxPXr1wEAPXv2RFBQECwseONDIqK6xOif6oMGDSr3o7ldu3ZVqaDyZGVl4YknnsBPP/1U4XFu77zzDmbOnGl4npmZCQ8Pj+oqkajSUlMBW9uyX7exAfipeMWcP38emzZtQm5uLlQqFYKDg9GxY0epyyIiompgdJj19vYu9rygoADx8fE4ffo0Jk+ebNS2nJycoFAokJqaWqw9NTUVrq6uJZa/ePEirly5guDgYEObXq8HAFhYWCAxMRFeXl7F1lGpVFCpVEbVRSQFW9vywyxVXEZGBnJzc+Hm5oawsDA0bNhQ6pKIiKiaGB1mv/7661LbP/zwQ2RnZxu1LaVSiR49emDnzp2G6bX0ej127tyJl156qcTy7dq1K3Y1MgC8//77yMrKwjfffMMeV6J6TAhh+NSoZ8+esLS0RKdOnTisgIiojjPZT/nHH38cvXv3xldffWXUejNnzsTkyZPRs2dP9O7dG4sWLYJGozHMbvDkk0+iadOmmD9/PqysrNCpU6di6zs6OgJAiXYiqj/OnTuHvXv34sknn4SVlRVkMlmJT5GIiKhuMlmYjYuLg5WVldHrTZw4ETdv3sTs2bORkpICb29vREdHGy4Ku3btGuRyo6fDJaJ6oLCwEDt27MChQ4cAAAcPHsTgwYMlroqIiGqSTAghjFlh3LhxxZ4LIZCcnIyjR4/igw8+wJw5c0xaoKllZmZCrVYjIyMDDg4OUpdD9ZxGAxTNFJWdzTGzxkhPT0dERASSk5MBAH379kVAQAAUCoXElRERUVUZk9eM7plVq9XFnsvlcrRt2xYfffQRhg4dauzmiIiMdubMGfz555/Iz8+HtbU1QkJC0KZNG6nLIiIiCRgVZnU6HaZOnYrOnTujQYMG1VUTEVGZjh07hs2bNwMAPDw8EBYWxk9ZiIjqMaMGoyoUCgwdOpS3jyUiybRv3x4ODg7w8/PDlClTGGSJiOo5o6+s6tSpEy5dulQdtRARlaroLl4AYGNjgxdeeAEBAQG8OJSIiIwPs5988gneeOMNbN68GcnJycjMzCz2ICIylYKCAkRFReGXX34pdgtr3giFiIiKVHjM7EcffYTXX38dI0aMAACMHj262G1tiyYs1+l0pq+SiOqdmzdvIiIiAmlpaQDu3c6aiIjoQRWemkuhUCA5ORkJCQnlLufv72+SwqoLp+ai2oRTc5Xu5MmT2LJlCwoKCmBra4tx48ahZcuWUpdFREQ1pFqm5irKvLU9rBKR+dJqtdi2bZthSEHLli0xduxY2BUlfiIiogcYNTXX/cMKiIhM7caNG4iPj4dMJsPAgQPh5+fHi7yIiKhcRoXZNm3aPDTQpqenV6kgIqq/PD09MXToULi5ucHT01PqcoiIyAwYFWbnzp1b4g5gRESVlZ+fj7/++gu+vr5o2LAhgHu3pSUiIqooo8LsI488Amdn5+qqhYjqkZSUFEREROD27dtIS0vDU089xaFMRERktAqHWf6SISJTEELg2LFjiI6Ohk6ng4ODA4YMGcKfMUREVClGz2ZARFRZeXl52Lx5M86cOQPg3jj8MWPGwMbGRuLKiIjIXFU4zOr1+uqsg4jquDt37uC3337DnTt3IJfLERgYiD59+rBHloiIqsSoMbNERJXl4OAAa2tr6PV6hIWFwd3dXeqSiIioDmCYJaJqk5eXB6VSCblcDoVCgQkTJkCpVMLa2lrq0oiIqI7gbOREVC2SkpLw448/Yvfu3YY2tVrNIEtERCbFMEtEJiWEQFxcHH755RfcvXsXZ8+ehVarlbosIiKqozjMgIhMJjc3F5s2bcL58+cBAB06dEBwcDCUSqXElRERUV3FMEtEJnH9+nVEREQgMzMTCoUCw4YNQ48ePThbARERVSuGWSKqsry8PKxatQr5+flo2LAhxo8fD1dXV6nLIiKieoBhloiqzMrKCsOGDcOlS5cwcuRIqFQqqUsiIqJ6gmGWiCrl6tWrkMvl8PDwAAB4e3uja9euHFZAREQ1imGWiIyi1+uxf/9+xMbGws7ODs8995zhdrQMskREVNMYZomowrKzs7Fx40ZcunQJANCyZUtYWPDHCBERSYe/hYioQi5fvozIyEhoNBpYWlpixIgR8Pb2lrosIiKq5xhmiahcQgjExsZi7969AABnZ2eEhYWhcePGEldGRETEMEtEFXDr1i0AQLdu3TB8+HBYWlpKXBEREdE9DLNEVCohBGQyGWQyGYKDg9GxY0d06NBB6rKIiIiKkUtdABHVLnq9Hjt27EBERASEEADuzSPLIEtERLURe2aJyCAjIwORkZG4fv06gHtzyXp6ekpbFBERUTkYZokIAHD+/Hls2rQJubm5UKlUCA4OZpAlIqJaj2GWqJ7T6XTYuXMn4uLiAABubm4ICwtDw4YNJa6MiIjo4Rhmieq5yMhIJCQkAAB69+6NIUOG8EYIRERkNvgbi6ie8/HxwdWrVxEcHIx27dpJXQ4REZFRGGaJ6pnCwkKkpKTA3d0dANC8eXPMmDEDSqVS4sqIiIiMx6m5iOqRO3fu4JdffsGKFStw8+ZNQzuDLBERmSv2zBLVE2fPnkVUVBTy8/NhbW2N7Oxs3pKWiIjMHsMsUR1XWFiI7du34+jRowAADw8PhIaGQq1WS1wZERFR1THMEtVht2/fRkREBFJSUgAAvr6+GDRoEBQKhcSVERERmQbDLJmMEEBOjtRVmBeNpnq3//fffyMlJQU2NjYYO3YsWrVqVb07JCIiqmEMs2QSQgB+fsDBg1JXQvfz9/eHVqtF37594eDgIHU5REREJsfZDMgkcnIYZKvC1xewsan6dm7duoVNmzahsLAQACCXyxEUFMQgS0REdRZ7ZsnkUlMBW1upqzAvNjaATFa1bZw8eRJbtmxBQUEBHBwcMHjwYNMUR0REVIsxzJLJ2doyzNYkrVaLbdu2IT4+HgDQokUL9O7dW9qiiIiIagjDLJEZS0tLQ0REBG7evAmZTAZ/f3/0798fcjlHEBERUf3AMEtkps6dO4fIyEgUFhbCzs4OoaGh8PT0lLosIiKiGsUwS2SmnJ2doVAo0Lx5c4wdOxa2HNtBRET1EMMskRnRaDSG0NqwYUNMmzYNTk5OkFX16jEiIiIzxYF1RGZACIGjR49i0aJFuHjxoqG9cePGDLJERFSvsWeWqJbLy8vD5s2bcebMGQDA6dOn4eXlJXFVREREtQPDLFEtduPGDURERODOnTuQy+UICAhA3759pS6LiIio1mCYJaqFhBA4fPgwYmJioNPpoFarERYWBnd3d6lLIyIiqlUYZolqocuXLyM6OhoA0K5dO4wePRrW1tYSV0VERFT7MMwS1UItW7ZE9+7d4ezsjN69e/MiLyIiojIwzBLVAkWzFXTs2BE2NjYAgODgYImrIiIiqv04NReRxHJycrBmzRps3boVmzZtghBC6pKIiIjMBntm6zEhgJwc02xLozHNduqb69evIyIiApmZmVAoFGjdurXUJREREZkVhtl6SgjAzw84eFDqSuonIQQOHDiAXbt2QQiBhg0bYvz48XB1dZW6NCIiIrPCMFtP5eRUT5D19QX+N+STypCTk4ONGzfiwoULAIBOnTph1KhRUKlUEldGRERkfhhmCampgK2tabZlYwPwwvvyyeVy3Lp1CxYWFhg+fDi6devG2QqIiIgqiWGWYGtrujBLpSu6qEsmk8HKygoTJkyAXC6Hi4uLxJURERGZN85mQFTNsrOzsXLlShw9etTQ5ubmxiBLRERkAuyZJapGly9fRmRkJDQaDZKTk9GlSxeOjSUiIjIhhlmiaqDX67Fnzx7s3bsXANC4cWOMHz+eQZaIiMjEGGaJTCwrKwsbNmzAlStXAADdunXD8OHDYWlpKW1hREREdRDDLJEJabVaLF26FNnZ2bC0tMSoUaPQpUsXqcsiIiKqsxhmiUxIqVSiV69eOHv2LMaPH49GjRpJXRIREVGdxjBLVEWZmZkoKCgwBFc/Pz/069cPFhb89iIiIqpunJqLqArOnz+PJUuWYN26dSgoKABw76YIDLJEREQ1g79xiSpBp9Nh586diIuLAwA4OjoiNzeXF3kRERHVMIZZIiPdvXsXkZGR+PfffwEAvXv3xpAhQ9gbS0REJIFaMczg+++/h6enJ6ysrODj44PDhw+XuexPP/2E/v37o0GDBmjQoAECAwPLXZ7IlM6dO4cff/wR//77L1QqFSZMmIDhw4czyBIREUlE8jC7du1azJw5E3PmzMHx48fRtWtXBAUFIS0trdTlY2Nj8eijj2L37t2Ii4uDh4cHhg4diqSkpBqunOobIQTi4uKQl5eHJk2aYPr06Wjfvr3UZREREdVrMiGEkLIAHx8f9OrVC9999x2Ae3dO8vDwwMsvv4xZs2Y9dH2dTocGDRrgu+++w5NPPvnQ5TMzM6FWq5GRkQEHB4cq119bCQHk5JT9ukYDuLjc+392NmBrWzN1mbuMjAwcPXoUAwcOhEKhkLocIiKiOsmYvCZpz6xWq8WxY8cQGBhoaJPL5QgMDDRcWPMwOTk5KCgoQMOGDUt9PT8/H5mZmcUedZ0QgJ8fYGdX9qMoyFL5zp49i927dxueq9VqBAQEMMgSERHVEpKG2Vu3bkGn08HlgWTl4uKClJSUCm3j7bffRpMmTYoF4vvNnz8farXa8PDw8Khy3bVdTg5w8GDFlvX1BWxsqrcec1RYWIgtW7Zg/fr12Lt3Ly5fvix1SURERFQKs75q5bPPPsOaNWsQGxsLKyurUpd55513MHPmTMPzzMzMehFoi6Smlj+EwMYGkMlqrh5zcPv2bURERBj+oPL19UWzZs0kroqIiIhKI2mYdXJygkKhQGpqarH21NRUuLq6lrvuV199hc8++ww7duxAly5dylxOpVJBpVKZpF5zZGvL8bDGOHXqFDZv3gytVgsbGxuMHTsWrVq1krosIiIiKoOkwwyUSiV69OiBnTt3Gtr0ej127tyJvn37lrneF198gY8//hjR0dHo2bNnTZRK9cD27duxYcMGaLVaNG/eHNOnT2eQJSIiquUkH2Ywc+ZMTJ48GT179kTv3r2xaNEiaDQaTJ06FQDw5JNPomnTppg/fz4A4PPPP8fs2bOxevVqeHp6Gj4KtrOzg52dnWTHQebP3d0dANC/f38MHDgQcrnkM9cRERHRQ0geZidOnIibN29i9uzZSElJgbe3N6Kjow0XhV27dq1YqPjhhx+g1WoRFhZWbDtz5szBhx9+WJOlUx2QnZ1t+COoY8eOcHFxgZOTk8RVERERUUVJPs9sTasP88xqNPem3wI4h2xZtFottm3bhn/++QfPPfcce/WJiIhqEWPymuQ9s0Q1LS0tDREREbh58yZkMhkuXbpU7kWEREREVHsxzFK9IYRAfHw8tm7disLCQtjZ2SE0NBSenp5Sl0ZERESVxDBL9YJWq8XmzZtx6tQpAICXlxfGjh0LW47BICIiMmsMs1Qv7N27F6dOnYJMJsOgQYPg5+cHGe8WQUREZPYYZqleGDBgAJKTk+Hv78+7eREREdUhnEiT6qT8/HwcPHgQRZN1KJVKPPHEEwyyREREdQx7ZqnOSU5ORkREBNLT0wEA/fr1k7giIiIiqi4Ms1RnCCFw5MgR/PXXX9DpdFCr1eyJJSIiquMYZqlOyMvLQ1RUFBISEgAAbdu2xZgxY2BtbS1xZURERFSdGGbJ7N24cQPr16/H3bt3IZfLMWTIEPj4+HC2AiIionqAYZbMnhACmZmZcHR0RFhYGJo2bSp1SURERFRDGGbJLOn1esjl9ybjaNq0KSZOnIhmzZrByspK4sqIiIioJnFqLjI7169fx+LFi5GSkmJoa9OmDYMsERFRPcQwS2ZDCIEDBw5g+fLluH37Nnbt2iV1SURERCQxDjMgs6DRaLBp0yZcuHABANCpUyeMGjVK4qqIiIhIagyzVOtdvXoVkZGRyMrKgoWFBYYNG4bu3btztgIiIiJimKXa7dq1a/j1118hhECjRo0wfvx4uLi4SF0WERER1RIMs1Srubu7w9PTE/b29hg5ciSUSqXUJREREVEtwjBLtc61a9fg5uYGS0tLyOVyPProo7C0tJS6LCIiIqqFOJsB1Rp6vR6xsbFYvnw5tm/fbmhnkCUiIqKysGeWaoWsrCxs2LABV65cAQDodLpiN0YgIiIiKg3DLEnu4sWL2LBhA3JycmBpaYlRo0ahS5cuUpdFREREZoBhliSj1+uxe/du7N+/HwDg4uKCsLAwODk5SVwZERERmQuGWZKMRqPBsWPHAAA9evRAUFAQx8cSERGRURhmawkhgJwc02xLozHNdqqbvb09QkJCoNVq0alTJ6nLISIiIjPEMFsLCAH4+QEHD0pdSfXS6XTYtWsXmjVrhrZt2wIA2rRpI3FVREREZM54qXgtkJNTPUHW1xewsTH9disjIyMD4eHhOHjwIP744w/k5eVJXRIRERHVAeyZrWVSUwFbW9Nsy8YGkMlMs62qSExMxKZNm5CXlweVSoXg4GBYWVlJXRYRERHVAQyztYytrenCrNR0Oh1iYmJw6NAhAECTJk0QFhaGBg0aSFwZERER1RUMs1QtCgoKEB4ejhs3bgAA+vTpg8DAQCgUCokrIyIiorqEYZaqhaWlJVxdXZGeno6QkBDDBV9EREREpsQwSyZTWFiIgoICWFtbAwCGDRuGAQMGQK1WS1wZERER1VWczYBMIj09HcuWLcP69euh1+sB3OudZZAlIiKi6sSeWaqy06dP488//4RWq4W1tTXu3LmDRo0aSV0WERER1QMMs1RpBQUFiI6OxvHjxwEAzZo1Q2hoKBwcHCSujIiIiOoLhlmqlFu3biEiIgKpqakAgP79+2PgwIGQyzlyhYiIiGoOwywZTQiBDRs2IDU1FTY2Nhg3bhy8vLykLouIiIjqIYZZMppMJsPo0aOxc+dOjB49Gvb29lKXRERERPUUPxOmCklLS8Pff/9teO7q6orHHnuMQZaIiIgkxZ5ZKpcQAvHx8di6dSv0ej0aNWqEpk2bSl0WEREREQCGWSqHVqvFli1bDD2yLVu2hKOjo7RFEREREd2HYZZKlZqaivXr1+P27duQyWQYNGgQ/Pz8IJPJpC6NiIiIyIBhlko4fvw4tm7dCp1OB3t7e4SGhqJ58+ZSl0VERERUAsMslZCXlwedTodWrVph7NixsLGxkbokIiIiolIxzBIAQK/XG2540LdvX6jVanTo0IHDCoiIiKhW49Rc9ZwQAocPH8bSpUuh1WoB3JtHtmPHjgyyREREVOuxZ7Yey8vLQ1RUFBISEgDcGyvbp08fiasiIiIiqjiG2XoqKSkJERERuHv3LuRyOYYMGQIfHx+pyyIiIiIyCsNsPSOEwKFDhxATEwO9Xg9HR0eEhYXxRghERERklhhm65m9e/ciNjYWANC+fXuMHj0aVlZW0hZFREREVEkMs/VMjx49cOLECfTr1w+9evXiRV5ERERk1hhm6zghBC5dugQvLy8AgJ2dHV566SVYWPDUExERkfnj1Fx1WE5ODn7//XesXLkSZ86cMbQzyBIREVFdwVRTR129ehWRkZHIysqCQqFAQUGB1CURERERmRzDbB0jhMD+/fuxe/duCCHQqFEjjB8/Hi4uLlKXRkRERGRyDLN1iEajwYYNG3Dp0iUAQJcuXTBy5EgolUqJKyMiIiKqHgyzdUhSUhIuXboECwsLjBgxAt7e3pytgIiIiOo0htk6pE2bNhg6dCi8vLzg7OwsdTlERERE1Y6zGZixrKwsrFu3DhkZGYa2vn37MsgSERFRvcGeWTN18eJFbNy4ERqNBlqtFo8//rjUJRERERHVOIZZM6PX6xEbG4t9+/YBAJydnTFs2DCJqyIiIiKSBsOsGcnMzERkZCSuXbsGAOjevTuGDRsGS0tLiSsjIiIikgbDrJlISUnBihUrkJubC6VSieDgYHTq1EnqsoiIiIgkxTBrJho1agR7e3uo1WqEhYWhUaNGUpdEREREJDmG2VosKysLdnZ2kMlksLS0xKRJk2BrawsLC542IiIiIoBTc9VaiYmJWLx4seFCLwBQq9UMskRERET3YZitZXQ6HbZv3441a9YgLy8P//zzD/R6vdRlEREREdVK7OarRRwd7+D33yORkpIEAPDx8cGQIUMgl/NvDiIiIqLSMMzWEu3bJ2DMmD+QkpIPKysrjBkzBu3atZO6LCIiIqJajWG2FsjOzkJoaCQsLHRwc3PHhAmhcHR0lLosIiIiolqPYbYWsLOzR3T0MDRokI6IiAA4OCikLomIiIjILDDMSuTMmTNwdHRE06ZNAQBHj/YEACiYY4mIiIgqjGG2hhUUFGD79u04duwYHB0dMX36dABWUpdFREREZJZqxWXy33//PTw9PWFlZQUfHx8cPny43OXXr1+Pdu3awcrKCp07d8bWrVtrqNKquXXrFpYtW4Zjx44BADp16gSlUilxVURERETmS/Iwu3btWsycORNz5szB8ePH0bVrVwQFBSEtLa3U5Q8ePIhHH30U06ZNw4kTJxASEoKQkBCcPn26his3zt9//42lS5ciNTUVNjY2ePzxxxEQEMBpt4iIiIiqQCaEEFIW4OPjg169euG7774DAOj1enh4eODll1/GrFmzSiw/ceJEaDQabN682dDWp08feHt7Y8mSJQ/dX2ZmJtRqNTIyMuDg4GC6AymFEEBmZiF27NiC06fjAQAeHp4YOXIc7OzsDctpNICLy73/Z2cDtrbVWhYRERFRrWZMXpO0W1Cr1eLYsWMIDAw0tMnlcgQGBiIuLq7UdeLi4ootDwBBQUFlLp+fn4/MzMxij5qSkwM0aKDAhg0aCAHExvrj6aefgKurPezsYHgUBVkiIiIiMo6kF4DdunULOp0OLg+kORcXF5w7d67UdVJSUkpdPiUlpdTl58+fj7lz55qm4EoQQoZNm0Lg7JyGK1c8y13W1xewsamZuoiIiIjqgjo/m8E777yDmTNnGp5nZmbCw8OjRvZtY3Nv2ABgA8CzQsvLZNVcFBEREVEdImmYdXJygkKhQGpqarH21NRUuLq6lrqOq6urUcurVCqoVCrTFGwkmYzjX4mIiIiqk6RjZpVKJXr06IGdO3ca2vR6PXbu3Im+ffuWuk7fvn2LLQ8AMTExZS5PRERERHWX5MMMZs6cicmTJ6Nnz57o3bs3Fi1aBI1Gg6lTpwIAnnzySTRt2hTz588HAMyYMQP+/v5YsGABRo4ciTVr1uDo0aNYunSplIdBRERERBKQPMxOnDgRN2/exOzZs5GSkgJvb29ER0cbLvK6du1asblY+/Xrh9WrV+P999/Hu+++i9atW2PTpk3o1KmTVIdARERERBKRfJ7ZmlaT88wSERERkfHMZp5ZIiIiIqKqYJglIiIiIrPFMEtEREREZothloiIiIjMFsMsEREREZkthlkiIiIiMlsMs0RERERkthhmiYiIiMhsMcwSERERkdlimCUiIiIis8UwS0RERERmi2GWiIiIiMwWwywRERERmS0LqQuoaUIIAEBmZqbElRARERFRaYpyWlFuK0+9C7NZWVkAAA8PD4krISIiIqLyZGVlQa1Wl7uMTFQk8tYher0eN27cgL29PWQyWbXvLzMzEx4eHrh+/TocHByqfX9kejyH5o/n0PzxHJo3nj/zV9PnUAiBrKwsNGnSBHJ5+aNi613PrFwuh7u7e43v18HBgd/AZo7n0PzxHJo/nkPzxvNn/mryHD6sR7YILwAjIiIiIrPFMEtEREREZothtpqpVCrMmTMHKpVK6lKokngOzR/PofnjOTRvPH/mrzafw3p3ARgRERER1R3smSUiIiIis8UwS0RERERmi2GWiIiIiMwWwywRERERmS2GWRP4/vvv4enpCSsrK/j4+ODw4cPlLr9+/Xq0a9cOVlZW6Ny5M7Zu3VpDlVJZjDmHP/30E/r3748GDRqgQYMGCAwMfOg5p+pn7PdhkTVr1kAmkyEkJKR6C6SHMvYc3r17Fy+++CLc3NygUqnQpk0b/jyVkLHnb9GiRWjbti2sra3h4eGB1157DXl5eTVULT1o7969CA4ORpMmTSCTybBp06aHrhMbG4vu3btDpVKhVatWCA8Pr/Y6SyWoStasWSOUSqX45ZdfxJkzZ8QzzzwjHB0dRWpqaqnLHzhwQCgUCvHFF1+Is2fPivfff19YWlqKU6dO1XDlVMTYczhp0iTx/fffixMnToiEhAQxZcoUoVarxb///lvDlVMRY89hkcuXL4umTZuK/v37izFjxtRMsVQqY89hfn6+6NmzpxgxYoTYv3+/uHz5soiNjRXx8fE1XDkJYfz5W7VqlVCpVGLVqlXi8uXLYvv27cLNzU289tprNVw5Fdm6dat47733xIYNGwQAsXHjxnKXv3TpkrCxsREzZ84UZ8+eFf/5z3+EQqEQ0dHRNVPwfRhmq6h3797ixRdfNDzX6XSiSZMmYv78+aUuP2HCBDFy5MhibT4+PmL69OnVWieVzdhz+KDCwkJhb28vfv311+oqkR6iMuewsLBQ9OvXT/z8889i8uTJDLMSM/Yc/vDDD6Jly5ZCq9XWVIlUDmPP34svvigGDx5crG3mzJnC19e3WuukiqlImH3rrbdEx44di7VNnDhRBAUFVWNlpeMwgyrQarU4duwYAgMDDW1yuRyBgYGIi4srdZ24uLhiywNAUFBQmctT9arMOXxQTk4OCgoK0LBhw+oqk8pR2XP40UcfwdnZGdOmTauJMqkclTmHUVFR6Nu3L1588UW4uLigU6dOmDdvHnQ6XU2VTf9TmfPXr18/HDt2zDAU4dKlS9i6dStGjBhRIzVT1dWmPGNR43usQ27dugWdTgcXF5di7S4uLjh37lyp66SkpJS6fEpKSrXVSWWrzDl80Ntvv40mTZqU+KammlGZc7h//34sW7YM8fHxNVAhPUxlzuGlS5ewa9cuPPbYY9i6dSsuXLiAF154AQUFBZgzZ05NlE3/U5nzN2nSJNy6dQt+fn4QQqCwsBDPPfcc3n333ZoomUygrDyTmZmJ3NxcWFtb11gt7JklqoLPPvsMa9aswcaNG2FlZSV1OVQBWVlZeOKJJ/DTTz/ByclJ6nKokvR6PZydnbF06VL06NEDEydOxHvvvYclS5ZIXRpVQGxsLObNm4fFixfj+PHj2LBhA7Zs2YKPP/5Y6tLIDLFntgqcnJygUCiQmpparD01NRWurq6lruPq6mrU8lS9KnMOi3z11Vf47LPPsGPHDnTp0qU6y6RyGHsOL168iCtXriA4ONjQptfrAQAWFhZITEyEl5dX9RZNxVTm+9DNzQ2WlpZQKBSGtvbt2yMlJQVarRZKpbJaa6b/V5nz98EHH+CJJ57A008/DQDo3LkzNBoNnn32Wbz33nuQy9nXVtuVlWccHBxqtFcWYM9slSiVSvTo0QM7d+40tOn1euzcuRN9+/YtdZ2+ffsWWx4AYmJiylyeqldlziEAfPHFF/j4448RHR2Nnj171kSpVAZjz2G7du1w6tQpxMfHGx6jR4/GoEGDEB8fDw8Pj5osn1C570NfX19cuHDB8IcIAJw/fx5ubm4MsjWsMucvJyenRGAt+sNECFF9xZLJ1Ko8U+OXnNUxa9asESqVSoSHh4uzZ8+KZ599Vjg6OoqUlBQhhBBPPPGEmDVrlmH5AwcOCAsLC/HVV1+JhIQEMWfOHE7NJTFjz+Fnn30mlEqliIiIEMnJyYZHVlaWVIdQ7xl7Dh/E2QykZ+w5vHbtmrC3txcvvfSSSExMFJs3bxbOzs7ik08+keoQ6jVjz9+cOXOEvb29+P3338WlS5fEX3/9Jby8vMSECROkOoR6LysrS5w4cUKcOHFCABALFy4UJ06cEFevXhVCCDFr1izxxBNPGJYvmprrzTffFAkJCeL777/n1Fzm7D//+Y9o1qyZUCqVonfv3uK///2v4TV/f38xefLkYsuvW7dOtGnTRiiVStGxY0exZcuWGq6YHmTMOWzevLkAUOIxZ86cmi+cDIz9Prwfw2ztYOw5PHjwoPDx8REqlUq0bNlSfPrpp6KwsLCGq6Yixpy/goIC8eGHHwovLy9hZWUlPDw8xAsvvCDu3LlT84WTEEKI3bt3l/q7rei8TZ48Wfj7+5dYx9vbWyiVStGyZUuxfPnyGq9bCCFkQrA/n4iIiIjME8fMEhEREZHZYpglIiIiIrPFMEtEREREZothloiIiIjMFsMsEREREZkthlkiIiIiMlsMs0RERERkthhmiYiIiMhsMcwSEQEIDw+Ho6Oj1GVUmkwmw6ZNm8pdZsqUKQgJCamReoiIagrDLBHVGVOmTIFMJivxuHDhgtSlITw83FCPXC6Hu7s7pk6dirS0NJNsPzk5GcOHDwcAXLlyBTKZDPHx8cWW+eabbxAeHm6S/ZXlww8/NBynQqGAh4cHnn32WaSnpxu1HQZvIqooC6kLICIypWHDhmH58uXF2ho3bixRNcU5ODggMTERer0eJ0+exNSpU3Hjxg1s3769ytt2dXV96DJqtbrK+6mIjh07YseOHdDpdEhISMBTTz2FjIwMrF27tkb2T0T1C3tmiahOUalUcHV1LfZQKBRYuHAhOnfuDFtbW3h4eOCFF15AdnZ2mds5efIkBg0aBHt7ezg4OKBHjx44evSo4fX9+/ejf//+sLa2hoeHB1555RVoNJpya5PJZHB1dUWTJk0wfPhwvPLKK9ixYwdyc3Oh1+vx0Ucfwd3dHSqVCt7e3oiOjjasq9Vq8dJLL8HNzQ1WVlZo3rw55s+fX2zbRcMMWrRoAQDo1q0bZDIZBg4cCKB4b+fSpUvRpEkT6PX6YjWOGTMGTz31lOH5H3/8ge7du8PKygotW7bE3LlzUVhYWO5xWlhYwNXVFU2bNkVgYCDGjx+PmJgYw+s6nQ7Tpk1DixYtYG1tjbZt2+Kbb74xvP7hhx/i119/xR9//GHo5Y2NjQUAXL9+HRMmTICjoyMaNmyIMWPG4MqVK+XWQ0R1G8MsEdULcrkc3377Lc6cOYNff/0Vu3btwltvvVXm8o899hjc3d1x5MgRHDt2DLNmzYKlpSUA4OLFixg2bBhCQ0Px999/Y+3atdi/fz9eeuklo2qytraGXq9HYWEhvvnmGyxYsABfffUV/v77bwQFBWH06NH4559/AADffvstoqKisG7dOiQmJmLVqlXw9PQsdbuHDx8GAOzYsQPJycnYsGFDiWXGjx+P27dvY/fu3Ya29PR0REdH47HHHgMA7Nu3D08++SRmzJiBs2fP4scff0R4eDg+/fTTCh/jlStXsH37diiVSkObXq+Hu7s71q9fj7Nnz2L27Nl49913sW7dOgDAG2+8gQkTJmDYsGFITk5GcnIy+vXrh4KCAgQFBcHe3h779u3DgQMHYGdnh2HDhkGr1Va4pv9r735Dmuz6OIB/nxk2XTOwktwLC9INoayWq8wisn9GxnCJlkJCZqKpoRX1wrQRWhZOKPojiEY2UgwiydTohWULwv6oULllzf4QBBko0pbmzvMivLiX07Abnufe7u8H9uJc1/md63euvfnt7FwbEfkYQUTkI9LT04Wfn59QKBTSKykpyWPfxsZGMWfOHKldW1srZs+eLbWVSqW4cuWKx9iMjAyxf/9+t2MdHR1CJpMJh8PhMebX8W02m1Cr1SI6OloIIYRKpRKlpaVuMTqdTuTk5AghhMjLyxNxcXHC5XJ5HB+AuHnzphBCCLvdLgCI58+fu/VJT08Xer1eauv1erF3716pXVVVJVQqlRgbGxNCCLFx40ZRVlbmNkZdXZ0IDQ31mIMQQpSUlAiZTCYUCoWQy+UCgAAgTCbTpDFCCHHgwAGxc+fOSXMdv7ZGo3G7B9+/fxcBAQGira1tyvGJyHdxzywR+ZQNGzbg0qVLUluhUAD4uUp56tQp9Pb2YmhoCD9+/IDT6cS3b98QGBg4YZzCwkLs27cPdXV10lflixYtAvBzC0JPTw/MZrPUXwgBl8sFu92OyMhIj7kNDg5i1qxZcLlccDqdWLt2LaqrqzE0NIRPnz4hNjbWrX9sbCy6u7sB/NwisHnzZmg0GsTHxyMhIQFbtmz5W/cqLS0NmZmZuHjxImbOnAmz2Yxdu3ZBJpNJ87RYLG4rsWNjY1PeNwDQaDRoamqC0+nEtWvX0NXVhby8PLc+Fy5cQE1NDd6/fw+Hw4GRkREsW7Zsyny7u7vR19cHpVLpdtzpdOLNmzd/cAeIyBewmCUin6JQKBAeHu52rL+/HwkJCcjOzkZpaSmCg4Px8OFDZGRkYGRkxGNRduLECaSmpqK5uRktLS0oKSlBfX09EhMTMTw8jKysLOTn50+ICwsLmzQ3pVKJZ8+eQSaTITQ0FAEBAQCAoaGh385Lq9XCbrejpaUF9+7dQ3JyMjZt2oQbN278NnYyO3bsgBACzc3N0Ol06OjoQGVlpXR+eHgYRqMRBoNhQqxcLp90XH9/f+k9OH36NLZv3w6j0YiTJ08CAOrr63H48GFUVFQgJiYGSqUSZ8+exePHj6fMd3h4GCtWrHD7EDHun/KQHxH977GYJSKf9/TpU7hcLlRUVEirjuP7M6eiVquhVqtRUFCA3bt3o7a2FomJidBqtXj58uWEovl3ZDKZx5igoCCoVCpYLBasX79eOm6xWLBy5Uq3fikpKUhJSUFSUhLi4+Px9etXBAcHu403vj91bGxsynzkcjkMBgPMZjP6+vqg0Wig1Wql81qtFlarddrz/FVRURHi4uKQnZ0tzXPNmjXIycmR+vy6surv7z8hf61Wi4aGBoSEhCAoKOhv5UREvoMPgBGRzwsPD8fo6CjOnz+Pt2/foq6uDpcvX560v8PhQG5uLtrb2/Hu3TtYLBZ0dnZK2weOHj2KR48eITc3F11dXXj9+jVu3bo17QfA/urIkSMoLy9HQ0MDrFYrjh07hq6uLhw8eBAAYDKZcP36dfT29sJms6GxsRHz58/3+EcPISEhCAgIQGtrKz5//ozBwcFJr5uWlobm5mbU1NRID36NKy4uxtWrV2E0GvHixQu8evUK9fX1KCoqmtbcYmJiEBUVhbKyMgBAREQEnjx5gra2NthsNhw/fhydnZ1uMQsXLkRPTw+sViu+fPmC0dFRpKWlYe7cudDr9ejo6IDdbkd7ezvy8/Px8ePHaeVERL6DxSwR+bylS5fCZDKhvLwcixcvhtlsdvtZq1/5+flhYGAAe/bsgVqtRnJyMrZt2waj0QgAiIqKwv3792Gz2bBu3TosX74cxcXFUKlUf5xjfn4+CgsLcejQISxZsgStra1oampCREQEgJ9bFM6cOYPo6GjodDr09/fjzp070krzX82YMQPnzp1DVVUVVCoV9Hr9pNeNi4tDcHAwrFYrUlNT3c5t3boVt2/fxt27d6HT6bB69WpUVlZiwYIF055fQUEBqqur8eHDB2RlZcFgMCAlJQWrVq3CwMCA2yotAGRmZkKj0SA6Ohrz5s2DxWJBYGAgHjx4gLCwMBgMBkRGRiIjIwNOp5MrtUT/Yv8RQoj/dxJERERERH+CK7NERERE5LVYzBIRERGR12IxS0RERERei8UsEREREXktFrNERERE5LVYzBIRERGR12IxS0RERERei8UsEREREXktFrNERERE5LVYzBIRERGR12IxS0RERERe678HtVHW2gc9hwAAAABJRU5ErkJggg==",
      "text/plain": [
       "<Figure size 800x600 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import matplotlib.pyplot as plt # Import pyplot\n",
    "from sklearn.metrics import roc_curve # Import roc_curve\n",
    "\n",
    "fpr, tpr, _ = roc_curve(y_test, y_pred_prob)\n",
    "plt.figure(figsize=(8, 6))\n",
    "plt.plot(fpr, tpr, label=f\"Logistic Regression (AUC = {auc:.2f})\", color=\"blue\")\n",
    "plt.plot([0, 1], [0, 1], linestyle=\"--\", color=\"gray\")  # 参考线\n",
    "plt.xlabel(\"False Positive Rate\")\n",
    "plt.ylabel(\"True Positive Rate\")\n",
    "plt.title(\"ROC Curve for Logistic Regression Model\")\n",
    "plt.legend()\n",
    "plt.show()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
